+++
# Date this page was created.
date = "2018-04-11"

# Project title.
title = "Protein Databases"

weight = 70
# Project summary to display on homepage.
summary = ""

# Optional image to display on homepage (relative to `static/img/` folder).
#image_preview = "bubbles.jpg"

# Tags: can be used for filtering projects.
# Example: `tags = ["machine-learning", "deep-learning"]`
tags = ["Databases"]

# Optional external URL for project (replaces project detail page).
external_link = ""

# Does the project detail page use math formatting?
math = false

# Optional featured image (relative to `static/img/` folder).
#[header]
#image = "headers/bubbles-wide.jpg"
#caption = "My caption :smile:"


+++
---

## Protein Sequence Databases

* A protein sequence database refers to a database that uses computer functions to analyze biological information. Computer algorithms are used to compare DNA and protein sequences to detect evolutionary relationships among structures, functions, and sequences. The sequences of various genomes produce a large amount of DNA sequence data and biological information, and have been applied to study the function of genes to predict previously unknown gene functions. 
![proseq](/img/database/proseq.jpg) 
* Now people's attention is mainly focused on predicting protein structure and function from the only amino acid sequence.


[Wekipedia](https://en.wikipedia.org/wiki/Sequence_analysis#Protein_Structure_Prediction)


---
{{% alert note %}}
**Database:** 
{{% /alert %}}

### [ENZYME](https://enzyme.expasy.org/)

**ENZYME** is a repository of information relative to the nomenclature of enzymes.And it describes each type of characterized enzyme for which an EC (Enzyme Commission) number has been provided

### [Interpro](http://www.ebi.ac.uk/interpro/)

**InterPro** provides functional analysis of proteins by classifying them into families and predicting domains and important sites.

### [KABAT](https://labworm.com/tool/kabat-database)

The **Kabat** database includes nucleotide sequences, sequences of T cell receptors for antigens (TCR), major histocompatibility complex (MHC) class I and II molecules, and other proteins of immunological interest.

### [PIR](https://pir.georgetown.edu/)

**Protein Information Resource**, part of UniProt

### [Uniport](http://www.uniprot.org/)

**The Universal Protein Resource** (UniProt) provides the scientific community with a single, centralized, authoritative resource for protein sequences and functional information.

---

## Protein structure Databases

* **Protein structure** is the three-dimensional arrangement of atoms in an amino acid-chain molecule. Proteins are polymers – specifically polypeptides – formed from sequences of amino acids, the monomers of the polymer. A single amino acid monomer may also be called a residue indicating a repeating unit of a polymer.
![prostr](/img/database/prostr.png) 
* Protein structures range in size from tens to several thousand amino acids.[2] By physical size, proteins are classified as nanoparticles, between 1–100 nm. Very large aggregates can be formed from protein subunits. For example, many thousands of actin molecules assemble into a microfilament.
* A protein generally undergo reversible structural changes in performing its biological function. The alternative structures of the same protein are referred to as different conformational isomers, or simply, conformations, and transitions between them are called conformational changes.

[Wekipedia](https://en.wikipedia.org/wiki/Protein_structure)


---
{{% alert note %}}
**Database:** 
{{% /alert %}}

### [CATH](http://www.cathdb.info/)

**The CATH database** is a free, publicly available online resource that provides information on the evolutionary relationships of protein domains.(95 million protein domains classified into 6,119 superfamilies)

### [PDB](https://www.rcsb.org/)

**Protein Data Bank** builds upon archive-information about the 3D shapes of proteins, nucleic acids, and complex assemblies that helps students and researchers understand all aspects of biomedicine and agriculture, from protein synthesis to health and disease.

### [PMP](https://www.proteinmodelportal.org/)

**Protein model portal** (PMP) gives access to various models computed by comparative modeling methods provided by different partner sites, and provides access to various interactive services for model building, and quality assessment.

### [Prosite](https://prosite.expasy.org/)

**PROSITE** consists of documentation entries describing protein domains, families and functional sites as well as associated patterns and profiles to identify them.

### [SCOP2](http://scop2.mrc-lmb.cam.ac.uk/)

SCOP2 is a successor of **Structural classification of proteins** (SCOP).  the main focus of SCOP2 is on proteins that are structurally characterized and deposited in the PDB.

### [SWISS-model](https://swissmodel.expasy.org/repository/)

The **SWISS-MODEL** Repository is a database of annotated 3D protein structure models generated by the SWISS-MODEL homology-modelling pipeline.

*Image Citation*:[*1*](https://www.news-medical.net/life-sciences/Amino-Acids-and-Protein-Sequences.aspx) [*2*](https://www.khanacademy.org/science/biology/macromolecules/proteins-and-amino-acids/a/orders-of-protein-structure)

---
