<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.38.2" />
  <meta name="author" content="Computational Biology Lab">

  
  
  
  
    
      
    
  
  <meta name="description" content="Table of Contents  Microscopic Phenotype Analysis  Microscopic Image Formats Management  OMERO | OME Remote Objects SCIFIO | SCientific Image Format Input and Output  PheWAS  PheWAS | Phenome-wide association study SPAtest   Laser Scanning Microscopy  FISH  ImageJ GoIFISH  Colocalization Analysis  BioImageXD ViBE-Z | Virtual Brain Explorer for Zebrafish  3D Image Analysis  u-track UltraTracer   Conventional Fluorescence microscopy  High-throughput Screening  histoCAT | HIStology TOpography Cytometry Analysis Toolbox CellProfiler  Cell Phenotyping  PhenoRipper histoCAT | HIStology TOpography Cytometry Analysis Toolbox   Super-resolution Imaging  Morphometric Analysis  MIB | Microscopy Image Browser BioImageXD  Single Molecule Detection and Localization  rapidSTORM ViSP   Cryo-EM  Neuron Tracing  UltraTracer webKnossos  Single Particle Analysis  EMAN  National Center for Macromolecular Imaging, The Verna and Marrs McLean Department of Biochemistry and Molecular Biology, Baylor College of Medicine, One Baylor Plaza, Houston, TX, USA  EMRinger   Find more tools: OMICTOOLS   Microscopic Phenotype Analysis Microscopic phenotype analysis deals with all tools usefull for describing, identifying or classifying phenotypes at microscopic scale.">

  
  <link rel="alternate" hreflang="en-us" href="/too2/new_pheno/bioimaging/">

  


  

  
  
  <meta name="theme-color" content="#3f51b5">
  
  
  
  
    
  
  
    
    
      
        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
      
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">
  
  
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7cMerriweather%7cRoboto&#43;Mono">
  
  <link rel="stylesheet" href="/styles.css">
  

  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/too2/new_pheno/bioimaging/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="og:site_name" content="CityU-Bioinformatics">
  <meta property="og:url" content="/too2/new_pheno/bioimaging/">
  <meta property="og:title" content="Bioimaging | CityU-Bioinformatics">
  <meta property="og:description" content="Table of Contents  Microscopic Phenotype Analysis  Microscopic Image Formats Management  OMERO | OME Remote Objects SCIFIO | SCientific Image Format Input and Output  PheWAS  PheWAS | Phenome-wide association study SPAtest   Laser Scanning Microscopy  FISH  ImageJ GoIFISH  Colocalization Analysis  BioImageXD ViBE-Z | Virtual Brain Explorer for Zebrafish  3D Image Analysis  u-track UltraTracer   Conventional Fluorescence microscopy  High-throughput Screening  histoCAT | HIStology TOpography Cytometry Analysis Toolbox CellProfiler  Cell Phenotyping  PhenoRipper histoCAT | HIStology TOpography Cytometry Analysis Toolbox   Super-resolution Imaging  Morphometric Analysis  MIB | Microscopy Image Browser BioImageXD  Single Molecule Detection and Localization  rapidSTORM ViSP   Cryo-EM  Neuron Tracing  UltraTracer webKnossos  Single Particle Analysis  EMAN  National Center for Macromolecular Imaging, The Verna and Marrs McLean Department of Biochemistry and Molecular Biology, Baylor College of Medicine, One Baylor Plaza, Houston, TX, USA  EMRinger   Find more tools: OMICTOOLS   Microscopic Phenotype Analysis Microscopic phenotype analysis deals with all tools usefull for describing, identifying or classifying phenotypes at microscopic scale.">
  <meta property="og:locale" content="en-us">
  
  <meta property="article:published_time" content="2018-04-11T00:00:00&#43;00:00">
  
  <meta property="article:modified_time" content="2018-04-11T00:00:00&#43;00:00">
  

  

  <title>Bioimaging | CityU-Bioinformatics</title>

</head>
<body id="top" data-spy="scroll" data-target="#toc" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/">CityU-Bioinformatics</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#bioinformatics">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/program/r">
            
            <span>R Programming</span>
            
          </a>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Course Tutorials</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/tutorial/bms4001/">
                
                <span>BMS4001 tutorials</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/bms8110/">
                
                <span>BMS8110 tutorials</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/resources/">
                
                <span>Online Resources</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Database</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/base2/dna">
                
                <span>DNA Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/express">
                
                <span>Genome Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/rna">
                
                <span>RNA Databses</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/protein">
                
                <span>Protein Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pheno">
                
                <span>Phenotype Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pathway">
                
                <span>Pathway Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/special">
                
                <span>Other Databases</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Tools</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/too2/genomics">
                
                <span>Genomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/epigenomics">
                
                <span>Epigenomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/transcriptomics">
                
                <span>Transcriptomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/proteomics">
                
                <span>Proteomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/metabolomics">
                
                <span>Metabolomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/phenomics">
                
                <span>Phenomics</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Journals</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/journal/comprehensive">
                
                <span>Multidisciplinary</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/specific">
                
                <span>Bioinformatics Specific</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/conferences">
                
                <span>Conferences</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/server/">
            
            <span>Servers</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>


<article class="article" itemscope itemtype="http://schema.org/Article">

  


  <div class="article-container">
    <h1 itemprop="name">Bioimaging</h1>

    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-04-11 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Apr 11, 2018
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Computational Biology Lab">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    15 min read
  </span>
  

  
  

  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=Bioimaging&amp;url=%2ftoo2%2fnew_pheno%2fbioimaging%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=%2ftoo2%2fnew_pheno%2fbioimaging%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-facebook"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=%2ftoo2%2fnew_pheno%2fbioimaging%2f&amp;title=Bioimaging"
         target="_blank" rel="noopener">
        <i class="fa fa-linkedin"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=%2ftoo2%2fnew_pheno%2fbioimaging%2f&amp;title=Bioimaging"
         target="_blank" rel="noopener">
        <i class="fa fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=Bioimaging&amp;body=%2ftoo2%2fnew_pheno%2fbioimaging%2f">
        <i class="fa fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>


    <div class="article-style" itemprop="articleBody">
      

<hr />

<p><img src="/img/tools/pheno/bioimage.jpg"  width="400" height="400" alt="anno" align="center"></p>

<p><span id="top"></span></p>

<h2>Table of Contents</h2>
<nav id="TableOfContents">
<ul>
<li><a href="#microscopic-phenotype-analysis">Microscopic Phenotype Analysis</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-microscopic-image-formats-management-font"><font color=#CD5555 face="黑体">Microscopic Image Formats Management</font></a>
<ul>
<li><a href="#omero-ome-remote-objects">OMERO | OME Remote Objects</a></li>
<li><a href="#scifio-scientific-image-format-input-and-output">SCIFIO | SCientific Image Format Input and Output</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-phewas-font"><font color=#CD5555 face="黑体">PheWAS</font></a>
<ul>
<li><a href="#phewas-phenome-wide-association-study">PheWAS | Phenome-wide association study</a></li>
<li><a href="#spatest">SPAtest</a></li>
</ul></li>
</ul></li>
<li><a href="#laser-scanning-microscopy">Laser Scanning Microscopy</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-fish-font"><font color=#CD5555 face="黑体">FISH</font></a>
<ul>
<li><a href="#imagej">ImageJ</a></li>
<li><a href="#goifish">GoIFISH</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-colocalization-analysis-font"><font color=#CD5555 face="黑体">Colocalization Analysis</font></a>
<ul>
<li><a href="#bioimagexd">BioImageXD</a></li>
<li><a href="#vibe-z-virtual-brain-explorer-for-zebrafish">ViBE-Z | Virtual Brain Explorer for Zebrafish</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-3d-image-analysis-font"><font color=#CD5555 face="黑体">3D Image Analysis</font></a>
<ul>
<li><a href="#u-track">u-track</a></li>
<li><a href="#ultratracer">UltraTracer</a></li>
</ul></li>
</ul></li>
<li><a href="#conventional-fluorescence-microscopy">Conventional Fluorescence microscopy</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-high-throughput-screening-font"><font color=#CD5555 face="黑体">High-throughput Screening</font></a>
<ul>
<li><a href="#histocat-histology-topography-cytometry-analysis-toolbox">histoCAT | HIStology TOpography Cytometry Analysis Toolbox</a></li>
<li><a href="#cellprofiler">CellProfiler</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-cell-phenotyping-font"><font color=#CD5555 face="黑体">Cell Phenotyping</font></a>
<ul>
<li><a href="#phenoripper">PhenoRipper</a></li>
<li><a href="#histocat-histology-topography-cytometry-analysis-toolbox-1">histoCAT | HIStology TOpography Cytometry Analysis Toolbox</a></li>
</ul></li>
</ul></li>
<li><a href="#super-resolution-imaging">Super-resolution Imaging</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-morphometric-analysis-font"><font color=#CD5555 face="黑体">Morphometric Analysis</font></a>
<ul>
<li><a href="#mib-microscopy-image-browser">MIB | Microscopy Image Browser</a></li>
<li><a href="#bioimagexd-1">BioImageXD</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-single-molecule-detection-and-localization-font"><font color=#CD5555 face="黑体">Single Molecule Detection and Localization</font></a>
<ul>
<li><a href="#rapidstorm">rapidSTORM</a></li>
<li><a href="#visp">ViSP</a></li>
</ul></li>
</ul></li>
<li><a href="#cryo-em">Cryo-EM</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-neuron-tracing-font"><font color=#CD5555 face="黑体">Neuron Tracing</font></a>
<ul>
<li><a href="#ultratracer-1">UltraTracer</a></li>
<li><a href="#webknossos">webKnossos</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-single-particle-analysis-font"><font color=#CD5555 face="黑体">Single Particle Analysis</font></a>
<ul>
<li><a href="#eman">EMAN</a></li>
</ul></li>
<li><a href="#national-center-for-macromolecular-imaging-the-verna-and-marrs-mclean-department-of-biochemistry-and-molecular-biology-baylor-college-of-medicine-one-baylor-plaza-houston-tx-usa">National Center for Macromolecular Imaging, The Verna and Marrs McLean Department of Biochemistry and Molecular Biology, Baylor College of Medicine, One Baylor Plaza, Houston, TX, USA</a>
<ul>
<li><a href="#emringer">EMRinger</a></li>
</ul></li>
</ul></li>
<li><a href="#find-more-tools-omictools-https-omictools-com-phenomics-category">Find more tools: <a href="https://omictools.com/phenomics-category" target="_blank"><strong>OMICTOOLS</strong></a></a></li>
</ul>
</nav>


<h1 id="microscopic-phenotype-analysis">Microscopic Phenotype Analysis</h1>

<p><p align="justify"><strong>Microscopic phenotype analysis</strong> deals with all tools usefull for describing, identifying or classifying phenotypes at microscopic scale. Microscopic sample can, for example, be cells or microscopic organisms (worms, parasites..) imaged by all kinds of microscopic devices.</p>

<h2 id="font-color-cd5555-face-黑体-microscopic-image-formats-management-font"><font color=#CD5555 face="黑体">Microscopic Image Formats Management</font></h2>

<p><p align="justify">Microscopy image formats management tools are used to convert image files into desired formats, add metadata, facilitate open-exchange, and more.</p>

<hr />

<h3 id="omero-ome-remote-objects">OMERO | OME Remote Objects</h3>

<p><p align="justify">A software platform that enables access to and use of a wide range of biological data. OMERO uses a server-based middleware application to provide a unified interface for images, matrices and tables. OMERO&rsquo;s design and flexibility have enabled its use for light-microscopy, high-content-screening, electron-microscopy and even non-image-genotype data.</p>

<p><a href="http://www.openmicroscopy.org/omero/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2012 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22373911" target="_blank">OMERO: flexible, model-driven data management for experimental biology.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Wellcome Trust Centre for Gene Regulation and Expression, College of Life Sciences, University of Dundee, Dundee, Scotland, UK</p>

<hr />

<h3 id="scifio-scientific-image-format-input-and-output">SCIFIO | SCientific Image Format Input and Output</h3>

<p><p align="justify">A library software to create extensible framework for the reading, writing, and translation of images. SCIFIO was designed to provide the architecture that will equally facilitate: i) the conversion of additional formats into supported open-exchange formats such as OME-TIFF and ii) the integration of additional scientific open-exchange formats. The extensible design of SCIFIO facilitates community contribution, the establishment of domain-specific metadata standards, and integration into a unified system capable of adapting to the demands of scientific imaging analysis.</p>

<p><a href="http://scif.io/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2016 BMC Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27927161" target="_blank">SCIFIO: an extensible framework to support scientific image formats.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Laboratory for Optical and Computational Instrumentation, University of Wisconsin at Madison, Madison, WI, USA;</p>

<p>Morgridge Institute for Research, Madison, WI, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-phewas-font"><font color=#CD5555 face="黑体">PheWAS</font></h2>

<p><p align="justify">The genome-wide association study (GWAS) is a powerful approach for studying the genetic complexities of human disease. Unfortunately, GWASs often fail to identify clinically significant associations and describing function can be a challenge. GWAS is a phenotype-to-genotype approach. It is now possible to conduct a converse genotype-to-phenotype approach using extensive electronic medical records to define a phenome. This approach associates a single genetic variant with many phenotypes across the phenome and is called a phenome-wide association study (PheWAS).</p>

<hr />

<h3 id="phewas-phenome-wide-association-study">PheWAS | Phenome-wide association study</h3>

<p><p align="justify">Tests associations between a genetic variant or clinical factor of interest and a compendium of clinical outcomes or phenotypes. PheWAS finds disease-gene associations using ICD9 billing codes. It can serve to construct hypotheses around a variety of exposures. This tool implements a propensity score (PS) method for PS matching or PS adjustment. It employs a penalized maximum likelihood (PML) in logistic regression that can reduce bias in the parameter estimates.</p>

<p><a href="https://github.com/PheWAS/PheWAS" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li><p>(2018 Bioinformatics) <a href="https://doi.org/10.1093/bioinformatics/bty306" target="_blank">Evaluating statistical approaches to leverage large clinical datasets for uncovering therapeutic and adverse medication effects.</a></p></li>

<li><p>(2014 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24733291" target="_blank">R PheWAS: data analysis and plotting tools for phenome-wide association studies in the R environment.</a></p></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Biostatistics, Vanderbilt University Medical Center, Nashville, TN, USA;</p>

<p>Department of Biomedical Informatics, Vanderbilt University Medical Center, Nashville, TN, USA</p>

<hr />

<h3 id="spatest">SPAtest</h3>

<p><p align="justify">Performs a score test for a single variant based on saddlepoint approximation single variant test in genetic associations. SPAtest is an R package implements in the fastspa package that can accurate calculate p-values even for extremely unbalanced case-control outcomes. It provides a scalable solution for this large-scale problem and contribute to finding genetic component of complex traits.</p>

<p><a href="https://cran.r-project.org/web/packages/SPAtest/index.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Am J Hum Genet) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28602423" target="_blank">A Fast and Accurate Algorithm to Test for Binary Phenotypes and Its Application to PheWAS.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Biostatistics, University of Michigan, Ann Arbor, MI, USA;</p>

<p>Center for Statistical Genetics, University of Michigan, Ann Arbor, MI, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="laser-scanning-microscopy">Laser Scanning Microscopy</h1>

<p><p align="justify">Confocal microscopy is an established light microscopical technique for imaging fluorescently labeled specimens with significant three-dimensional structure. Applications of confocal microscopy in the biomedical sciences include the imaging of the spatial distribution of macromolecules in either fixed or living cells, the automated collection of 3D data, the imaging of multiple labeled specimens and the measurement of physiological events in living cells. The laser scanning confocal microscope continues to be chosen for most routine work although a number of instruments have been developed for more specific applications.</p>

<hr />

<h2 id="font-color-cd5555-face-黑体-fish-font"><font color=#CD5555 face="黑体">FISH</font></h2>

<p><p align="justify">Fluorescence in situ hybridization (FISH) is used to study the organization and the positioning of specific DNA sequences within the cell nucleus. Analyzing the data from FISH images is a tedious process that invokes an element of subjectivity.</p>

<hr />

<h3 id="imagej">ImageJ</h3>

<p><p align="justify">Provides easy installation on arbitrary platforms and a simple user interface. ImageJ is a platform for biological image analysis and an open source Java image processing program inspired by NIH Image. The ImageJ community can communicate through an interdisciplinary forum where knowledge is disseminated and questions that might appear difficult to experts in one scientific field may be easily answered by an expert in another field.</p>

<p><a href="https://imagej.net/Welcome" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 BMC Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29187165" target="_blank">ImageJ2: ImageJ for the next generation of scientific image data.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Laboratory for Optical and Computational Instrumentation, University of Wisconsin at Madison, Madison, WI, USA;</p>

<p>Morgridge Institute for Research, Madison, WI, USA</p>

<hr />

<h3 id="goifish">GoIFISH</h3>

<p><p align="justify">A widely applicable, user-friendly system tailored for the objective and semi-automated visualization, detection and quantification of genomic alterations and protein expression obtained from fluorescence in situ analysis. In a sample set of HER2-positive breast cancers GoIFISH is highly robust in visual analysis and its accuracy compares favorably to other leading image analysis methods.</p>

<p><a href="https://sourceforge.net/projects/goifish/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2014 Genome Biol) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25168174" target="_blank">GoIFISH: a system for the quantification of single cell heterogeneity from IFISH images.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>University of Cambridge, Cancer Research UK Cambridge Institute, Robinson Way, Cambridge, UK;</p>

<p>Department of Genetics, Institute for Cancer Research, Oslo, Norway;</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-colocalization-analysis-font"><font color=#CD5555 face="黑体">Colocalization Analysis</font></h2>

<p><p align="justify">The measurement of colocalization requires images of two fluorophores that are aligned, with no cross talk, and that the intensities remain within the response range of the microscope. Quantitation depends upon differentiating between the presence and absence of fluorescence, and measurements should be made within biologically relevant regions of interest. Co-occurrence can be measured simply by area or with the M1 and M2 coefficients, and should be compared to random distributions.</p>

<hr />

<h3 id="bioimagexd">BioImageXD</h3>

<p><p align="justify">Analyzes, processes and visualizes multi-dimensional microscopy images. BioImageXD puts open-source computer science tools for three-dimensional visualization and analysis into the hands of all researchers, through a user-friendly graphical interface tuned to the needs of biologists. BioImageXD has no restrictive licenses or undisclosed algorithms and enables publication of precise, reproducible and modifiable workflows. It allows simple construction of processing pipelines and should enable biologists to perform challenging analyses of complex processes.</p>

<p><a href="http://www.bioimagexd.net/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2012 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22743773" target="_blank">BioImageXD: an open, general-purpose and high-throughput image-processing platform.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Biochemistry and Food Chemistry, University of Turku, Turku, Finland</p>

<hr />

<h3 id="vibe-z-virtual-brain-explorer-for-zebrafish">ViBE-Z | Virtual Brain Explorer for Zebrafish</h3>

<p><p align="justify">An imaging and image analysis framework for virtual colocalization studies in larval zebrafish brains. ViBE-Z contains a database with precisely aligned gene expression patterns, an anatomical atlas, and a software. ViBE-Z software creates high-quality data sets through fusion of multiple confocal stacks, it aligns 3D gene or protein expression patterns of different zebrafish larvae of a given developmental stage to a common reference, and it stores the aligned patterns in the ViBE-Z database. Together with the anatomical segmentation (ViBE-Z atlas), this database allows users to explore the colocalization of all contained patterns in their anatomical context.</p>

<p><a href="http://vibez.informatik.uni-freiburg.de/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2012 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22706672" target="_blank">ViBE-Z: a framework for 3D virtual colocalization analysis in zebrafish larval brains.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Computer Science Department, Albert-Ludwigs-University Freiburg, Freiburg, Germany;</p>

<p>BIOSS Centre for Biological Signalling Studies, Freiburg, Germany</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-3d-image-analysis-font"><font color=#CD5555 face="黑体">3D Image Analysis</font></h2>

<p><p align="justify">Find and compare the best bioinformatics software tools for analyzing 3D laser scanning microscopy images. Tools are ranked by the biomedical research community.</p>

<hr />

<h3 id="u-track">u-track</h3>

<p><p align="justify">A multiple-particle tracking Matlab software that is designed to (1) track dense particle fields, (2) close gaps in particle trajectories resulting from detection failure, and (3) capture particle merging and splitting events resulting from occlusion or genuine aggregation and dissociation events. Its core is based on formulating correspondence problems as linear assignment problems and searching for a globally optimal solution.</p>

<p><a href="https://www.utsouthwestern.edu/labs/danuser/software/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2008 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18641657" target="_blank">Robust single-particle tracking in live-cell time-lapse sequences.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Cell Biology, The Scripps Research Institute, La Jolla, CA, USA</p>

<hr />

<h3 id="ultratracer">UltraTracer</h3>

<p><p align="justify">Designed to extend any base neuron-tracing algorithm to be able to trace virtually unlimited data volumes. UltraTracer was applied to neuron-tracing algorithms with different design principles and tested on challenging human and mouse neuron datasets that have hundreds of billions of voxels. Results indicate that UltraTracer is scalable, accurate, and about 3 to 6 times more efficient compared to other state-of-the-art approaches.</p>

<p><a href="https://github.com/Vaa3D/vaa3d_tools/tree/master/released_plugins/v3d_plugins/ultratracer" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28362437" target="_blank">Automatic tracing of ultra-volumes of neuronal images</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Allen Institute for Brain Science, Seattle, WA, USA;</p>

<p>Erasmus University Medical Center Rotterdam, Rotterdam, Netherlands;</p>

<p>Janelia Research Campus, Howard Hughes Medical Institute, Ashburn, VA, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="conventional-fluorescence-microscopy">Conventional Fluorescence microscopy</h1>

<p><p align="justify">The wide-field fluorescence microscope is a modification of the compound optical microscope, arranged to admit light of a selected wavelength to fluorescently labeled specimens and to image them through the emitted radiation. Since fluorescence emission has a lifetime in the nanosecond scale, continuous excitation is needed to collect an image having fluorescence as contrast mechanism. In a classical wide-field optical fluorescence microscope, the light source of a fluorescent microscope is a gas-vapor arc lamp, using either a discrete (Mercury) or a continuous (Xenon) light-emission spectrum.</p>

<hr />

<h2 id="font-color-cd5555-face-黑体-high-throughput-screening-font"><font color=#CD5555 face="黑体">High-throughput Screening</font></h2>

<p>Millions of cells are present in thousands of images created in high-throughput screening (HTS). Biologists could classify each of these cells into a phenotype by visual inspection. But in the presence of millions of cells this visual classification task becomes infeasible. Biologists train classification models on a few thousand visually classified example cells and iteratively improve the training data by visual inspection of the important misclassified phenotypes.</p>

<hr />

<h3 id="histocat-histology-topography-cytometry-analysis-toolbox">histoCAT | HIStology TOpography Cytometry Analysis Toolbox</h3>

<p><p align="justify">Uses a computational multiplexed image cytometry analysis toolbox to enable the interactive, quantitative, and comprehensive exploration of phenotypes of individual cells, cell-to-cell interactions, microenvironments, and morphological structures within intact tissues. miCAT will be useful in all areas of tissue-based research.</p>

<p><a href="https://github.com/BodenmillerGroup/histoCAT" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28783155" target="_blank">histoCAT: analysis of cell phenotypes and interactions in multiplex image cytometry data.</a></li>
</ul>

<hr />

<h3 id="cellprofiler">CellProfiler</h3>

<p><p align="justify">A free, open-source system designed for flexible, high-throughput cell image analysis. CellProfiler can address a variety of biological questions quantitatively, including standard assays (for example, cell count, size, per-cell protein levels) and complex morphological assays (for example, cell/organelle shape or subcellular patterns of DNA or protein staining).</p>

<p><a href="http://cellprofiler.org/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2006 Genome Biol) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17076895" target="_blank">CellProfiler: image analysis software for identifying and quantifying cell phenotypes.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Whitehead Institute for Biomedical Research, Cambridge, MA, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-cell-phenotyping-font"><font color=#CD5555 face="黑体">Cell Phenotyping</font></h2>

<p><p align="justify">Find and compare the best bioinformatics software tools for phenotying cells using conventional fluorescence microscopy images. Tools are ranked by the biomedical research community.</p>

<hr />

<h3 id="phenoripper">PhenoRipper</h3>

<p><p align="justify">Allows user to explore high-content microscopy images. PhenoRipper assists users in comparison of images and based on similarity of image phenotypes. It uses cluster analysis to identify superblock types, representing the most common block type co-occurrence patterns. It profiles each image by the frequency of occurrence of superblock types.</p>

<p><a href="http://awlab.ucsf.edu/Web_Site/PhenoRipper/default.htm" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2012 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22743764" target="_blank">PhenoRipper: software for rapidly profiling microscopy images.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Green Center for Systems Biology, University of Texas Southwestern Medical Center, Dallas, TX, USA;</p>

<p>Department of Pharmacology, University of Texas Southwestern Medical Center, Dallas, TX, USA</p>

<hr />

<h3 id="histocat-histology-topography-cytometry-analysis-toolbox-1">histoCAT | HIStology TOpography Cytometry Analysis Toolbox</h3>

<p><p align="justify">Uses a computational multiplexed image cytometry analysis toolbox to enable the interactive, quantitative, and comprehensive exploration of phenotypes of individual cells, cell-to-cell interactions, microenvironments, and morphological structures within intact tissues. miCAT will be useful in all areas of tissue-based research.</p>

<p><a href="https://github.com/BodenmillerGroup/histoCAT" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28783155" target="_blank">histoCAT: analysis of cell phenotypes and interactions in multiplex image cytometry data.</a></li>
</ul>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="super-resolution-imaging">Super-resolution Imaging</h1>

<p><p align="justify">In recent years, <strong>super-resolution</strong> optical imaging techniques have been developed to overcome the diffraction limit in fluorescence microscopy. Among these techniques, methods based on stochastic switching and localization of individual molecules, such as stochastic optical reconstruction microscopy (STORM) (<a href="https://www.ncbi.nlm.nih.gov/pubmed/16896339" target="_blank">Rust et al. 2006</a>) and (fluorescent) photoactivated localization microscopy ((F)PALM), achieve sub-diffraction-limit resolutions by sequentially imaging and localizing fluorescent emitters in the sample. <a href="https://www.ncbi.nlm.nih.gov/pubmed/25431749" target="_blank">Source</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-morphometric-analysis-font"><font color=#CD5555 face="黑体">Morphometric Analysis</font></h2>

<p><p align="justify">Find and compare the best bioinformatics software tools for analysing sample morphology on super-resolution microscopy images. Tools are ranked by the biomedical research community.</p>

<hr />

<h3 id="mib-microscopy-image-browser">MIB | Microscopy Image Browser</h3>

<p><p align="justify">Serves for effective segmentation of multidimensional datasets. MIB can recognize several number of imaging formats and offers a variety of image processing tools. It also simplifies utilization and quantification of acquired data. It permits users to segment large datasets, to realize 3D visualization, and to quantify images and models. Its parameters enable users to insert plugin s to customize the program for specific needs.</p>

<p><a href="http://mib.helsinki.fi/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2016 PLoS Biol) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26727152" target="_blank">Microscopy Image Browser: A Platform for Segmentation and Analysis of Multidimensional Datasets.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Electron Microscopy Unit, Institute of Biotechnology, University of Helsinki, Helsinki, Finland</p>

<hr />

<h3 id="bioimagexd-1">BioImageXD</h3>

<p><p align="justify">Analyzes, processes and visualizes multi-dimensional microscopy images. BioImageXD puts open-source computer science tools for three-dimensional visualization and analysis into the hands of all researchers, through a user-friendly graphical interface tuned to the needs of biologists. BioImageXD has no restrictive licenses or undisclosed algorithms and enables publication of precise, reproducible and modifiable workflows. It allows simple construction of processing pipelines and should enable biologists to perform challenging analyses of complex processes.</p>

<p><a href="http://www.bioimagexd.net/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2012 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22743773" target="_blank">BioImageXD: an open, general-purpose and high-throughput image-processing platform.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Biochemistry and Food Chemistry, University of Turku, Turku, Finland</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-single-molecule-detection-and-localization-font"><font color=#CD5555 face="黑体">Single Molecule Detection and Localization</font></h2>

<p><p align="justify">The quality of super-resolution images obtained by single-molecule localization microscopy (SMLM) depends largely on the software used to detect and accurately localize point sources.</p>

<hr />

<h3 id="rapidstorm">rapidSTORM</h3>

<p><p align="justify">A localization microscopy evaluation program. rapidSTORM is designed to find and precisely localize fluorophores in super-resolvable movies such as those generated by the PALM or dSTORM techniques.</p>

<p><a href="https://github.com/stevewolter/rapidSTORM" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2012 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23132113" target="_blank">rapidSTORM: accurate, fast open-source software for localization microscopy.</a></li>
</ul>

<p><strong>Issues</strong>:</p>

<p><a href="https://github.com/stevewolter/rapidSTORM/issues" target="_blank">https://github.com/stevewolter/rapidSTORM/issues</a></p>

<hr />

<h3 id="visp">ViSP</h3>

<p><p align="justify">An interactive, freely available, cross-platform 3D localization representation tool. ViSP advances the interpretation of 3D single-particle super-resolution data (and the features are also compatible with 2D data sets). With an ever-growing number of 3D microscopy techniques and applications available, the features that ViSP offers are of critical importance in the greater microscopy research community.</p>

<p><a href="https://science.institut-curie.org/research/multiscale-physics-biology-chemistry/umr168-physical-chemistry/team-dahan/softwares/visp-software-2/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2013 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23900246" target="_blank">ViSP: representing single-particle localizations in three dimensions.</a></li>
</ul>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="cryo-em">Cryo-EM</h1>

<p><p align="justify"><strong>Cryo-electron microscopy</strong> of biological matter has made important advances in the past decades. It has become a research tool that further expands the scope of structural research into unique areas of cell and molecular biology, and it could augment the materials research portfolio in the study of soft and hybrid materials.</p>

<h2 id="font-color-cd5555-face-黑体-neuron-tracing-font"><font color=#CD5555 face="黑体">Neuron Tracing</font></h2>

<p>Find and compare the best bioinformatics software tools for tracing neurons on cryo-Electron microscopy images. Tools are ranked by the biomedical research community.</p>

<hr />

<h3 id="ultratracer-1">UltraTracer</h3>

<p><p align="justify">Designed to extend any base neuron-tracing algorithm to be able to trace virtually unlimited data volumes. UltraTracer was applied to neuron-tracing algorithms with different design principles and tested on challenging human and mouse neuron datasets that have hundreds of billions of voxels. Results indicate that UltraTracer is scalable, accurate, and about 3 to 6 times more efficient compared to other state-of-the-art approaches.</p>

<p><a href="https://github.com/Vaa3D/vaa3d_tools/tree/master/released_plugins/v3d_plugins/ultratracer" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28362437" target="_blank">Automatic tracing of ultra-volumes of neuronal images</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Allen Institute for Brain Science, Seattle, WA, USA;</p>

<p>Erasmus University Medical Center Rotterdam, Rotterdam, Netherlands</p>

<hr />

<h3 id="webknossos">webKnossos</h3>

<p><p align="justify">Allows 3D image data annotation in various 3D image analysis settings in connectomics and other fields. webKnossos is an in-browser annotation tool. The software accelerates human 3D data interaction for electron microscopic (EM)-based connectomics in browser by about 4- to 13-fold, which likely saturates human interaction speed with 3D EM data of nervous tissue using flight mode.</p>

<p><a href="https://webknossos.org/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28604722" target="_blank">webKnossos: efficient online 3D data annotation for connectomics.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Connectomics, Max Planck Institute for Brain Research, Frankfurt, Germany;</p>

<p>Scalable minds UG (haftungsbeschränkt) &amp; Co. KG, Potsdam, Germany</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-single-particle-analysis-font"><font color=#CD5555 face="黑体">Single Particle Analysis</font></h2>

<p><p align="justify">Single particle electron microscopy is a versatile technique for the structural analysis of protein complexes in near-native conditions. While tremendous progress has been made during the past few decades in techniques for specimen preparation, imaging, and image analysis, the field is still in development.</p>

<hr />

<h3 id="eman">EMAN</h3>

<p><p align="justify">Reconstructs single particle from electron cryo-microscopy (cryoEM) images. EMAN is capable of processing large data sets and focuses on processing data from transmission electron microscopes. The application is mainly dedicated to single particle rebuilding but includes more than 40 functionalities allowing users to perform helical reconstruction or single particle tomography as well as interacting with external software such as Frealign or Relion.</p>

<p><a href="http://blake.bcm.tmc.edu/EMAN2/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li><p>(2016 Methods Enzymol) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27572727" target="_blank">Single-Particle Refinement and Variability Analysis in EMAN2.1.</a></p></li>

<li><p>(1999 J Struct Biol) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10600563" target="_blank">EMAN: semiautomated software for high-resolution single-particle reconstructions.</a></p></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<h2 id="national-center-for-macromolecular-imaging-the-verna-and-marrs-mclean-department-of-biochemistry-and-molecular-biology-baylor-college-of-medicine-one-baylor-plaza-houston-tx-usa">National Center for Macromolecular Imaging, The Verna and Marrs McLean Department of Biochemistry and Molecular Biology, Baylor College of Medicine, One Baylor Plaza, Houston, TX, USA</h2>

<h3 id="emringer">EMRinger</h3>

<p><p align="justify">A tool that assesses the precise fitting of an atomic model into the map during refinement and shows how radiation damage alters scattering from negatively charged amino acids. EMRinger is useful for monitoring progress in resolving and modeling high-resolution features in cryo-EM.</p>

<p><a href="https://github.com/fraser-lab/EMRinger" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2015 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26280328" target="_blank">EMRinger: side chain-directed model and map validation for 3D cryo-electron microscopy.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Bioengineering and Therapeutic Sciences, University of California, San Francisco, San Francisco, CA, USA;</p>

<p>Graduate Group in Biophysics, University of California, San Francisco, San Francisco, CA, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="find-more-tools-omictools-https-omictools-com-phenomics-category">Find more tools: <a href="https://omictools.com/phenomics-category" target="_blank"><strong>OMICTOOLS</strong></a></h1>

<p><a href="https://www.amazon.com/Microscope-OMAX-40X-2000X-Mechanical-Homeschool/dp/B006MX03Q0" target="_blank"><em>Image Citation</em></a></p>

<hr />

    </div>

    


<div class="article-tags">
  
  <a class="btn btn-primary btn-outline" href="/tags/phenomics" target="_blank">phenomics</a>
  
</div>




    
    
    <div class="article-widget">
      <div class="hr-light"></div>
      <h3>Related</h3>
      <ul>
        
        <li><a href="/too2/new_pheno/neurophysiology/">Neorophysiology</a></li>
        
        <li><a href="/too2/new_pheno/medicalimage/">Medical Imaging</a></li>
        
        <li><a href="/too2/new_pheno/neuroinformatics/">Computational Neuroinformatics</a></li>
        
        <li><a href="/too2/new_pheno/brain/">Brain Connectomics</a></li>
        
      </ul>
    </div>
    

    

    


  </div>
</article>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2018 &middot; 

      Copyright© 2018 Lin QI of Dr.Xin Wang's Lab. zzx. All rights reserved. 

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    <script async defer src="//maps.googleapis.com/maps/api/js?key=AIzaSyD3AGpKDAfo_LjBRHd-YkYmMjBDlIHfdlo"></script>
    

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gmaps.js/0.4.25/gmaps.min.js" integrity="sha256-7vjlAeb8OaTrCXZkCNun9djzuB2owUsaO72kXaFDBJs=" crossorigin="anonymous"></script>
    
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
      

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

