+++
# Date this page was created.
date = "2018-04-11"

# Project title.
title = "Lipidomics"

weight = 20
# Project summary to display on homepage.
summary = "Including: Lipidomics Databases, MS-based untargeted lipidomics, MS-based targeted lipidomics."

# Optional image to display on homepage (relative to `static/img/` folder).
image_preview = "tools/meta/lipidomics.jpg"

# Tags: can be used for filtering projects.
# Example: `tags = ["machine-learning", "deep-learning"]`
tags = ["metabolomics"]

# Optional external URL for project (replaces project detail page).
#external_link = "https://cityu-bioinformatics.netlify.com/tools/seq_new/sequence alignment/"


# Does the project detail page use math formatting?
math = false

# Optional featured image (relative to `static/img/` folder).
#[header]
#image = "headers/bubbles-wide.jpg"
#caption = "My caption :smile:"


+++
---

<img src="/img/tools/meta/lipidomics.jpg"  width="500" height="400" alt="anno" align="center">

<span id="top"></span>

{{% toc %}}

<p align="justify">Lipidomics or profiling specific classes of lipids biomarkers also provides a unique opportunity to identify diseases other than cancer. Biolipomics describes profiling classes of lipidomic analytes that are bioactive and may be a more direct analysis of functional lipomics.

---

# Lipidomics Databases

<p align="justify">Development of databases and the related bioinformatics tools has become an essential part of the omics studies. Over the recent years, empowered by high-throughput technologies for omics fields, the creation of databases devoted to lipids has been undertaken. Consequently, lipid-centered databases have been developed that enable researchers to comfortably analyze lipid species. 

---

## LIPID MAPS | LIPID Metabolites And Pathways Strategy

<p align="justify">Organizes lipids into eight well-defined categories that cover eukaryotic and prokaryotic sources. LIPID MAPS filters data in functional hierarchies involving lipids, reactions, and pathways. It is a multi-institutional effort created in 2003 and uses a systems biology approach and sophisticated mass spectrometers (MS), all of the major — and many minor — lipid species in mammalian cells. The database quantitates the changes in species in response to perturbation.

[**Official Website**](http://www.lipidmaps.org/)

**Publications**:

* (2012 J Chem Educ) [LIPID MAPS-Nature Lipidomics Gateway: An Online Resource for Students and Educators Interested in Lipids.](https://www.ncbi.nlm.nih.gov/pubmed/24764601)

**Institutions(s)**:

University of California, San Diego, La Jolla, CA, USA

---

## LipidBlast | Lipidomic spectral libraries : Lipidomics databases

<p align="justify">Current tandem mass spectral libraries for lipid annotations in metabolomics are limited in size and diversity. LipidBlast is a freely available computer-generated tandem mass spectral library of 212,516 spectra covering 119,200 compounds from 26 lipid compound classes, including phospholipids, glycerolipids, bacterial lipoglycans and plant glycolipids.

[**Official Website**](http://fiehnlab.ucdavis.edu/projects/LipidBlast)

**Publications**:

* (2013 Nat Methods) [LipidBlast in silico tandem mass spectrometry database for lipid identification.](https://www.ncbi.nlm.nih.gov/pubmed/23817071)

**Institutions(s)**:

Metabolics Core, UC Davis Genome Center, University of California, Davis, Davis, CA, USA

[<i class="fa fa-hand-o-up fa-1x "></i>Top](#top)

---

# MS-based untargeted lipidomics

<p align="justify">Due to its sensitivity and selectivity **mass spectrometry** (MS) is the method of choice for qualitative and quantitative lipidomic analysis. Although it is not yet possible to detect and quantify all individual lipids in a given cellular system, the aim of lipidomic analysis is to determine as many individual lipids as possible.

---

## LipidMatch | Lipid identification : MS-based untargeted lipidomics

<p align="justify">Allows annotation of lipids across a wide range of high resolution tandem mass spectrometry (MS/MS) studies. LipidMatch is based on an accurate assumption of multiple co-eluting lipids sharing m/z values. It was evaluated using data acquired from a Q-Exactive orbitrap mass spectrometer and an Agilent 6540 Q-TOF. The tool contains a large diversity in lipid types of any current open-source software platform and a rule-based strategy for identification and summed fragment intensity based strategy for ranking top hits.

[**Official Website**](http://secim.ufl.edu/secim-tools/lipidmatch/)

**Publications**:

* (2017 BMC Bioinformatics) [LipidMatch: an automated workflow for rule-based lipid identification using untargeted high-resolution tandem mass spectrometry data.](https://www.ncbi.nlm.nih.gov/pubmed/28693421)

**Institutions(s)**:

Department of Chemistry, University of Florida, Gainesville, FL, USA;

College of Engineering, University of Florida, Gainesville, FL, USA

---

## Quantinetix | Image signal quantification : Mass spectrometry

<p align="justify">Provides quantitation of target compounds (taking into account biological matrix effect) following Mass spectrometry imaging experiments. Quantinetix is a Quantitative Imaging Mass Spectrometry Software which offers normalization to get “real images” and provides concentration of target compounds. It is specially designed to ADMET study, PK/PD study, Toxicity study, In support of Whole Body Autoradiography and Proteomics and Lipidomics studies.

[**Official Website**](https://www.imabiotech.com/quantinetix/)

**Video**:

[Video](https://www.youtube.com/watch?v=jEWtjR7KUUs)

* (2010 Bioinformatics) [Metscape: a Cytoscape plug-in for visualizing and interpreting metabolomic data in the context of human metabolic networks.](https://www.ncbi.nlm.nih.gov/pubmed/20139469)

**MAINTAINER**:

ImaBiotech <fontaine.edma@imabiotech.com>

[<i class="fa fa-hand-o-up fa-1x "></i>Top](#top)

---

# MS-based targeted lipidomics

Taken together, LC-ESI-MS is currently the gold-standard technique for targeted lipidomics of second messengers when in ‘discovery’ mode with the research goal of identifying all possible species in a given tissue/ sample, whereas shotgun methodologies enable highthroughput identification and quantification of changes in known profiles.

---

## MRMPROBS | Multiple Reaction Monitoring-based PROBabilistic System

<p align="justify">Allows metabolome analysis of large-scale multiple reaction monitoring (MRM) assays. MRMPROBS is a universal program for targeted metabolomics using multiple reaction monitoring (MRM)- or selected reaction monitoring (SRM), as well as SCAN and data independent tandem mass spectrometry (MS/MS) acquisition (DIA) data. The software was developed for metabolomics but can also be applied to lipidomics and proteomics studies.

[**Official Website**](http://prime.psc.riken.jp/Metabolomics_Software/MRMPROBS/index.html)

**Publications**:

* (2014 Bioinformatics) [MRMPROBS suite for metabolomics using large-scale MRM assays.](https://www.ncbi.nlm.nih.gov/pubmed/24753485)

**Institutions(s)**:

RIKEN Center for Sustainable Resource Science, Tsurumi-ku, Yokohama, Kanagawa, Japan;

Department of Biotechnology, Graduate School of Engineering, Osaka University, Suita, Osaka, Japan

---

## MRM-DIFF | Multiple Reaction Monitoring-based DIFFerential

<p align="justify">Allows multiple reaction monitoring (MRM)-based differential analysis. MRM-DIFF supports chromatographic alignment and compound identification with estimation of isotopic peaks. The software semi-automatically performs lipid identification and quantification of large scale MRM datasets. It includes three features: (1) use of every peaks detected by each transition, (2) manual curation of the identification and quantification results through the GUI interface, and (3) support of all data processing steps and pooled quality control (QC) data sets.

[**Official Website**](http://prime.psc.riken.jp/Metabolomics_Software/MRMPROBS/index.html)

**Publications**:

* (2014 Front Genet) [MRM-DIFF: data processing strategy for differential analysis in large scale MRM-based lipidomics studies.](https://www.ncbi.nlm.nih.gov/pubmed/25688256)

**Institutions(s)**:

Metabolome Informatics Research Team, Metabolomics Research Group, RIKEN Center for Sustainable Resource Science, Yokohama, Japan

[<i class="fa fa-hand-o-up fa-1x "></i>Top](#top)

---

# Find more tools: [**OMICTOOLS**](https://omictools.com/metabolomics-category)

[*Image Citation*](https://www.thermofisher.com/hk/en/home/industrial/mass-spectrometry/mass-spectrometry-learning-center/mass-spectrometry-applications-area/metabolomics-mass-spectrometry/lipidomics-workflows.html)

---