<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.38.2" />
  <meta name="author" content="Computational Biology Lab">

  
  
  
  
    
      
    
  
  <meta name="description" content="Table of Contents  Metabolic profiles analysis  Metabolic Network analysis  BLASTX PiMP | Polyomics integrated Metabolomics Pipeline  Metabolite enrichment analysis  MetScape | Data integration : Mass spectrometry analysis FELLA | Metabolite enrichment analysis   MS-based untargeted Metabolomics  mGWAS  MWASTools | mGWAS RegScan   NMR-based Metabolomics/MS-based targeted metabolomics  Spectral corrections  NMRPipe NMRPro  Spectral deconvolution  NMRPipe BATMAN | Bayesian AuTomated Metabolite Analyser for NMR data  Chemometric analysis  MetaboAnalyst speaq | spectrum alignment and quantitation  Metabolite identification  W4M | Workflow4Metabolomics MetaboAnalyst   Find more tools: OMICTOOLS   Metabolic profiles analysis The further development of spectroscopic and spectrometric tools for high throughput analyses of selected biochemical pathways is crucial to the acquisition of metabolome data sets of sufficient quality for metabonomics and metabolomics.">

  
  <link rel="alternate" hreflang="en-us" href="/too2/new_meta/profiles/">

  


  

  
  
  <meta name="theme-color" content="#3f51b5">
  
  
  
  
    
  
  
    
    
      
        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
      
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">
  
  
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7cMerriweather%7cRoboto&#43;Mono">
  
  <link rel="stylesheet" href="/styles.css">
  

  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/too2/new_meta/profiles/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="og:site_name" content="CityU-Bioinformatics">
  <meta property="og:url" content="/too2/new_meta/profiles/">
  <meta property="og:title" content="Metabolic Profiles | CityU-Bioinformatics">
  <meta property="og:description" content="Table of Contents  Metabolic profiles analysis  Metabolic Network analysis  BLASTX PiMP | Polyomics integrated Metabolomics Pipeline  Metabolite enrichment analysis  MetScape | Data integration : Mass spectrometry analysis FELLA | Metabolite enrichment analysis   MS-based untargeted Metabolomics  mGWAS  MWASTools | mGWAS RegScan   NMR-based Metabolomics/MS-based targeted metabolomics  Spectral corrections  NMRPipe NMRPro  Spectral deconvolution  NMRPipe BATMAN | Bayesian AuTomated Metabolite Analyser for NMR data  Chemometric analysis  MetaboAnalyst speaq | spectrum alignment and quantitation  Metabolite identification  W4M | Workflow4Metabolomics MetaboAnalyst   Find more tools: OMICTOOLS   Metabolic profiles analysis The further development of spectroscopic and spectrometric tools for high throughput analyses of selected biochemical pathways is crucial to the acquisition of metabolome data sets of sufficient quality for metabonomics and metabolomics.">
  <meta property="og:locale" content="en-us">
  
  <meta property="article:published_time" content="2018-04-11T00:00:00&#43;00:00">
  
  <meta property="article:modified_time" content="2018-04-11T00:00:00&#43;00:00">
  

  

  <title>Metabolic Profiles | CityU-Bioinformatics</title>

</head>
<body id="top" data-spy="scroll" data-target="#toc" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/">CityU-Bioinformatics</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#bioinformatics">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/program/r">
            
            <span>R Programming</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/tutorial/resources/">
            
            <span>Online Resources</span>
            
          </a>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Course Tutorials</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/tutorial/bms4001/">
                
                <span>BMS4001 tutorials</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/bms8110/">
                
                <span>BMS8110 tutorials</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Databases</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/base2/dna">
                
                <span>DNA Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/express">
                
                <span>Genome Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/rna">
                
                <span>RNA Databses</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/protein">
                
                <span>Protein Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pheno">
                
                <span>Phenotype Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pathway">
                
                <span>Pathway Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/special">
                
                <span>Other Databases</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Tools</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/too2/genomics">
                
                <span>Genomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/epigenomics">
                
                <span>Epigenomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/transcriptomics">
                
                <span>Transcriptomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/proteomics">
                
                <span>Proteomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/metabolomics">
                
                <span>Metabolomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/phenomics">
                
                <span>Phenomics</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Journals</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/journal/comprehensive">
                
                <span>Multidisciplinary</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/specific">
                
                <span>Bioinformatics Specific</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/conferences">
                
                <span>Conferences</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/server/">
            
            <span>Servers</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>


<article class="article" itemscope itemtype="http://schema.org/Article">

  


  <div class="article-container">
    <h1 itemprop="name">Metabolic Profiles</h1>

    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-04-11 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Apr 11, 2018
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Computational Biology Lab">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    10 min read
  </span>
  

  
  

  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=Metabolic%20Profiles&amp;url=%2ftoo2%2fnew_meta%2fprofiles%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=%2ftoo2%2fnew_meta%2fprofiles%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-facebook"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=%2ftoo2%2fnew_meta%2fprofiles%2f&amp;title=Metabolic%20Profiles"
         target="_blank" rel="noopener">
        <i class="fa fa-linkedin"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=%2ftoo2%2fnew_meta%2fprofiles%2f&amp;title=Metabolic%20Profiles"
         target="_blank" rel="noopener">
        <i class="fa fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=Metabolic%20Profiles&amp;body=%2ftoo2%2fnew_meta%2fprofiles%2f">
        <i class="fa fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>


    <div class="article-style" itemprop="articleBody">
      

<hr />

<p><img src="/img/tools/meta/profile.gif"  width="500" height="400" alt="anno" align="center"></p>

<p><span id="top"></span></p>

<h2>Table of Contents</h2>
<nav id="TableOfContents">
<ul>
<li><a href="#metabolic-profiles-analysis">Metabolic profiles analysis</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-metabolic-network-analysis-font"><font color=#CD5555 face="黑体">Metabolic Network analysis</font></a>
<ul>
<li><a href="#blastx">BLASTX</a></li>
<li><a href="#pimp-polyomics-integrated-metabolomics-pipeline">PiMP | Polyomics integrated Metabolomics Pipeline</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-metabolite-enrichment-analysis-font"><font color=#CD5555 face="黑体">Metabolite enrichment analysis</font></a>
<ul>
<li><a href="#metscape-data-integration-mass-spectrometry-analysis">MetScape | Data integration : Mass spectrometry analysis</a></li>
<li><a href="#fella-metabolite-enrichment-analysis">FELLA | Metabolite enrichment analysis</a></li>
</ul></li>
</ul></li>
<li><a href="#ms-based-untargeted-metabolomics">MS-based untargeted Metabolomics</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-mgwas-font"><font color=#CD5555 face="黑体">mGWAS</font></a>
<ul>
<li><a href="#mwastools-mgwas">MWASTools | mGWAS</a></li>
<li><a href="#regscan">RegScan</a></li>
</ul></li>
</ul></li>
<li><a href="#nmr-based-metabolomics-ms-based-targeted-metabolomics">NMR-based Metabolomics/MS-based targeted metabolomics</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-spectral-corrections-font"><font color=#CD5555 face="黑体">Spectral corrections</font></a>
<ul>
<li><a href="#nmrpipe">NMRPipe</a></li>
<li><a href="#nmrpro">NMRPro</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-spectral-deconvolution-font"><font color=#CD5555 face="黑体">Spectral deconvolution</font></a>
<ul>
<li><a href="#nmrpipe-1">NMRPipe</a></li>
<li><a href="#batman-bayesian-automated-metabolite-analyser-for-nmr-data">BATMAN | Bayesian AuTomated Metabolite Analyser for NMR data</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-chemometric-analysis-font"><font color=#CD5555 face="黑体">Chemometric analysis</font></a>
<ul>
<li><a href="#metaboanalyst">MetaboAnalyst</a></li>
<li><a href="#speaq-spectrum-alignment-and-quantitation">speaq | spectrum alignment and quantitation</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-metabolite-identification-font"><font color=#CD5555 face="黑体">Metabolite identification</font></a>
<ul>
<li><a href="#w4m-workflow4metabolomics">W4M | Workflow4Metabolomics</a></li>
<li><a href="#metaboanalyst-1">MetaboAnalyst</a></li>
</ul></li>
</ul></li>
<li><a href="#find-more-tools-omictools-https-omictools-com-metabolomics-category">Find more tools: <a href="https://omictools.com/metabolomics-category" target="_blank"><strong>OMICTOOLS</strong></a></a></li>
</ul>
</nav>


<h1 id="metabolic-profiles-analysis">Metabolic profiles analysis</h1>

<p><p align="justify">The further development of spectroscopic and spectrometric tools for high throughput analyses of selected biochemical pathways is crucial to the acquisition of metabolome data sets of sufficient quality for metabonomics and metabolomics. Whilst metabolic fingerprinting or metabonomics assumes it is not necessary to determine levels of all individual metabolites for classification or response readouts, metabolic profiling or metabolomics absolutely requires the identification and quantification of as broad a class of metabolites as possible.</p>

<hr />

<h2 id="font-color-cd5555-face-黑体-metabolic-network-analysis-font"><font color=#CD5555 face="黑体">Metabolic Network analysis</font></h2>

<h3 id="blastx">BLASTX</h3>

<p><p align="justify"><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank"><strong>The Basic Local Alignment Search Tool</strong> (BLAST)</a> finds regions of   local similarity between sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance of matches. BLAST can be used to infer functional and evolutionary relationships between sequences as well as help identify members of gene families.A BLAST search enables a researcher to compare a query sequence with a library or database of sequences, and identify library sequences that resemble the query sequence above a certain threshold.</p>

<p><p align="justify">Different types of BLASTs are available according to the query sequences. For example, following the discovery of a previously unknown gene in the mouse, a scientist will typically perform a BLAST search of the human genome to see if humans carry a similar gene; BLAST will identify sequences in the human genome that resemble the mouse gene based on similarity of sequence. The BLAST algorithm and program were designed by <code>Stephen Altschul</code>, <code>Warren Gish</code>, <code>Webb Miller</code>, <code>Eugene Myers</code>, and <code>David J. Lipman</code> at the <a href="https://en.wikipedia.org/wiki/National_Institutes_of_Health" target="_blank">National Institutes of Health</a> and was published in the <a href="https://en.wikipedia.org/wiki/Journal_of_Molecular_Biology" target="_blank">Journal of Molecular Biology</a> in 1990 and cited over 50,000 times.</p>

<p><img src="/img/tools/Blast.png" width= 800 height= 500 alt="blast" align="center">
<strong>Input</strong>: Input sequences (in <a href="https://en.wikipedia.org/wiki/FASTA_format" target="_blank">FASTA</a> or Genbank format) and weight matrix.</p>

<p><strong>Output</strong>: BLAST output can be delivered in a variety of formats. These formats include HTML, plain text, and XML formatting.</p>

<p><p align="justify">The details about <strong>process</strong> and <strong>algorithm</strong> coule be found <a href="https://en.wikipedia.org/wiki/BLAST" target="_blank">here</a> and <a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi" target="_blank">official website</a></p>

<hr />

<h3 id="pimp-polyomics-integrated-metabolomics-pipeline">PiMP | Polyomics integrated Metabolomics Pipeline</h3>

<p><p align="justify">Allows users to analyze and visualize liquid chromatography – mass spectrometry (LC-MS) data. PiMP is a comprehensive and integrated web enabled pipeline that consists of five tasks: (1) project administration, (2) data upload, (3) quality control, (4) analysis parameters and (5) data interpretation. Users can define the experimental design, specify metadata and share the project with collaborators with a chosen level of permission. It aims at automatization and standardization of metabolomics analysis.</p>

<p><a href="http://polyomics.mvls.gla.ac.uk/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28961954" target="_blank">PiMP my metabolome: An integrated, web-based tool for LC-MS metabolomics data.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Glasgow Polyomics, University of Glasgow, Glasgow, Scotland;</p>

<p>School of Computing Science, University of Glasgow, Glasgow, Scotland</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-metabolite-enrichment-analysis-font"><font color=#CD5555 face="黑体">Metabolite enrichment analysis</font></h2>

<p>While many tools exist for performing enrichment analysis of transcriptomic and proteomic data in order to interpret them in biological terms, almost no equivalent tools exist for metabolomic data.</p>

<hr />

<h3 id="metscape-data-integration-mass-spectrometry-analysis">MetScape | Data integration : Mass spectrometry analysis</h3>

<p><p align="justify">Provides a bioinformatics framework for the visualization and interpretation of metabolomic and expression profiling data in the context of human metabolism. MetScape allows users to build and analyze networks of genes and compounds, identify enriched pathways from expression profiling data, and visualize changes in metabolite data. MetScape uses an internal relational database that integrates data from KEGG and EHMN.</p>

<p><a href="http://metscape.ncibi.org/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li><p>(2012 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22135418" target="_blank">Metscape 2 bioinformatics tool for the analysis and visualization of metabolomics and gene expression data.</a></p></li>

<li><p>(2010 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20139469" target="_blank">Metscape: a Cytoscape plug-in for visualizing and interpreting metabolomic data in the context of human metabolic networks.</a></p></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>National Center for Integrative Biomedical Informatics, University of Michigan, Ann Arbor, MI, USA</p>

<hr />

<h3 id="fella-metabolite-enrichment-analysis">FELLA | Metabolite enrichment analysis</h3>

<p><p align="justify">Enables enrichment of metabolomics data using KEGG reactions, enzymes, modules and pathways. FELLA is a metabolomics data enrichment tool that combines pathway enrichment with the flexibility of sub-network analysis. The method builds sub-pathway representations of the biology at several molecular levels, derived through a null diffusive process on a curated graph object built from the KEGG database.</p>

<p><a href="https://bioconductor.org/packages/devel/bioc/html/FELLA.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 PLoS One) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29211807" target="_blank">Null diffusion-based enrichment for metabolomics data.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Departament d&rsquo;Enginyeria de Sistemes, Automàtica i Informàtica Industrial, Universitat Politècnica de Catalunya, Barcelona, Spain</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="ms-based-untargeted-metabolomics">MS-based untargeted Metabolomics</h1>

<p><p align="justify">Metabolomics is an emerging field of study in post-genomics, which aims at comprehensive analysis of small organic molecules in biological systems (Patti et al., 2012). Techniques of mass spectrometry coupled to liquid chromatography (LC–MS) stand out as dominant methods in metabolomic experiments (Dettmer et al., 2007).</p>

<hr />

<h2 id="font-color-cd5555-face-黑体-mgwas-font"><font color=#CD5555 face="黑体">mGWAS</font></h2>

<p>Genome-wide association scans with high-throughput metabolic profiling provide unprecedented insights into how genetic variation influences metabolism and complex disease.</p>

<h3 id="mwastools-mgwas">MWASTools | mGWAS</h3>

<p><p align="justify">Allows users to visualize the results from metabolome-wide association studies (MWAS) analysis. MWASTools performs essential quality control (QC) analyses via Principal Component Analysis (PCA) and by computing the coefficients of variation (CV) of individual metabolic features. It is composed of four functional parts: (1) QC analysis; (2) MWAS analysis; (3) visualization of MWAS results; and (4) metabolite assignment using correlation analysis.</p>

<p><a href="https://bioconductor.org/packages/release/bioc/html/MWASTools.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Bioinformatics) <a href="https://doi.org/10.1093/bioinformatics/btx477" target="_blank">MWASTools: an R/Bioconductor package for metabolome-wide association studies.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Computational and Systems Medicine, Department of Surgery and Cancer, Imperial College London, UK;</p>

<p>Division of Myocardial Function, National Heart and Lung Institute, Imperial College London, UK</p>

<hr />

<h3 id="regscan">RegScan</h3>

<p><p align="justify">A command line tool for performing fast association analysis between allele frequencies and continuous traits. RegScan uses linear regression to estimate marker effects on continuous traits.</p>

<p><a href="https://www.geenivaramu.ee/en/tools/regscan" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2015 Brief Bioinform) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24008273" target="_blank">RegScan: a GWAS tool for quick estimation of allele effects on continuous traits and their combinations.</a></li>
</ul>

<p><strong>MAINTAINER</strong>:</p>

<p>Toomas Haller <a href="mailto:toomas.haller@ut.ee" target="_blank">toomas.haller@ut.ee</a></p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="nmr-based-metabolomics-ms-based-targeted-metabolomics">NMR-based Metabolomics/MS-based targeted metabolomics</h1>

<p><p align="justify">Elucidation of the chemical composition of biological samples is a main focus of systems biology and metabolomics. Because nuclear magnetic resonance (NMR) spectroscopy is a rich source of molecular information, it has a unique potential for this task.</p>

<p><p align="justify">Targeted metabolomics assay platforms are extremely useful for hypothesis testing. These experiments require extensive planning to define the key metabolites in biochemical pathways of interest. Rate-limiting precursors, end-product pools, critical intermediates, and potential alternate precursors or products need to be incorporated into an assay platform.</p>

<h2 id="font-color-cd5555-face-黑体-spectral-corrections-font"><font color=#CD5555 face="黑体">Spectral corrections</font></h2>

<p>The spectra processing step is crucial in metabolomics approaches, especially for proton NMR metabolomics profiling. During this step, noise reduction, baseline correction, peak alignment and reduction of the 1D (1)H-NMR spectral data are required in order to allow biological information to be highlighted through further statistical analyses. Above all, data reduction (binning or bucketing) strongly impacts subsequent statistical data analysis and potential biomarker discovery.</p>

<hr />

<h3 id="nmrpipe">NMRPipe</h3>

<p><p align="justify">An extensive software system for processing, analyzing, and exploiting NMR spectroscopic data.</p>

<p><a href="https://spin.niddk.nih.gov/NMRPipe/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(1995 J Biomol NMR) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8520220" target="_blank">NMRPipe: a multidimensional spectral processing system based on UNIX pipes.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Laboratory of Chemical Physics, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, MD, USA</p>

<hr />

<h3 id="nmrpro">NMRPro</h3>

<p><p align="justify">An extensible web component that can be easily integrated in current web applications and databases, providing NMR processing and visualization functionalities. NMRPro is highly extensible to include new functionalities according to the needs of each application. It integrates server-side processing with client-side interactive visualization through three parts: a python package to efficiently process large NMR datasets on the server-side, a Django App managing server-client interaction, and SpecdrawJS for client-side interactive visualization.</p>

<p><a href="https://github.com/phnmnl/container-nmrpro" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2016 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27153725" target="_blank">NMRPro: an integrated web component for interactive processing and visualization of NMR spectra.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Bioinformatics Center, Institute for Chemical Research, Kyoto University, Kyoto, Japan</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-spectral-deconvolution-font"><font color=#CD5555 face="黑体">Spectral deconvolution</font></h2>

<p>NMR spectroscopy has been an important tool for metabolomics although severe overlap of signals has limited the number of compounds, which can be unambiguously identified and quantified. Therefore, deconvolution of NMR spectra is one of the greatest challenges for NMR-based metabolomics.</p>

<hr />

<h3 id="nmrpipe-1">NMRPipe</h3>

<p><p align="justify">An extensive software system for processing, analyzing, and exploiting NMR spectroscopic data.</p>

<p><a href="https://spin.niddk.nih.gov/NMRPipe/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(1995 J Biomol NMR) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8520220" target="_blank">NMRPipe: a multidimensional spectral processing system based on UNIX pipes.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Laboratory of Chemical Physics, National Institute of Diabetes and Digestive and Kidney Diseases, National Institutes of Health, Bethesda, MD, USA</p>

<hr />

<h3 id="batman-bayesian-automated-metabolite-analyser-for-nmr-data">BATMAN | Bayesian AuTomated Metabolite Analyser for NMR data</h3>

<p><p align="justify">An R package for estimating metabolite concentrations from Nuclear Magnetic Resonance spectral data using a specialised MCMC algorithm. BATMAN deconvolutes peaks from 1-dimensional NMR spectra, automatically assigns them to specific metabolites from a target list and obtains concentration estimates. The Bayesian model incorporates information on characteristic peak patterns of metabolites and is able to account for shifts in the position of peaks commonly seen in NMR spectra of biological samples. It applies a Markov Chain Monte Carlo (MCMC) algorithm to sample from a joint posterior distribution of the model parameters and obtains concentration estimates with reduced error compared with conventional numerical integration and comparable to manual deconvolution by experienced spectroscopists.</p>

<p><a href="http://batman.r-forge.r-project.org/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li><p>(2014 Nat Protoc) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24853927" target="_blank">Bayesian deconvolution and quantification of metabolites in complex 1D NMR spectra using BATMAN.</a></p></li>

<li><p>(2012 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22635605" target="_blank">BATMAN&ndash;an R package for the automated quantification of metabolites from nuclear magnetic resonance spectra using a Bayesian model.</a></p></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Computational and Systems Medicine, Department of Surgery and Cancer, Imperial College London, London, UK;</p>

<p>Department of Statistical Science, University College London, London, UK</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-chemometric-analysis-font"><font color=#CD5555 face="黑体">Chemometric analysis</font></h2>

<p>Chemometric analysis programs for MS-based metabolomic analysis.</p>

<h3 id="metaboanalyst">MetaboAnalyst</h3>

<p><p align="justify">Provides a user-friendly, web-based analytical pipeline for high-throughput metabolomics studies. In particular, MetaboAnalyst aims to offer a variety of commonly used procedures for metabolomic data processing, normalization, multivariate statistical analysis, as well as data annotation. The current implementation focuses on exploratory statistical analysis, functional interpretation, and advanced statistics for translational metabolomics studies.</p>

<p><a href="http://www.metaboanalyst.ca/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li><p>(2018 Nucleic Acids Res) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29762782" target="_blank">MetaboAnalyst 4.0: towards more transparent and integrative metabolomics analysis.</a></p></li>

<li><p>(2011 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21712247" target="_blank">MetATT: a web-based metabolomics tool for analyzing time-series and two-factor datasets.</a></p></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Institute of Parasitology, and Department of Animal Science, McGill University, Ste Ann de Bellevue, Canada;</p>

<p>Department of Microbiology and Immunology, McGill University, Montreal, Canada</p>

<hr />

<h3 id="speaq-spectrum-alignment-and-quantitation">speaq | spectrum alignment and quantitation</h3>

<p><p align="justify">Allows Nuclear Magnetic Resonance (NMR) spectroscopy data analysis. Speaq enables raw spectra alignment and quantitation but also an analysis based on features whereby the spectra are converted to peaks which are then grouped and turned into features. The underlying core paradigm is to efficiently summarize spectra with little user interaction, high speed and most importantly little loss of information whilst greatly reducing the dimensions of the data. The package can be combined with existing tools to improve performance.</p>

<p><a href="https://cran.r-project.org/web/packages/speaq/index.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li><p>(2018 PLoS Comput Biol) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29494588" target="_blank">speaq 2.0: A complete workflow for high-throughput 1D NMR spectra processing and quantification.</a></p></li>

<li><p>(2011 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22014236" target="_blank">An integrated workflow for robust alignment and simplified quantitative analysis of NMR spectrometry data.</a></p></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Advanced Database Research and Modelling (ADReM), Department of Mathematics and Computer Science, University of Antwerp, Antwerp, Belgium</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-metabolite-identification-font"><font color=#CD5555 face="黑体">Metabolite identification</font></h2>

<p>Identification and quantification programs for NMR-based metabolomic analysis.</p>

<h3 id="w4m-workflow4metabolomics">W4M | Workflow4Metabolomics</h3>

<p><p align="justify">Permits comprehensive metabolomics data pre-processing, statistical analysis and interpretation. W4M includes computational modules for data normalization, multivariate analysis and annotation. It can create interactive web-based documents showing the results of the analyses, and users can share them with collaborators directly on the platform. This tool enables multi-omics analyses in a global systems-biology approach.</p>

<p><a href="http://workflow4metabolomics.org/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Int J Biochem Cell Biol) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28710041" target="_blank">Create, run, share, publish, and reference your LC-MS, FIA-MS, GC-MS, and NMR data analysis workflows with the Workflow4Metabolomics 3.0 Galaxy online infrastructure for metabolomics.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>INRA, UMR 1019, PFEM, Saint Genes Champanelle, France;</p>

<p>CNRS, UPMC, FR2424, ABiMS, Station Biologique, Roscoff, France; INRA, UMR 1331, PF MetaToul-AXIOM, Toxalim, Toulouse, France</p>

<hr />

<h3 id="metaboanalyst-1">MetaboAnalyst</h3>

<p><p align="justify">Provides a user-friendly, web-based analytical pipeline for high-throughput metabolomics studies. In particular, MetaboAnalyst aims to offer a variety of commonly used procedures for metabolomic data processing, normalization, multivariate statistical analysis, as well as data annotation. The current implementation focuses on exploratory statistical analysis, functional interpretation, and advanced statistics for translational metabolomics studies.</p>

<p><a href="http://www.metaboanalyst.ca/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li><p>(2018 Nucleic Acids Res) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29762782" target="_blank">MetaboAnalyst 4.0: towards more transparent and integrative metabolomics analysis.</a></p></li>

<li><p>(2011 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21712247" target="_blank">MetATT: a web-based metabolomics tool for analyzing time-series and two-factor datasets.</a></p></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Institute of Parasitology, and Department of Animal Science, McGill University, Ste Ann de Bellevue, Canada;</p>

<p>Department of Microbiology and Immunology, McGill University, Montreal, Canada</p>

<hr />

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="find-more-tools-omictools-https-omictools-com-metabolomics-category">Find more tools: <a href="https://omictools.com/metabolomics-category" target="_blank"><strong>OMICTOOLS</strong></a></h1>

<p><a href="https://www.aau.at/en/social-ecology/research/long-term-socio-ecological-research-and-environmental-history/" target="_blank"><em>Image Citation</em></a></p>

<hr />

    </div>

    


<div class="article-tags">
  
  <a class="btn btn-primary btn-outline" href="/tags/metabolomics" target="_blank">metabolomics</a>
  
</div>




    
    
    <div class="article-widget">
      <div class="hr-light"></div>
      <h3>Related</h3>
      <ul>
        
        <li><a href="/too2/new_meta/drug/">Drug Discovery</a></li>
        
        <li><a href="/too2/new_meta/lipidomics/">Lipidomics</a></li>
        
        <li><a href="/too2/new_meta/engineer/">Metabolic Engineering</a></li>
        
      </ul>
    </div>
    

    

    


  </div>
</article>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2018 &middot; 

      Copyright© 2018 Lin QI of Dr.Xin Wang's Lab. zzx. All rights reserved. 

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    <script async defer src="//maps.googleapis.com/maps/api/js?key=AIzaSyD3AGpKDAfo_LjBRHd-YkYmMjBDlIHfdlo"></script>
    

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gmaps.js/0.4.25/gmaps.min.js" integrity="sha256-7vjlAeb8OaTrCXZkCNun9djzuB2owUsaO72kXaFDBJs=" crossorigin="anonymous"></script>
    
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
      

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

