<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.38.2" />
  <meta name="author" content="Computational Biology Lab">

  
  
  
  
    
      
    
  
  <meta name="description" content="Table of Contents  Protein Sequence Analysis  Amino Acid Repeat Prediction  RADAR REPRO  Subcellular Localizaton Prediction  TargetP pRoloc  Protein Clustering  CD-HIT USEARCH   MS-based (Mass Spectrometry) untargeted Proteomics  2D gel Image Analysis  digeR Plasmo2D  In Silico Analysis  OpenMS multiplierz  Phosphorylation Site Assignment  Ascore Phosphonormalizer  PTM Identification  PIQED SUMmOn  Glycosylation Identification  SugarQb MAGIC   MS-based Metaproteomics    Unipept MPA   Antibody Array Analysis    PANDA   Find more tools: OMICTOOLS   Protein Sequence Analysis Protein sequence analysis tools are used to predict specific functions, activities, origin, or localization of proteins based on their amino-acid sequence.">

  
  <link rel="alternate" hreflang="en-us" href="/too2/new_proteo/pro_seq/">

  


  

  
  
  <meta name="theme-color" content="#3f51b5">
  
  
  
  
    
  
  
    
    
      
        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
      
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">
  
  
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7cMerriweather%7cRoboto&#43;Mono">
  
  <link rel="stylesheet" href="/styles.css">
  

  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/too2/new_proteo/pro_seq/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="og:site_name" content="CityU-Bioinformatics">
  <meta property="og:url" content="/too2/new_proteo/pro_seq/">
  <meta property="og:title" content="Protein Sequence/Expression | CityU-Bioinformatics">
  <meta property="og:description" content="Table of Contents  Protein Sequence Analysis  Amino Acid Repeat Prediction  RADAR REPRO  Subcellular Localizaton Prediction  TargetP pRoloc  Protein Clustering  CD-HIT USEARCH   MS-based (Mass Spectrometry) untargeted Proteomics  2D gel Image Analysis  digeR Plasmo2D  In Silico Analysis  OpenMS multiplierz  Phosphorylation Site Assignment  Ascore Phosphonormalizer  PTM Identification  PIQED SUMmOn  Glycosylation Identification  SugarQb MAGIC   MS-based Metaproteomics    Unipept MPA   Antibody Array Analysis    PANDA   Find more tools: OMICTOOLS   Protein Sequence Analysis Protein sequence analysis tools are used to predict specific functions, activities, origin, or localization of proteins based on their amino-acid sequence.">
  <meta property="og:locale" content="en-us">
  
  <meta property="article:published_time" content="2018-04-11T00:00:00&#43;00:00">
  
  <meta property="article:modified_time" content="2018-04-11T00:00:00&#43;00:00">
  

  

  <title>Protein Sequence/Expression | CityU-Bioinformatics</title>

</head>
<body id="top" data-spy="scroll" data-target="#toc" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/">CityU-Bioinformatics</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#bioinformatics">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Programming</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/program/r">
                
                <span>R Language</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Tutorials</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/tutorial/getstart/">
                
                <span>Getting Started With R</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/bms4001/">
                
                <span>BMS4001 tutorials</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/courses/">
                
                <span>Online Courses</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/resources/">
                
                <span>Other Resources</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Database</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/base2/dna">
                
                <span>DNA Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/express">
                
                <span>Genome Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/rna">
                
                <span>RNA Databses</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/protein">
                
                <span>Protein Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pheno">
                
                <span>Phenotype Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pathway">
                
                <span>Pathway Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/special">
                
                <span>Other Databases</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Tools</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/too2/genomics">
                
                <span>Genomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/epigenomics">
                
                <span>Epigenomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/transcriptomics">
                
                <span>Transcriptomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/proteomics">
                
                <span>Proteomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/metabolomics">
                
                <span>Metabolomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/phenomics">
                
                <span>Phenomics</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Journals</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/journal/comprehensive">
                
                <span>Multidisciplinary</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/specific">
                
                <span>Bioinformatics Specific</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/conferences">
                
                <span>Conferences</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/server/">
            
            <span>Servers</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>


<article class="article" itemscope itemtype="http://schema.org/Article">

  


  <div class="article-container">
    <h1 itemprop="name">Protein Sequence/Expression</h1>

    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-04-11 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Apr 11, 2018
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Computational Biology Lab">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    11 min read
  </span>
  

  
  

  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=Protein%20Sequence%2fExpression&amp;url=%2ftoo2%2fnew_proteo%2fpro_seq%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=%2ftoo2%2fnew_proteo%2fpro_seq%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-facebook"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=%2ftoo2%2fnew_proteo%2fpro_seq%2f&amp;title=Protein%20Sequence%2fExpression"
         target="_blank" rel="noopener">
        <i class="fa fa-linkedin"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=%2ftoo2%2fnew_proteo%2fpro_seq%2f&amp;title=Protein%20Sequence%2fExpression"
         target="_blank" rel="noopener">
        <i class="fa fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=Protein%20Sequence%2fExpression&amp;body=%2ftoo2%2fnew_proteo%2fpro_seq%2f">
        <i class="fa fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>


    <div class="article-style" itemprop="articleBody">
      

<hr />

<p><img src="/img/tools/protein/pro_seq.png"  width="500" height="400" alt="anno" align="center"></p>

<p><span id="top"></span></p>

<h2>Table of Contents</h2>
<nav id="TableOfContents">
<ul>
<li><a href="#protein-sequence-analysis">Protein Sequence Analysis</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-amino-acid-repeat-prediction-font"><font color=#CD5555 face="黑体">Amino Acid Repeat Prediction</font></a>
<ul>
<li><a href="#radar">RADAR</a></li>
<li><a href="#repro">REPRO</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-subcellular-localizaton-prediction-font"><font color=#CD5555 face="黑体">Subcellular Localizaton Prediction</font></a>
<ul>
<li><a href="#targetp">TargetP</a></li>
<li><a href="#proloc">pRoloc</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-protein-clustering-font"><font color=#CD5555 face="黑体">Protein Clustering</font></a>
<ul>
<li><a href="#cd-hit">CD-HIT</a></li>
<li><a href="#usearch">USEARCH</a></li>
</ul></li>
</ul></li>
<li><a href="#ms-based-mass-spectrometry-untargeted-proteomics">MS-based (Mass Spectrometry) untargeted Proteomics</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-2d-gel-image-analysis-font"><font color=#CD5555 face="黑体">2D gel Image Analysis</font></a>
<ul>
<li><a href="#diger">digeR</a></li>
<li><a href="#plasmo2d">Plasmo2D</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-in-silico-analysis-font"><font color=#CD5555 face="黑体">In Silico Analysis</font></a>
<ul>
<li><a href="#openms">OpenMS</a></li>
<li><a href="#multiplierz">multiplierz</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-phosphorylation-site-assignment-font"><font color=#CD5555 face="黑体">Phosphorylation Site Assignment</font></a>
<ul>
<li><a href="#ascore">Ascore</a></li>
<li><a href="#phosphonormalizer">Phosphonormalizer</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-ptm-identification-font"><font color=#CD5555 face="黑体">PTM Identification</font></a>
<ul>
<li><a href="#piqed">PIQED</a></li>
<li><a href="#summon">SUMmOn</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-glycosylation-identification-font"><font color=#CD5555 face="黑体">Glycosylation Identification</font></a>
<ul>
<li><a href="#sugarqb">SugarQb</a></li>
<li><a href="#magic">MAGIC</a></li>
</ul></li>
</ul></li>
<li><a href="#ms-based-metaproteomics">MS-based Metaproteomics</a>
<ul>
<li>
<ul>
<li><a href="#unipept">Unipept</a></li>
<li><a href="#mpa">MPA</a></li>
</ul></li>
</ul></li>
<li><a href="#antibody-array-analysis">Antibody Array Analysis</a>
<ul>
<li>
<ul>
<li><a href="#panda">PANDA</a></li>
</ul></li>
</ul></li>
<li><a href="#find-more-tools-omictools-https-omictools-com-proteomics-category">Find more tools: <a href="https://omictools.com/proteomics-category" target="_blank"><strong>OMICTOOLS</strong></a></a></li>
</ul>
</nav>


<h1 id="protein-sequence-analysis">Protein Sequence Analysis</h1>

<p><p align="justify">Protein sequence analysis tools are used to predict specific functions, activities, origin, or localization of proteins based on their amino-acid sequence. Software tools are also used to analysis high-throughput proteomics data sequences obtained by mass-spectrometry.</p>

<h2 id="font-color-cd5555-face-黑体-amino-acid-repeat-prediction-font"><font color=#CD5555 face="黑体">Amino Acid Repeat Prediction</font></h2>

<h3 id="radar">RADAR</h3>

<p><p align="justify">A tool to identify short composition biased and gapped approximate repeats, as well as complex repeat architectures involving many different types of repeats in a query sequence. RADAR is useful because many large proteins have evolved by internal duplication and many internal sequence repeats correspond to functional and structural units.</p>

<p><a href="https://www.ebi.ac.uk/Tools/pfa/radar/" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="http://www.ebi.ac.uk/Tools/pfa/radar/help/" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Li et al., 2015) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25845596" target="_blank">The EMBL-EBI bioinformatics web and programmatic tools framework</a>.  Nucleic Acids Res.</li>
<li>(Heger and Holm, 2000) <a href="https://www.ncbi.nlm.nih.gov/pubmed/10966575" target="_blank">Rapid automatic detection and alignment of repeats in protein sequences</a>.  Proteins.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK</p>

<hr />

<h3 id="repro">REPRO</h3>

<p><p align="justify">The program is able to recognize distant repeats in a single query sequence. The technique relies on a variation of the Smith-Waterman local alignment strategy to find non-overlapping top-scoring local alignments, followed by a graph-based iterative clustering procedure to delineate the repeat set(s) based on consistency of the pairwise top-alignments.</p>

<p><a href="http://www.ibi.vu.nl/programs/reprowww/" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="http://www.ibi.vu.nl/programs/reprowww/info.php" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(George and Heringa, 2000) <a href="https://www.ncbi.nlm.nih.gov/pubmed/11203383" target="_blank">The REPRO server: finding protein internal sequence repeats through the Web</a>.  Trends Biochem Sci.</li>
<li>(Heringa and Argos, 1993) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8108381" target="_blank">A method to recognize distant repeats in protein sequences</a>.  Proteins.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>European Molecular Biology Laboratory, Heidelberg, Germany</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-subcellular-localizaton-prediction-font"><font color=#CD5555 face="黑体">Subcellular Localizaton Prediction</font></h2>

<h3 id="targetp">TargetP</h3>

<p><p align="justify">Allows users to predict eukaryotic proteins location. TargetP is a web application that scores N-terminal pre-sequences in a submitted protein. The software indicates chloroplast transit peptide (cTP), mitochondrial targeting peptide (mTP) and secretory pathway signal peptide (SP) predicted localization. The application includes parameters which allow choosing between in Plants and Non-Plants version, personalized cutoffs and the possibility to determine cleavage sites.</p>

<p><a href="http://www.cbs.dtu.dk/services/TargetP/" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="http://www.ibi.vu.nl/programs/reprowww/info.php" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Emanuelsson et al., 2007) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17446895" target="_blank">Locating proteins in the cell using TargetP, SignalP and related tools</a>.  Nat Protoc.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Stockholm Bioinformatics Center, Albanova, Stockholm University, Stockholm, Sweden; Center for Biological Sequence Analysis, Technical University of Denmark, Lyngby, Denmark</p>

<hr />

<h3 id="proloc">pRoloc</h3>

<p><p align="justify">Provides machine learning and visualization methods for interrogating and analyzing on quantitative mass spectrometry (MS) data to infer protein sub-cellular localization. PRoloc is suited for spatial proteomics data analysis provided as an R package that performs sub-cellular localization prediction from experimental and condition-specific MS-based quantitative proteomics data. The software allows classification of proteins to tens of sub-cellular compartments.</p>

<p><a href="http://bioconductor.org/packages/release/bioc/html/pRoloc.html" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="http://bioconductor.org/packages/release/bioc/manuals/pRoloc/man/pRoloc.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Breckels et al., 2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27175778" target="_blank">Learning from Heterogeneous Data Sources: An Application in Spatial Proteomics</a>.  PLoS Comput Biol.</li>
<li>(Crook et al., 2018) <a href="https://www.biorxiv.org/content/early/2018/05/22/282269" target="_blank">A Bayesian Mixture Modelling Approach For Spatial Proteomics</a>.  BioRxiv.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Computational Proteomics Unit, Department of Biochemistry, University of Cambridge, Cambridge, UK</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-protein-clustering-font"><font color=#CD5555 face="黑体">Protein Clustering</font></h2>

<h3 id="cd-hit">CD-HIT</h3>

<p><p align="justify">A widely used program for clustering biological sequences to reduce sequence redundancy and improve the performance of other sequence analyses. In response to the rapid increase in the amount of sequencing data produced by the next-generation sequencing technologies, a new CD-HIT program accelerated with a novel parallelization strategy and some other techniques has been developed to allow efficient clustering of such datasets.</p>

<p><a href="http://weizhong-lab.ucsd.edu/cd-hit/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Fu et al., 2012) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23060610" target="_blank">CD-HIT: accelerated for clustering the next-generation sequencing data</a>.  Bioinformatics.</li>
<li>(Li and Godzik, 2006) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16731699" target="_blank">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences</a>. Bioinformatics.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Center for Research in Biological Systems, University of California San Diego, La Jolla, CA, USA</p>

<hr />

<h3 id="usearch">USEARCH</h3>

<p><p align="justify">Searches and clusters algorithms that can be orders of magnitude. USEARCH is a sequence analysis software which combines different algorithms into a single package. This software searches in database for top global hits and provides several NGS read processing features such as dereplication, paired read overlapping, quality filtering, FASTQ file statistics or chimeric sequence filtering.</p>

<p><a href="http://www.drive5.com/usearch/" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://toolshed.g2.bx.psu.edu/repository?repository_id=13df57e58dc7e075" target="_blank"><strong>Galaxy</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Edgar, 2010) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20709691" target="_blank">Search and clustering orders of magnitude faster than BLAST</a>.  Bioinformatics.</li>
<li>(Edgar and Flyvbjerg, 2015) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26139637" target="_blank">Error filtering, pair assembly and error correction for next-generation sequencing reads</a>.  Bioinformatics.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Micro- and Nanotechnology, Technical University of Denmark, Lyngby, Denmark</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="ms-based-mass-spectrometry-untargeted-proteomics">MS-based (Mass Spectrometry) untargeted Proteomics</h1>

<p>A number of technologies can be used to study proteomes, but arguably none is more powerful than mass spectrometry. There are two fundamentally different MS-based strategies for analyzing proteomes: discovery-based identification and targeted quantification (<a href="https://www.ncbi.nlm.nih.gov/pubmed/23547294" target="_blank">Doerr, 2013</a>; <a href="https://www.ncbi.nlm.nih.gov/pubmed/22669653" target="_blank">Picotti and Aebersold, 2012</a>). With a discovery-based strategy, the goal is usually to identify as many proteins as possible. The goal of a targeted proteomics experiment is to monitor a select few proteins of interest with high sensitivity, reproducibility and quantitative accuracy. <a href="https://en.wikipedia.org/wiki/List_of_mass_spectrometry_software" target="_blank">Wikipedia</a></p>

<h2 id="font-color-cd5555-face-黑体-2d-gel-image-analysis-font"><font color=#CD5555 face="黑体">2D gel Image Analysis</font></h2>

<h3 id="diger">digeR</h3>

<p><p align="justify">Provides a graphical user interface (GUI) for the visualization of protein post-translational modification (PTM) changes between different biological states. digeR is an R package for analyzing potential protein PTM changes in 2D-DIGE (2D gel) study using spots correlation. It can also be used to support other ‘omic’ data analysis in a similar manner and to assist biologist to look for panels of biomarkers which would improve the diagnosis and prognosis of the disease.</p>

<p><a href="https://cran.r-project.org/src/contrib/Archive/digeR/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Fan et al., 2009) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19706743" target="_blank">digeR: a graphical user interface R package for analyzing 2D-DIGE data</a>.  Bioinformatics.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>UCD School of Medicine and Medical Science, University College Dublin, Dublin, Ireland; UCD Conway Institute of Biomolecular and Biomolecular Research, University College Dublin, Dublin, Ireland</p>

<hr />

<h3 id="plasmo2d">Plasmo2D</h3>

<p><p align="justify">Performs gel alignment and protein identification computationally. Plasmo2D assists in the identification of proteins in the P. falciparum 3D7 proteome from the 2-DE gel image. It allows users to overlap the markers of the gel image on the marker provided by the software using the image resizes buttons in the software. This too can display a list of potential candidate proteins corresponding to a spot of interest.</p>

<p><a href="http://utlab3.biochem.iisc.ernet.in/Plasmo2D/index.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Khachane et al., 2005) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16335988" target="_blank">&ldquo;Plasmo2D&rdquo;: an ancillary proteomic tool to aid identification of proteins from Plasmodium falciparum</a>.  J Proteome Res.<br /></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Biochemistry, Indian Institute of Science, Bangalore, India</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-in-silico-analysis-font"><font color=#CD5555 face="黑体">In Silico Analysis</font></h2>

<h3 id="openms">OpenMS</h3>

<p><p align="justify">Allows to manage and analyse Liquid chromatography coupled to mass spectrometry (LC-MS) data. OpenMS is a programming library and tool collection integrated into full-featured workflow systems, such as KNIME, Galaxy and WS-PGRADE, to facilitate bioinformatics research in the field of MS on all levels. The software provides pre-built and ready-to-use tools for analysis of both proteomics and non-targeted metabolomics data.</p>

<p><a href="https://www.openms.de/" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://github.com/OpenMS/OpenMS" target="_blank"><strong>Github</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Lange et al., 2007) <a href="https://www.ncbi.nlm.nih.gov/pubmed/17646306" target="_blank">A geometric approach for the alignment of liquid chromatography-mass spectrometry data</a>.  Bioinformatics.</li>
<li>(Pfeuffer et al., 2017) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28559010" target="_blank">OpenMS - A platform for reproducible analysis of mass spectrometry data</a>.  J Biotechnol.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Applied Bioinformatics, Department for Computer Science, University of Tuebingen, Tuebingen, Germany; Center for Bioinformatics, University of Tuebingen, Tuebingen, Germany</p>

<hr />

<h3 id="multiplierz">multiplierz</h3>

<p><p align="justify">Manages proteomic mass spectrometry workflows and data analysis. Multiplierz provides a toolset of multiple methods for peptide identification, quantitation, reporting, as well as tools for easily manipulating standard data formats. This software is a Python library compatible with new reporting formats and high-level tools to achieve post-perform proteomic analyses. The architecture of the software environment has seamless integration with native data files via mzAPI.</p>

<p><a href="https://github.com/BlaisProteomics/multiplierz" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://github.com/MaxAlex/multiplierz/wiki" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Parikh et al., 2009) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19874609" target="_blank">multiplierz: an extensible API based desktop environment for proteomics data analysis</a>.  BMC Bioinformatics.</li>
<li>(Alexander et al., 2017) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28686798" target="_blank">multiplierz v2.0: A Python-based ecosystem for shared access and analysis of native mass spectrometry data</a>.  Proteomics.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Cancer Biology and Blais Proteomics Center, Dana- Farber Cancer Institute, Boston, MA, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-phosphorylation-site-assignment-font"><font color=#CD5555 face="黑体">Phosphorylation Site Assignment</font></h2>

<h3 id="ascore">Ascore</h3>

<p><p align="justify">Processes localization-specific probability for every phosphorylation site within a data set. Ascore allows users to detect peptides, shows the degree of certainty and the non-localized peptides. This software can determine the proper phosphorylation site alignment by calculating the differences in site placement at the level of the site-determining ions. It avoids the potential inconsistencies that may happen from manual validation.</p>

<p><a href="http://ascore.med.harvard.edu/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Beausoleil et al., 2006) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16964243" target="_blank">A probability-based approach for high-throughput protein phosphorylation analysis and site localization</a>.  Nat Biotechnol.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Cell Biology, Harvard Medical School, Boston, MA, USA; Department of Genetics and Norris Cotton Cancer Center, Lebanon, New Hampshire, NH, USA</p>

<hr />

<h3 id="phosphonormalizer">Phosphonormalizer</h3>

<p><p align="justify">Provides pairwise normalization of phosphoproteomics data. Phosphonormalizer allows analysis of large-scale experiments in various treatment conditions. It incorporates non-enriched data as a reference for normalizing the enriched data. This tool can be applied to studies of other post-translational modifications (PTMs) than phosphorylation that are commonly studied using enrichment and that can also be detected in non-enriched samples with sensitive methods.</p>

<p><a href="https://bioconductor.org/packages/release/bioc/html/phosphonormalizer.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Saraei et al., 2017) <a href="https://doi.org/10.1093/bioinformatics/btx573" target="_blank">Phosphonormalizer: an R package for normalization of MS-based label-free phosphoproteomics</a>.  Bioinformatics.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Turku Centre for Biotechnology, University of Turku and Åbo Akademi, Turku, Finland; Department of Future Technologies, University of Turku, Turku, Finland</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-ptm-identification-font"><font color=#CD5555 face="黑体">PTM Identification</font></h2>

<h3 id="piqed">PIQED</h3>

<p><p align="justify">Provides a complete, automated workflow for post translational modification (PTM) identification, quantification, and statistical testing from exclusively data-independent acquisition-mass spectrometry (DIA-MS) data. PIQED is a workflow and open-source software that enables a two-fold reduction of acquisition time because both identification and quantification are achieved with a single DIA analysis. PTM-specific capabilities of this package include site localization scoring and filtering, peptide consolidation to modification site-level, and optional local or global total-ion chromatogram (TIC) normalization.</p>

<p><a href="https://github.com/jgmeyerucsd/PIQEDia/tree/v0.1.2" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://github.com/jgmeyerucsd/PIQEDia/raw/master/Tutorial_and_manual.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Meyer et al., 2017) <a href="https://doi.org/10.1101/141382" target="_blank">DIA-Pipe: Identification and Quantification of Post-Translational Modifications using exclusively Data-Independent Acquisition</a>. BioRxiv.</li>
<li>(Meyer et al., 2017) <a href="https://www.nature.com/articles/nmeth.4334" target="_blank">PIQED: automated identification and quantification of protein modifications from DIA-MS data</a>.  Nat Methods.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Buck Institute for Research on Aging, Novato, CA, USA; Department of Pathology, Boston Children’s Hospital and Harvard Medical School, Boston, MA, USA</p>

<hr />

<h3 id="summon">SUMmOn</h3>

<p><p align="justify">An automated pattern recognition tool that detects diagnostic PTM fragment ion series within complex MS/MS spectra, to identify modified peptides and modification sites within these peptides. The results of the SUMmOn analysis are stored in an XML instance document. A dynamically generated XSLT style sheet is then used to create an HTML file that is formatted via CSS (cascading style sheet), and viewed in a web-browser.</p>

<p><a href="http://summon.sourceforge.net/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Pedrioli et al., 2006) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16791211" target="_blank">Automated identification of SUMOylation sites using mass spectrometry and SUMmOn pattern recognition software</a>.  Nat Methods.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Institute for Systems Biology, Seattle, WA, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-glycosylation-identification-font"><font color=#CD5555 face="黑体">Glycosylation Identification</font></h2>

<h3 id="sugarqb">SugarQb</h3>

<p><p align="justify">Enables genome-wide insights into protein glycosylation and glycan modifications in complex biological systems. SugarQb identifies intact glycopeptides, and maps and quantifies changes in protein glycosylation at a proteome scale. This method can assist user in investigation of the glycoproteome of mouse embryonic stem cells.</p>

<p><a href="http://ms.imp.ac.at/?action=sugarqb" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="http://ms.imp.ac.at/data/sugarqb/SugarQb_HowTo_2017-09-20.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Stadlmann et al., 2017) <a href="https://www.nature.com/articles/nature24015" target="_blank">Comparative glycoproteomics of stem cells identifies new players in ricin toxicity</a>.  Nature.<br /></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>IMBA, Institute of Molecular Biotechnology of the Austrian Academy of Sciences, Vienna, Austria; Institute of Molecular Pathology (IMP), Vienna, Austria</p>

<hr />

<h3 id="magic">MAGIC</h3>

<p><p align="justify">Identifies intact N-glycosylated peptides from a public protein database without requiring any prior information of proteins or glycans. MAGIC aims to support untargeted glycopeptide analysis while the newly implemented MAGIC+ is designed to perform targeted glycopeptide analysis that allows users to upload their own protein sequence file to find glycopeptides in their data. The search results from Mascot can be integrated with the results from MAGIC-web via Reports Integrator to generate a complete protein/peptide-glycan summary report. Independent of the above three modules, the fourth module, Glycan Search, allows users to find various glycans from a large glycan database stored in the web server, regardless the types of glycosylation. MAGIC-web has a user-friendly visualization interface for easy data uploading and processing, and result interpretation. MAGIC-web is free and open to all users and there is no log in requirement.</p>

<p><a href="http://magic.iis.sinica.edu.tw/index.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Lih et al., 2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27084943" target="_blank">MAGIC-web: a platform for untargeted and targeted N-linked glycoprotein identification.</a>  Nucleic Acids Res.</li>
<li>(Lynn et al., 2015) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25629585" target="_blank">MAGIC: an automated N-linked glycoprotein identification tool using a Y1-ion pattern matching algorithm and in silico MS² approach</a>. Anal Chem.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Bioinformatics Program, Taiwan International Graduate Program, Academia Sinica, Taipei, Taiwan; Institute of Information Science, Academia Sinica, Taipei, Taiwan</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="ms-based-metaproteomics">MS-based Metaproteomics</h1>

<h3 id="unipept">Unipept</h3>

<p><p align="justify">An open source web application that is designed for metaproteomics analysis with a focus on interactive data-visualization. Unipept is underpinned by a fast index built from UniProtKB and the NCBI taxonomy that enables quick retrieval of all UniProt entries in which a given tryptic peptide occurs. Unipept provides programmatic access to the metaproteomics analysis features. This enables integration of Unipept functionality in custom applications and data processing pipelines.</p>

<p><a href="http://magic.iis.sinica.edu.tw/index.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Mesuere et al., 2015) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25477242" target="_blank">The Unipept metaproteomics analysis pipeline</a>.  Proteomics.</li>
<li>(Mesuere et al., 2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26819472" target="_blank">Unipept web services for metaproteomics analysis</a>.  Bioinformatics.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Applied Mathematics, Computer Science and Statistics, Faculty of Sciences, Ghent University, Ghent, Belgium</p>

<hr />

<h3 id="mpa">MPA</h3>

<p><p align="justify">An intuitive open-source tool for metaproteomics data analysis and interpretation, which includes multiple search engines and the feature to decrease data redundancy by grouping protein hits to so-called meta-proteins. The functionality of the MetaProteomeAnalyzer is demonstrated using a sample of a microbial community taken from a biogas plant.</p>

<p><a href="https://github.com/compomics/meta-proteome-analyzer" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://github.com/compomics/meta-proteome-analyzer/wiki" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Muth et al., 2015) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25660940" target="_blank">The MetaProteomeAnalyzer: a powerful open-source software suite for metaproteomics data analysis and interpretation</a>.  J Proteome Res.<br /></li>
<li>(Muth et al., 2017) <a href="https://www.ncbi.nlm.nih.gov/pubmed/29215871" target="_blank">MPA Portable: a Stand-alone Software Package for Analyzing Metaproteome Samples on the Go</a>.  Anal Chem.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Max Planck Institute for Dynamics of Complex Technical Systems, Magdeburg, Germany</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="antibody-array-analysis">Antibody Array Analysis</h1>

<h3 id="panda">PANDA</h3>

<p><p align="justify">A web-based software program developed at Emory University for analyzing phosphorylation antibody arrays.</p>

<p><a href="http://www.mathcs.emory.edu/panda/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Zhong et al., 2008) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18794113" target="_blank">LKB1 is necessary for Akt-mediated phosphorylation of proapoptotic proteins</a>.  Cancer Res.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>The Winship Cancer Institute, Emory University School of Medicine, Atlanta, Georgia, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="find-more-tools-omictools-https-omictools-com-proteomics-category">Find more tools: <a href="https://omictools.com/proteomics-category" target="_blank"><strong>OMICTOOLS</strong></a></h1>

<p><a href="https://www.ebi.ac.uk/training/online/course/introduction-protein-classification-ebi/protein-classification" target="_blank"><em>Image Citation</em></a></p>

<hr />

    </div>

    


<div class="article-tags">
  
  <a class="btn btn-primary btn-outline" href="/tags/proteomics" target="_blank">proteomics</a>
  
</div>




    
    
    <div class="article-widget">
      <div class="hr-light"></div>
      <h3>Related</h3>
      <ul>
        
        <li><a href="/too2/new_proteo/structure/">Prediction of Protein Structure</a></li>
        
        <li><a href="/too2/new_proteo/interactions/">Protein Interactions</a></li>
        
        <li><a href="/too2/new_proteo/membrane/">Membrane Protein</a></li>
        
        <li><a href="/too2/new_proteo/immune/">Immune System</a></li>
        
      </ul>
    </div>
    

    

    


  </div>
</article>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2018 &middot; 

      Copyright© 2018 Lin QI of Dr.Xin Wang's Lab. zzx. All rights reserved. 

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    <script async defer src="//maps.googleapis.com/maps/api/js?key=AIzaSyD3AGpKDAfo_LjBRHd-YkYmMjBDlIHfdlo"></script>
    

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gmaps.js/0.4.25/gmaps.min.js" integrity="sha256-7vjlAeb8OaTrCXZkCNun9djzuB2owUsaO72kXaFDBJs=" crossorigin="anonymous"></script>
    
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
      

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

