<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.38.2" />
  <meta name="author" content="Computational Biology Lab">

  
  
  
  
    
      
    
  
  <meta name="description" content="Table of Contents  Immune System Analysis  Sequence-based B-cell Epitope Prediction  ABCpred BepiPred  Structure-based B-cell Epitope Prediction  LYRA  MHC Class I Epitope Prediction  ProPred1 GradDock  MHC Class II Epitope Prediction  HLAMatchmaker ProPred   Flow Cytometry  Cell Population Identification  FCS Express FlowJo   Mass Cytometry  SPADE (Spanning tree Progression of Density normalized Events viSNE | Data visualization : Flow cytometry  Find more tools: OMICTOOLS   Immune System Analysis Immunoinformatics involves the application of computational methods to immunological problems.">

  
  <link rel="alternate" hreflang="en-us" href="/too2/new_proteo/immune/">

  


  

  
  
  <meta name="theme-color" content="#3f51b5">
  
  
  
  
    
  
  
    
    
      
        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
      
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">
  
  
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7cMerriweather%7cRoboto&#43;Mono">
  
  <link rel="stylesheet" href="/styles.css">
  

  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/too2/new_proteo/immune/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="og:site_name" content="CityU-Bioinformatics">
  <meta property="og:url" content="/too2/new_proteo/immune/">
  <meta property="og:title" content="Immune System | CityU-Bioinformatics">
  <meta property="og:description" content="Table of Contents  Immune System Analysis  Sequence-based B-cell Epitope Prediction  ABCpred BepiPred  Structure-based B-cell Epitope Prediction  LYRA  MHC Class I Epitope Prediction  ProPred1 GradDock  MHC Class II Epitope Prediction  HLAMatchmaker ProPred   Flow Cytometry  Cell Population Identification  FCS Express FlowJo   Mass Cytometry  SPADE (Spanning tree Progression of Density normalized Events viSNE | Data visualization : Flow cytometry  Find more tools: OMICTOOLS   Immune System Analysis Immunoinformatics involves the application of computational methods to immunological problems.">
  <meta property="og:locale" content="en-us">
  
  <meta property="article:published_time" content="2018-04-11T00:00:00&#43;00:00">
  
  <meta property="article:modified_time" content="2018-04-11T00:00:00&#43;00:00">
  

  

  <title>Immune System | CityU-Bioinformatics</title>

</head>
<body id="top" data-spy="scroll" data-target="#toc" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/">CityU-Bioinformatics</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#bioinformatics">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Programming</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/program/r">
                
                <span>R Language</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Tutorials</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/tutorial/getstart/">
                
                <span>Getting Started With R</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/bms4001/">
                
                <span>BMS4001 tutorials</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/courses/">
                
                <span>Online Courses</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/resources/">
                
                <span>Other Resources</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Database</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/base2/dna">
                
                <span>DNA Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/express">
                
                <span>Genome Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/rna">
                
                <span>RNA Databses</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/protein">
                
                <span>Protein Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pheno">
                
                <span>Phenotype Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pathway">
                
                <span>Pathway Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/special">
                
                <span>Other Databases</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Tools</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/too2/genomics">
                
                <span>Genomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/epigenomics">
                
                <span>Epigenomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/transcriptomics">
                
                <span>Transcriptomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/proteomics">
                
                <span>Proteomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/metabolomics">
                
                <span>Metabolomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/phenomics">
                
                <span>Phenomics</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Journals</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/journal/comprehensive">
                
                <span>Multidisciplinary</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/specific">
                
                <span>Bioinformatics Specific</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/conferences">
                
                <span>Conferences</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/server/">
            
            <span>Servers</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>


<article class="article" itemscope itemtype="http://schema.org/Article">

  


  <div class="article-container">
    <h1 itemprop="name">Immune System</h1>

    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-04-11 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Apr 11, 2018
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Computational Biology Lab">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    9 min read
  </span>
  

  
  

  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=Immune%20System&amp;url=%2ftoo2%2fnew_proteo%2fimmune%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=%2ftoo2%2fnew_proteo%2fimmune%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-facebook"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=%2ftoo2%2fnew_proteo%2fimmune%2f&amp;title=Immune%20System"
         target="_blank" rel="noopener">
        <i class="fa fa-linkedin"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=%2ftoo2%2fnew_proteo%2fimmune%2f&amp;title=Immune%20System"
         target="_blank" rel="noopener">
        <i class="fa fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=Immune%20System&amp;body=%2ftoo2%2fnew_proteo%2fimmune%2f">
        <i class="fa fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>


    <div class="article-style" itemprop="articleBody">
      

<hr />

<p><img src="/img/tools/protein/immune.jpg"  width="500" height="400" alt="anno" align="center"></p>

<p><span id="top"></span></p>

<h2>Table of Contents</h2>
<nav id="TableOfContents">
<ul>
<li><a href="#immune-system-analysis">Immune System Analysis</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-sequence-based-b-cell-epitope-prediction-font"><font color=#CD5555 face="黑体">Sequence-based B-cell Epitope Prediction</font></a>
<ul>
<li><a href="#abcpred">ABCpred</a></li>
<li><a href="#bepipred">BepiPred</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-structure-based-b-cell-epitope-prediction-font"><font color=#CD5555 face="黑体">Structure-based B-cell Epitope Prediction</font></a>
<ul>
<li><a href="#lyra">LYRA</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-mhc-class-i-epitope-prediction-font"><font color=#CD5555 face="黑体">MHC Class I Epitope Prediction</font></a>
<ul>
<li><a href="#propred1">ProPred1</a></li>
<li><a href="#graddock">GradDock</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-mhc-class-ii-epitope-prediction-font"><font color=#CD5555 face="黑体">MHC Class II Epitope Prediction</font></a>
<ul>
<li><a href="#hlamatchmaker">HLAMatchmaker</a></li>
<li><a href="#propred">ProPred</a></li>
</ul></li>
</ul></li>
<li><a href="#flow-cytometry">Flow Cytometry</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-cell-population-identification-font"><font color=#CD5555 face="黑体">Cell Population Identification</font></a>
<ul>
<li><a href="#fcs-express">FCS Express</a></li>
<li><a href="#flowjo">FlowJo</a></li>
</ul></li>
</ul></li>
<li><a href="#mass-cytometry">Mass Cytometry</a>
<ul>
<li><a href="#spade-spanning-tree-progression-of-density-normalized-events">SPADE (Spanning tree Progression of Density normalized Events</a></li>
<li><a href="#visne-data-visualization-flow-cytometry">viSNE | Data visualization : Flow cytometry</a></li>
</ul></li>
<li><a href="#find-more-tools-omictools-https-omictools-com-proteomics-category">Find more tools: <a href="https://omictools.com/proteomics-category" target="_blank"><strong>OMICTOOLS</strong></a></a></li>
</ul>
</nav>


<h1 id="immune-system-analysis">Immune System Analysis</h1>

<p><p align="justify"><strong>Immunoinformatics</strong> involves the application of computational methods to immunological problems. Prediction of B- and T-cell epitopes has long been the focus of immunoinformatics, given the potential translational implications, and many tools have been developed. With the advent of next-generation sequencing (NGS) methods, an unprecedented wealth of information has become available that requires more-advanced immunoinformatics tools. Based on information from whole-genome sequencing, exome sequencing and RNA sequencing, it is possible to characterize with high accuracy an individual’s human leukocyte antigen (HLA) allotype (i.e., the individual set of HLA alleles of the patient), as well as changes arising in the HLA ligandome (the collection of peptides presented by the HLA) owing to genomic variation. This has allowed new opportunities for translational applications of epitope prediction, such as epitope-based design of prophylactic and therapeutic vaccines, and personalized cancer immunotherapies. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26589500" target="_blank">Source</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-sequence-based-b-cell-epitope-prediction-font"><font color=#CD5555 face="黑体">Sequence-based B-cell Epitope Prediction</font></h2>

<p><p align="justify">For over 30 years, computational methods have been developed for facilitating epitope recognition (El-Manzalawy and Honavar, 2010). In the past, the majority of the in silico methods were focused on linear epitopes. Most of these approaches are sequence-based and use amino acid-based propensity scales, such as hydrophilicity, solvent accessibility, secondary structure and flexibility; a score derived from the propensity scales is assigned to each residue, and the whole sequence is examined for high-scoring window fragments, which are then predicted as epitopes (Hopp and Woods, 1981; Parker et al., 1986; Pellequer et al., 1991; Emini et al., 1985; Karplus and Schulz, 1985; Kolaskar and Tongaonkar, 1990; Welling et al., 1985).</p>

<hr />

<h3 id="abcpred">ABCpred</h3>

<p><p align="justify">Predicts B cell epitope(s) in an antigen sequence, using artificial neural network. ABCpred is able to predict epitopes with 65.93% accuracy using recurrent neural network.</p>

<p><a href="http://crdd.osdd.net/raghava/abcpred/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2006 Proteins) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16894596" target="_blank">Prediction of continuous B-cell epitopes in an antigen using recurrent neural network</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Institute of Microbial Technology, Chandigarh, India</p>

<hr />

<h3 id="bepipred">BepiPred</h3>

<p><p align="justify">Predicts the location of linear B-cell epitopes using a combination of a hidden Markov model (HMM) and a propensity scale method. BepiPred is a web server constructed from a large set of structurally defined B cell epitopes. The interface can aid researchers with limited computational knowledge to use and understand the results to their full extent. Additionally, the advanced option allows more experienced researchers to further interpret the output based on additionally predicted structural features.</p>

<p><a href="http://www.cbs.dtu.dk/services/BepiPred/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Nucleic Acids Res) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28472356" target="_blank">BepiPred-2.0: improving sequence-based B-cell epitope prediction using conformational epitopes.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Bio and Health Informatics, Technical University of Denmark, Kgs, Lyngby, Denmark; La Jolla, Institute for Allergy and Immunology, La Jolla, CA, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-structure-based-b-cell-epitope-prediction-font"><font color=#CD5555 face="黑体">Structure-based B-cell Epitope Prediction</font></h2>

<p><p align="justify">In recent years, computational identification of immunogenic regions/segments in a given protein antigen has provided increasing assistance in guiding the experimental validation. Since the majority of the epitope area was dominated by discontinuous amino acids in the surface of protein antigens (Haste Andersen et al., 2006), a lot of efforts have been devoted into computing spatial/conformational epitopes based on protein structures. These methods can be roughly divided into two tracks, one of which proposing useful parameters to discriminate epitope residues from common surface ones, while the other focusing on various classification algorithm to improve the performance.</p>

<hr />

<h3 id="lyra">LYRA</h3>

<p><p align="justify">Implements a complete and automated method for building of B- and T-cell receptor structural models starting from their amino acid sequence alone. LYRA is based on the canonical structure method and can produce, easily and within minutes, extremely reliable models of lymphocyte receptors. It is freely available and easy to use for non-specialists. Upon submission, LYRA automatically generates alignments using ad hoc profiles, predicts the structural class of each hypervariable loop, selects the best templates in an automatic fashion, and provides within minutes a complete 3D model that can be downloaded or inspected online. Experienced users can manually select or exclude template structures according to case specific information.</p>

<p><a href="http://www.cbs.dtu.dk/services/LYRA/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2015 Nucleic Acids Res) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26007650" target="_blank">LYRA, a webserver for lymphocyte receptor structural modeling.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Center for Biological Sequence Analysis, Technical University of Denmark, Kgs Lyngby, Denmark</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-mhc-class-i-epitope-prediction-font"><font color=#CD5555 face="黑体">MHC Class I Epitope Prediction</font></h2>

<p><p align="justify">A key role in cell-mediated immunity is dedicated to the major histocompatibility complex (MHC) molecules that bind peptides for presentation on the cell surface. Several in silico methods capable of predicting peptide binding to MHC class I have been developed. The accuracy of these methods depends on the data available characterizing the binding specificity of the MHC molecules.</p>

<hr />

<h3 id="propred1">ProPred1</h3>

<p><p align="justify">An on-line service for identifying the MHC Class-I binding regions in antigens. It implements matrices for 47 MHC Class-I alleles, proteasomal and immunoproteasomal models. The main aim of ProPred1 is to help users in identifying the promiscuous regions.</p>

<p><a href="http://crdd.osdd.net/raghava/propred1/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2003 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/12761064" target="_blank">ProPred1: prediction of promiscuous MHC Class-I binding sites.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Institute of Microbial Technology, Chandigarh, India</p>

<hr />

<h3 id="graddock">GradDock</h3>

<p><p align="justify">Decouples global and local conformational searches and generates diverse native-like peptide conformations for peptide binding to major histocompatibility complex class I (pMHC-I). GradDock uses a ranking function specific for pMHC-I which was reconstructed from the Rosetta scoring terms. It provides three steps: (1) the generation of peptide candidates from the sequence; (2) the insertion of peptides into the MHC-I molecule; and (3) the ranking of the peptides. It can create native-like peptides with a certain degree of diversity, which is sufficient for describing the pMHC-I space.</p>

<p><a href="http://crdd.osdd.net/raghava/propred1/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Bioinformatics) <a href="https://academic.oup.com/bioinformatics/article-abstract/34/3/469/4160677?redirectedFrom=fulltext" target="_blank">GradDock: Rapid Simulation and Tailored Ranking Functions for Peptide-MHC Class I Docking.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Biological Sciences, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-mhc-class-ii-epitope-prediction-font"><font color=#CD5555 face="黑体">MHC Class II Epitope Prediction</font></h2>

<p><p align="justify">CD4+ T-cell epitopes play a crucial role in eliciting vigorous protective immune responses during peptide (epitope)-based vaccination. The prediction of these epitopes focuses on the peptide binding process by MHC class II proteins. The ability to account for MHC class II polymorphism is critical for epitope-based vaccine design tools, as different allelic variants can have different peptide repertoires. In addition, the specificity of CD4+ T-cells is often directed to a very limited set of immunodominant peptides in pathogen proteins. The ability to predict what epitopes are most likely to dominate an immune response remains a challenge.</p>

<hr />

<h3 id="hlamatchmaker">HLAMatchmaker</h3>

<p><p align="justify">Determines human leukocyte antigen (HLA) matching at epitope level in alloantibody responses. HLAMatchmaker utilizes an algorithm where each HLA antigen is considered as a string of amino acid configurations in antibody-accessible positions. This program can be used as a quantitative tool to define the degree of a mismatch such as mismatched eplets of triplets.</p>

<p><a href="http://www.epitopes.net/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2011 Tissue Antigens) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21410655" target="_blank">Antibody-reactive epitope determination with HLAMatchmaker and its clinical applications.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Division of Transplantation Pathology, Thomas E. Starzl Transplantation Institute, University of Pittsburgh Medical Center, Pittsburgh, PA, USA</p>

<hr />

<h3 id="propred">ProPred</h3>

<p><p align="justify">Analyses sequence data to predict MHC class II binding regions in antigenic protein sequences. Propred (generates text or graphic outputs, presents the binding score calculated from matrix in tabular format, selects an appropriate threshold for locating promiscuous binders using a threshold profile and performs analysis on any sub-sequence including the computation of the binding strength of all peptide frames.</p>

<p><a href="http://osddlinux.osdd.net/raghava/propred/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2001 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/11751237" target="_blank">ProPred: prediction of HLA-DR binding sites.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Bioinformatics Centre, Institute of Microbial Technology, Sector 39A, Chandigarth, India</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="flow-cytometry">Flow Cytometry</h1>

<p><p align="justify"><strong>Flow cytometry</strong> is a technology widely used in clinical practice and in research, particularly in the field of immunology. It is capable of interrogating a wide variety of markers on many different cell types on a single-cell basis, using fluorophore-conjugated antibodies. Although molecular as well as genomic studies have advanced the understanding of immunological processes and autoimmune diseases, the components of the human immune system and their functions have yet to be comprehensively described. Without such a reference “catalog” of the immune system, it is ultimately difficult to interpret the pathogenic significance of genetic, molecular, or phenotypic variants observed in diseases.</p>

<hr />

<h2 id="font-color-cd5555-face-黑体-cell-population-identification-font"><font color=#CD5555 face="黑体">Cell Population Identification</font></h2>

<p><p align="justify">Traditionally, the majority of flow cytometry experiments have been analyzed visually, either by serial manual inspection of one or two dimensions at a time (a process termed “gating”, with boundaries or “gates” defining cell populations of interest). However, these visual approaches are labor intensive and highly subjective, and they neglect information present in the data that are not visible to the human eye, thus representing a major obstacle to the automation and reproducibility of research.</p>

<hr />

<h3 id="fcs-express">FCS Express</h3>

<p><p align="justify">Enables flow and image cytometry data analysis. FCS Express is able to analyze flow cytometry data, with flow cytometry tools, plots, and gates. The software allows export to Powerpoint, pdf, Excel, Word, and Microsoft office.</p>

<p><a href="https://www.denovosoftware.com/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Forum</strong>:</p>

<p><a href="https://www.denovosoftware.com/site/forums.shtml" target="_blank">Forum</a></p>

<hr />

<h3 id="flowjo">FlowJo</h3>

<p><p align="justify">A flow cytometry data analysis software.. FlowJo contains a list of loaded samples (experimental data), statistics, gates, and other analyses, as well as tabular and graphical layouts. FlowJo provides features and tools for the creation of histogram and other plot overlays, cell cycle analysis, calcium flux analysis, proliferation analysis, quantitation, cluster identification and backgating display.</p>

<p><a href="https://www.flowjo.com/solutions/flowjo" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>CITATIONS</strong>:</p>

<ul>
<li>(2017) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5607261/" target="_blank">Circulating Mycobacterium tuberculosis DosR latency antigen-specific, polyfunctional, regulatory IL10+ Th17 CD4 T-cells differentiate latent from active tuberculosis</a></li>
</ul>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="mass-cytometry">Mass Cytometry</h1>

<p><p align="justify">Mass cytometry addresses the analytical challenges of polychromatic flow cytometry by using metal atoms as tags rather than fluorophores and atomic mass spectrometry as the detector rather than photon optics.</p>

<hr />

<h2 id="spade-spanning-tree-progression-of-density-normalized-events">SPADE (Spanning tree Progression of Density normalized Events</h2>

<p>)</p>

<p><p align="justify">Facilitates the analysis of cellular heterogeneity, the identification of cell types, and comparison of functional markers in response to perturbations, based on a versatile method. SPADE helps to organize high-dimensional cytometry data in an unsupervised manner, and to investigate natural and pathogenic cellular heterogeneity for biological insight. The SPADE algorithm consists of four components: (i) density-dependent downsampling, (ii) clustering, (iii) linking clusters with a minimum spanning tree, and (iv) upsampling to restore all cells in the final result. This modularized process allows more efficient sub-algorithms to replace the current components. In this sense, SPADE can be viewed as a framework for cytometric data analysis and visualization that has the capacity to be evolved and adapted.</p>

<p><a href="https://github.com/nolanlab/spade" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li><p>(2011 Nat Biotechnol) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21964415" target="_blank">Extracting a cellular hierarchy from high-dimensional cytometry data with SPADE.</a></p></li>

<li><p>(2011 Science) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21551058" target="_blank">Single-cell mass cytometry of differential immune and drug responses across a human hematopoietic continuum.</a></p></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Radiology, Stanford University, Stanford, CA, USA</p>

<hr />

<h2 id="visne-data-visualization-flow-cytometry">viSNE | Data visualization : Flow cytometry</h2>

<p><p align="justify">A tool that allows one to map high-dimensional cytometry data onto two dimensions, yet conserve the high-dimensional structure of the data.</p>

<p><a href="https://www.denovosoftware.com/site/manual/visne.htm" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2013 Nat Biotechnol) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23685480" target="_blank">viSNE enables visualization of high dimensional single-cell data and reveals phenotypic heterogeneity of leukemia.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Biological Sciences, Columbia Initiative for Systems Biology, Columbia University, New York, NY, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="find-more-tools-omictools-https-omictools-com-proteomics-category">Find more tools: <a href="https://omictools.com/proteomics-category" target="_blank"><strong>OMICTOOLS</strong></a></h1>

<p><a href="https://www.proprofs.com/quiz-school/story.php?title=quiz-lecture-2-components--innate-immune-system" target="_blank"><em>Image Citation</em></a></p>

<hr />

    </div>

    


<div class="article-tags">
  
  <a class="btn btn-primary btn-outline" href="/tags/proteomics" target="_blank">proteomics</a>
  
</div>




    
    
    <div class="article-widget">
      <div class="hr-light"></div>
      <h3>Related</h3>
      <ul>
        
        <li><a href="/too2/new_proteo/interactions/">Protein Interactions</a></li>
        
        <li><a href="/too2/new_proteo/membrane/">Membrane Protein</a></li>
        
        <li><a href="/too2/new_proteo/pro_seq/">Protein Sequence/Expression</a></li>
        
        <li><a href="/too2/new_proteo/structure/">Prediction of Protein Structure</a></li>
        
      </ul>
    </div>
    

    

    


  </div>
</article>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2018 &middot; 

      Copyright© 2018 Lin QI of Dr.Xin Wang's Lab. zzx. All rights reserved. 

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    <script async defer src="//maps.googleapis.com/maps/api/js?key=AIzaSyD3AGpKDAfo_LjBRHd-YkYmMjBDlIHfdlo"></script>
    

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gmaps.js/0.4.25/gmaps.min.js" integrity="sha256-7vjlAeb8OaTrCXZkCNun9djzuB2owUsaO72kXaFDBJs=" crossorigin="anonymous"></script>
    
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
      

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

