<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.38.2" />
  <meta name="author" content="Computational Biology Lab">

  
  
  
  
    
      
    
  
  <meta name="description" content="Table of Contents  DNA modification analysis  DNA Methylation Prediction  T-BioInfo MethSurv Nanopolish  DNA Methylation Deconvolution  FaST-LMM-EWASher ReFACTor | Reference-Free Adjustment for Cell-Type composition EDec | Epigenomic Deconvolution   DNA methylation array analysis  Differential Methylation Site Detection  limma | Linear Models for Microarray Data FastDMA RnBeads  Differential Methylation region Detection  ChAMP FastDMA RnBeads   Enrichment Analysis  Over-representation analysis  Blast2GO g:Profiler STEM | Short Time-series Expression Miner  Gene set enrichment analysis  GSEA | Gene Set Enrichment Analysis DESeq | Differential expression : HTS analysis edgeR | empirical analysis of DGE in R  Topology enrichment analysis  DAVID | Database for Annotation, Visualization and Integrated Discovery HOMER | Hypergeometric Optimization of Motif EnRichment GeneCodis | Topology enrichment analysis   Bisulfite Sequencing Data Analysis (BS-seq analysis)  Differential Methylation region Detection  methylKit | Methylation annotation : BS-seq analysis SMART | Specific Methylation Analysis and Report Tool DSS-single | Differentially methylated region detection : BS-seq analysis  Methylation Annotation  GBSA | Genome Bisulfite Sequencing Analyser methylKit | Methylation annotation : BS-seq analysis  EWAS  ReFACTor | Reference-Free Adjustment for Cell-Type composition FaST-LMM-EWASher   Find more tools: OMICTOOLS   DNA modification analysis Whole genome DNA methylation detection is one of the most important part of epigenetics research.">

  
  <link rel="alternate" hreflang="en-us" href="/too2/new_epi/modifications/">

  


  

  
  
  <meta name="theme-color" content="#3f51b5">
  
  
  
  
    
  
  
    
    
      
        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
      
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">
  
  
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7cMerriweather%7cRoboto&#43;Mono">
  
  <link rel="stylesheet" href="/styles.css">
  

  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/too2/new_epi/modifications/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="og:site_name" content="CityU-Bioinformatics">
  <meta property="og:url" content="/too2/new_epi/modifications/">
  <meta property="og:title" content="DNA Modifications | CityU-Bioinformatics">
  <meta property="og:description" content="Table of Contents  DNA modification analysis  DNA Methylation Prediction  T-BioInfo MethSurv Nanopolish  DNA Methylation Deconvolution  FaST-LMM-EWASher ReFACTor | Reference-Free Adjustment for Cell-Type composition EDec | Epigenomic Deconvolution   DNA methylation array analysis  Differential Methylation Site Detection  limma | Linear Models for Microarray Data FastDMA RnBeads  Differential Methylation region Detection  ChAMP FastDMA RnBeads   Enrichment Analysis  Over-representation analysis  Blast2GO g:Profiler STEM | Short Time-series Expression Miner  Gene set enrichment analysis  GSEA | Gene Set Enrichment Analysis DESeq | Differential expression : HTS analysis edgeR | empirical analysis of DGE in R  Topology enrichment analysis  DAVID | Database for Annotation, Visualization and Integrated Discovery HOMER | Hypergeometric Optimization of Motif EnRichment GeneCodis | Topology enrichment analysis   Bisulfite Sequencing Data Analysis (BS-seq analysis)  Differential Methylation region Detection  methylKit | Methylation annotation : BS-seq analysis SMART | Specific Methylation Analysis and Report Tool DSS-single | Differentially methylated region detection : BS-seq analysis  Methylation Annotation  GBSA | Genome Bisulfite Sequencing Analyser methylKit | Methylation annotation : BS-seq analysis  EWAS  ReFACTor | Reference-Free Adjustment for Cell-Type composition FaST-LMM-EWASher   Find more tools: OMICTOOLS   DNA modification analysis Whole genome DNA methylation detection is one of the most important part of epigenetics research.">
  <meta property="og:locale" content="en-us">
  
  <meta property="article:published_time" content="2018-04-11T00:00:00&#43;00:00">
  
  <meta property="article:modified_time" content="2018-04-11T00:00:00&#43;00:00">
  

  

  <title>DNA Modifications | CityU-Bioinformatics</title>

</head>
<body id="top" data-spy="scroll" data-target="#toc" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/">CityU-Bioinformatics</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#bioinformatics">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/program/r">
            
            <span>R Programming</span>
            
          </a>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Course Tutorials</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/tutorial/bms4001/">
                
                <span>BMS4001 tutorials</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/bms8110/">
                
                <span>BMS8110 tutorials</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/resources/">
                
                <span>Online Resources</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Database</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/base2/dna">
                
                <span>DNA Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/express">
                
                <span>Genome Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/rna">
                
                <span>RNA Databses</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/protein">
                
                <span>Protein Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pheno">
                
                <span>Phenotype Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pathway">
                
                <span>Pathway Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/special">
                
                <span>Other Databases</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Tools</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/too2/genomics">
                
                <span>Genomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/epigenomics">
                
                <span>Epigenomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/transcriptomics">
                
                <span>Transcriptomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/proteomics">
                
                <span>Proteomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/metabolomics">
                
                <span>Metabolomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/phenomics">
                
                <span>Phenomics</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Journals</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/journal/comprehensive">
                
                <span>Multidisciplinary</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/specific">
                
                <span>Bioinformatics Specific</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/conferences">
                
                <span>Conferences</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/server/">
            
            <span>Servers</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>


<article class="article" itemscope itemtype="http://schema.org/Article">

  


  <div class="article-container">
    <h1 itemprop="name">DNA Modifications</h1>

    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-04-11 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Apr 11, 2018
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Computational Biology Lab">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    18 min read
  </span>
  

  
  

  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=DNA%20Modifications&amp;url=%2ftoo2%2fnew_epi%2fmodifications%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=%2ftoo2%2fnew_epi%2fmodifications%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-facebook"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=%2ftoo2%2fnew_epi%2fmodifications%2f&amp;title=DNA%20Modifications"
         target="_blank" rel="noopener">
        <i class="fa fa-linkedin"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=%2ftoo2%2fnew_epi%2fmodifications%2f&amp;title=DNA%20Modifications"
         target="_blank" rel="noopener">
        <i class="fa fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=DNA%20Modifications&amp;body=%2ftoo2%2fnew_epi%2fmodifications%2f">
        <i class="fa fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>


    <div class="article-style" itemprop="articleBody">
      

<hr />

<p><img src="/img/tools/geno/modification.jpg"  width="400" height="300" alt="anno" align="center"></p>

<p><span id="top"></span></p>

<h2>Table of Contents</h2>
<nav id="TableOfContents">
<ul>
<li><a href="#dna-modification-analysis">DNA modification analysis</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-dna-methylation-prediction-font"><font color=#CD5555 face="黑体">DNA Methylation Prediction</font></a>
<ul>
<li><a href="#t-bioinfo">T-BioInfo</a></li>
<li><a href="#methsurv">MethSurv</a></li>
<li><a href="#nanopolish">Nanopolish</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-dna-methylation-deconvolution-font"><font color=#CD5555 face="黑体">DNA Methylation Deconvolution</font></a>
<ul>
<li><a href="#fast-lmm-ewasher">FaST-LMM-EWASher</a></li>
<li><a href="#refactor-reference-free-adjustment-for-cell-type-composition">ReFACTor | Reference-Free Adjustment for Cell-Type composition</a></li>
<li><a href="#edec-epigenomic-deconvolution">EDec | Epigenomic Deconvolution</a></li>
</ul></li>
</ul></li>
<li><a href="#dna-methylation-array-analysis">DNA methylation array analysis</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-differential-methylation-site-detection-font"><font color=#CD5555 face="黑体">Differential Methylation Site Detection</font></a>
<ul>
<li><a href="#limma-linear-models-for-microarray-data">limma | Linear Models for Microarray Data</a></li>
<li><a href="#fastdma">FastDMA</a></li>
<li><a href="#rnbeads">RnBeads</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-differential-methylation-region-detection-font"><font color=#CD5555 face="黑体">Differential Methylation region Detection</font></a>
<ul>
<li><a href="#champ">ChAMP</a></li>
<li><a href="#fastdma-1">FastDMA</a></li>
<li><a href="#rnbeads-1">RnBeads</a></li>
</ul></li>
</ul></li>
<li><a href="#enrichment-analysis">Enrichment Analysis</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-over-representation-analysis-font"><font color=#CD5555 face="黑体">Over-representation analysis</font></a>
<ul>
<li><a href="#blast2go">Blast2GO</a></li>
<li><a href="#g-profiler">g:Profiler</a></li>
<li><a href="#stem-short-time-series-expression-miner">STEM | Short Time-series Expression Miner</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-gene-set-enrichment-analysis-font"><font color=#CD5555 face="黑体">Gene set enrichment analysis</font></a>
<ul>
<li><a href="#gsea-gene-set-enrichment-analysis">GSEA | Gene Set Enrichment Analysis</a></li>
<li><a href="#deseq-differential-expression-hts-analysis">DESeq | Differential expression : HTS analysis</a></li>
<li><a href="#edger-empirical-analysis-of-dge-in-r">edgeR | empirical analysis of DGE in R</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-topology-enrichment-analysis-font"><font color=#CD5555 face="黑体">Topology enrichment analysis</font></a>
<ul>
<li><a href="#david-database-for-annotation-visualization-and-integrated-discovery">DAVID | Database for Annotation, Visualization and Integrated Discovery</a></li>
<li><a href="#homer-hypergeometric-optimization-of-motif-enrichment">HOMER | Hypergeometric Optimization of Motif EnRichment</a></li>
<li><a href="#genecodis-topology-enrichment-analysis">GeneCodis | Topology enrichment analysis</a></li>
</ul></li>
</ul></li>
<li><a href="#bisulfite-sequencing-data-analysis-bs-seq-analysis">Bisulfite Sequencing Data Analysis (BS-seq analysis)</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-differential-methylation-region-detection-font-1"><font color=#CD5555 face="黑体">Differential Methylation region Detection</font></a>
<ul>
<li><a href="#methylkit-methylation-annotation-bs-seq-analysis">methylKit | Methylation annotation : BS-seq analysis</a></li>
<li><a href="#smart-specific-methylation-analysis-and-report-tool">SMART | Specific Methylation Analysis and Report Tool</a></li>
<li><a href="#dss-single-differentially-methylated-region-detection-bs-seq-analysis">DSS-single | Differentially methylated region detection : BS-seq analysis</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-methylation-annotation-font"><font color=#CD5555 face="黑体">Methylation Annotation</font></a>
<ul>
<li><a href="#gbsa-genome-bisulfite-sequencing-analyser">GBSA | Genome Bisulfite Sequencing Analyser</a></li>
<li><a href="#methylkit-methylation-annotation-bs-seq-analysis-1">methylKit | Methylation annotation : BS-seq analysis</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-ewas-font"><font color=#CD5555 face="黑体">EWAS</font></a>
<ul>
<li><a href="#refactor-reference-free-adjustment-for-cell-type-composition-1">ReFACTor | Reference-Free Adjustment for Cell-Type composition</a></li>
<li><a href="#fast-lmm-ewasher-1">FaST-LMM-EWASher</a></li>
</ul></li>
</ul></li>
<li><a href="#find-more-tools-omictools-https-omictools-com-epigenomics-category">Find more tools: <a href="https://omictools.com/epigenomics-category" target="_blank"><strong>OMICTOOLS</strong></a></a></li>
</ul>
</nav>


<h1 id="dna-modification-analysis">DNA modification analysis</h1>

<p><p align="justify"><strong>Whole genome DNA methylation detection</strong> is one of the most important part of epigenetics research. It is supposed to have a great effect on cancers and tumors, and even be involved in the senility of human. In addition, it is believed that in medical aspects, DNA methylation may have a strong relationship with diabetes and immunological diseases (Jeong et al., 2014; Hackett et al., 2013; Duthie, 2011).</p>

<hr />

<h2 id="font-color-cd5555-face-黑体-dna-methylation-prediction-font"><font color=#CD5555 face="黑体">DNA Methylation Prediction</font></h2>

<h3 id="t-bioinfo">T-BioInfo</h3>

<p><p align="justify">Combines statistical analysis modules into pipelines to deal with heterogenous big data. T-BioInfo is an application that can be used for: (1) next-generation sequencing (NGS) data (transcriptomics, genomics/epigenetics, and DNA/RNA); (2) mass-spectroscopy; (3) structural biology; and (4) data integration and modeling (virology, data association, and data mining).</p>

<p><a href="https://server.t-bio.info/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4494065/" target="_blank">Differential Expression of Genes that Control Respiration Contribute to Thermal Adaptation in Redband Trout</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>University of Haifa, Israel Pine Biotech, Haifa, Israel</p>

<hr />

<h3 id="methsurv">MethSurv</h3>

<p><p align="justify">Correlates overall survival with DNA methylation levels. MethSurv allows to investigate methylation biomarkers that associate with the survival of various human cancers. It combines unsupervised hierarchical clustering and principal component analysis (PCA) for any particular gene. This tool can give a graphical overview of methylation differences between the cancer patients as well as gene subregions.</p>

<p><a href="https://biit.cs.ut.ee/methsurv/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Modhukur et al., 2017)<a href="https://www.ncbi.nlm.nih.gov/pubmed/29264942" target="_blank"> MethSurv: a web tool to perform multivariable survival analysis using DNA methylation data</a> Epigenomics.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Institute of Computer Science, University of Tartu, Tartu, Estonia; United Laboratories of Tartu University Hospital, Tartu University Hospital, Tartu, Estonia.</p>

<hr />

<h3 id="nanopolish">Nanopolish</h3>

<p><p align="justify">Provides a nanopore consensus algorithm using a signal-level hidden Markov model (HMM). The main subprograms of Nanopolish are: (i) nanopolish extract which extracts reads in FASTA or FASTQ format from a directory of FAST5 files; (ii) nanopolish eventalign which aligns signal-level events to k-mers of a reference genome; (iii) nanopolish variants which detects single nucleotide polymorphisms (SNPs) and indels with respect to a reference genome; and (iv) nanopolish variants –consensus which calculates an improved consensus sequence for a draft genome assembly. Furthermore, Nanopolish contains an experimental option that will use event durations to improve the consensus accuracy around homopolymers.</p>

<p><a href="https://github.com/jts/nanopolish" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Simpson et al., 2017) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28218898" target="_blank">Detecting DNA cytosine methylation using nanopore sequencing</a>.  Nat Methods.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Ontario Institute for Cancer Research, Toronto, Ontario, Canada; Department of Computer Science, University of Toronto, Toronto, Ontario, Canada; Department of Biomedical Engineering, Johns Hopkins University, Baltimore, MD, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-dna-methylation-deconvolution-font"><font color=#CD5555 face="黑体">DNA Methylation Deconvolution</font></h2>

<h3 id="fast-lmm-ewasher">FaST-LMM-EWASher</h3>

<p><p align="justify">An R version of FaST-LMM-EWASher, which performs epigenome-wide association analysis in the presence of confounders such as cell-type heterogeneity. A python version of this software is also available as part of Fast-LMM-Py.</p>

<p><a href="https://www.microsoft.com/en-us/download/details.aspx?id=52501" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Zou et al., 2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24464286" target="_blank">Epigenome-wide association studies without the need for cell-type composition</a>.  Nat Methods.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>eScience Research Group, Microsoft Research, Los Angeles, CA, USA; The Broad Institute of MIT and Harvard, Cambridge, MA, USA</p>

<hr />

<h3 id="refactor-reference-free-adjustment-for-cell-type-composition">ReFACTor | Reference-Free Adjustment for Cell-Type composition</h3>

<p><p align="justify">A method based on principal component analysis (PCA) and designed for the correction of cell type heterogeneity in epigenome-wide association studies (EWAS). ReFACTor tool is based on a variant of PCA and can be applied to any tissue. It selects the sites that can be reconstructed with low error using a low-rank approximation of the original methylation matrix. Moreover, ReFACTor does not use the phenotype in the selection process, making ReFACTor useful as part of a quality control step in EWAS.</p>

<p><a href="http://www.cs.tau.ac.il/~heran/cozygene/software/refactor.html" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://github.com/cozygene/refactor" target="_blank"><strong>Github</strong></a></p>

<p><a href="http://www.cs.tau.ac.il/~heran/cozygene/software/refactor_assets/README.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Rahmani et al., 2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27018579" target="_blank">Sparse PCA corrects for cell type heterogeneity in epigenome-wide association studies</a>.  Nat Methods.
PMID: 27018579</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Blavatnik School of Computer Science, Tel-Aviv University, Tel Aviv, Israel; Department of Medicine, University of California, San Francisco, CA, USA</p>

<hr />

<h3 id="edec-epigenomic-deconvolution">EDec | Epigenomic Deconvolution</h3>

<p><p align="justify">Provides accurate platform-independent estimation of cell type proportions, DNA methylation profiles and gene expression profiles of constituent cell type. EDec enables deconvolution of complex tumor tissues where highly accurate reference are enables. EDec reveals layers of biological information about distinct cell types within solid tumors and about their heterotypic interactions that were previously inaccessible at such large scale due to tissue heterogeneity.</p>

<p><a href="https://github.com/BRL-BCM/EDec" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://github.com/BRL-BCM/EDec" target="_blank"><strong>Github</strong></a></p>

<p><a href="http://www.cs.tau.ac.il/~heran/cozygene/software/refactor_assets/README.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Onuchic et al., 2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27851969" target="_blank">Epigenomic Deconvolution of Breast Tumors Reveals Metabolic Coupling between Constituent Cell Types.</a>  Cell Rep.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Molecular and Human Genetics Department, Baylor College of Medicine, Houston, TX, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="dna-methylation-array-analysis">DNA methylation array analysis</h1>

<p><p align="justify"><strong>DNA methylation</strong> is involved in numerous physiological processes and also disease states, such as cancer (Jones, 2012). This has raised wide interest in developing large-scale DNA methylation profiling technologies to improve our molecular understanding of diseases. The recently released Infinium HumanMethylation450 (Bibikova et al., 2011; Dedeurwaerder et al., 2011) is a preferred technology for studying the DNA methylomes of various cell types in large-scale studies, and there is a current explosion of data generated with this technology (Rakyan et al., 2011). Sequencing-based methods, although offering much higher genome coverage, are still not affordable by all laboratories, notably those with moderate budgets. Another reason for the success of DNA methylation arrays is the ease of reading and understanding the data generated, notably because microarrays have been widely used over the past decades, particularly for gene expression profiling.</p>

<h2 id="font-color-cd5555-face-黑体-differential-methylation-site-detection-font"><font color=#CD5555 face="黑体">Differential Methylation Site Detection</font></h2>

<h3 id="limma-linear-models-for-microarray-data">limma | Linear Models for Microarray Data</h3>

<p><p align="justify">Provides an integrated solution for analysing data from gene expression experiments. limma contains rich features for handling complex experimental designs and for information borrowing to overcome the problem of small sample sizes. It also contains particularly strong facilities for reading, normalizing and exploring such data. Recently, the capabilities of limma have been significantly expanded in two important directions: (i) it can perform both differential expression and differential splicing analyses of RNA-seq data; (ii) the package is now able to go past the traditional gene-wise expression analyses in a variety of ways, analysing expression profiles in terms of co-regulated sets of genes or in terms of higher-order expression signatures. This provides enhanced possibilities for biological interpretation of gene expression differences.</p>

<p><a href="http://bioconductor.org/packages/release/bioc/html/limma.html" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="http://bioconductor.org/packages/release/bioc/manuals/limma/man/limma.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Smyth et al., 2005) <a href="https://www.ncbi.nlm.nih.gov/pubmed/15657102" target="_blank">Use of within-array replicate spots for assessing differential expression in microarray experiments</a>. Bioinformatics.<br /></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Bioinformatics Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, VIC, Australia</p>

<hr />

<h3 id="fastdma">FastDMA</h3>

<p><p align="justify">A software analyzing Illumina Infinium HumanMethylation450 BeadChip data, which is featured as multiple core parallel computing.</p>

<p><a href="http://bioinfo.au.tsinghua.edu.cn/member/jgu/fastdma/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Wu et al., 2013) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24040221" target="_blank">FastDMA: an infinium humanmethylation450 beadchip analyzer</a>.  PLoS One.<br /></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Bioinformatics Division/Center for Synthetic and Systems Biology, Tsinghua National Laboratory for Information Science and Technology (TNLIST), Department of Automation, Tsinghua University, Beijing, China</p>

<hr />

<h3 id="rnbeads">RnBeads</h3>

<p><p align="justify">An R package for comprehensive analysis of DNA methylation data obtained with any experimental protocol that provides single-CpG resolution, including Infinium 450K microarray and bisulfite sequencing protocols, but also MeDIP-seq and MBD-seq.</p>

<p><a href="https://rnbeads.org/" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://toolshed.g2.bx.psu.edu/repository?repository_id=e9e57142d30f1062" target="_blank"><strong>Galaxy</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Assenov et al., 2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25262207" target="_blank">Comprehensive analysis of DNA methylation data with RnBeads</a>.  Nat Methods.<br /></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Max Planck Institute for Informatics, Saarbrücken, Germany</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-differential-methylation-region-detection-font"><font color=#CD5555 face="黑体">Differential Methylation region Detection</font></h2>

<h3 id="champ">ChAMP</h3>

<p><p align="justify">Allows Illumina HumanMethylation BeadChip analysis. ChAMP is an integrated analysis pipeline including functions for (i) filtering low quality probes, adjustment for Infinium I and Infinium II probe design, (ii) batch effect correction, detecting differentially methylated positions (DMPs), (iii) finding differentially methylated regions (DMRs) and (iv) detection of copy number aberrations. The software also allows detection of differentially methylated genomic blocks (DMB) and Gene Set Enrichment Analysis (GSEA).</p>

<p><a href="http://bioconductor.org/packages/release/bioc/html/ChAMP.html" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="http://www.bioconductor.org/packages/2.13/bioc/manuals/ChAMP/man/ChAMP.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Morris et al., 2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24336642" target="_blank">ChAMP: 450k Chip Analysis Methylation Pipeline</a>.  Bioinformatics.</li>
<li>(Butcher and Beck, 2015) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25461817" target="_blank">Probe Lasso: a novel method to rope in differentially methylated regions with 450K DNA methylation data</a>. Methods.</li>
<li>(Tian et al., 2017) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28961746" target="_blank">ChAMP: Updated Methylation Analysis Pipeline for Illumina BeadChips</a>.  Bioinformatics.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>CAS Key Lab of Computational Biology, CAS-MPG Partner Institute for Computational Biology, Shanghai Institute for Biological Sciences, Chinese Academy of Sciences, Shanghai, China</p>

<hr />

<h3 id="fastdma-1">FastDMA</h3>

<p><p align="justify">A software analyzing Illumina Infinium HumanMethylation450 BeadChip data, which is featured as multiple core parallel computing.</p>

<p><a href="http://bioinfo.au.tsinghua.edu.cn/member/jgu/fastdma/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Wu et al., 2013) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24040221" target="_blank">FastDMA: an infinium humanmethylation450 beadchip analyzer</a>.  PLoS One.<br /></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Bioinformatics Division/Center for Synthetic and Systems Biology, Tsinghua National Laboratory for Information Science and Technology (TNLIST), Department of Automation, Tsinghua University, Beijing, China</p>

<hr />

<h3 id="rnbeads-1">RnBeads</h3>

<p><p align="justify">An R package for comprehensive analysis of DNA methylation data obtained with any experimental protocol that provides single-CpG resolution, including Infinium 450K microarray and bisulfite sequencing protocols, but also MeDIP-seq and MBD-seq.</p>

<p><a href="https://rnbeads.org/" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://toolshed.g2.bx.psu.edu/repository?repository_id=e9e57142d30f1062" target="_blank"><strong>Galaxy</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Assenov et al., 2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25262207" target="_blank">Comprehensive analysis of DNA methylation data with RnBeads</a>.  Nat Methods.<br /></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Max Planck Institute for Informatics, Saarbrücken, Germany</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="enrichment-analysis">Enrichment Analysis</h1>

<h2 id="font-color-cd5555-face-黑体-over-representation-analysis-font"><font color=#CD5555 face="黑体">Over-representation analysis</font></h2>

<h3 id="blast2go">Blast2GO</h3>

<p><p align="justify">Permits functional annotation, management, and data mining of novel sequence data. Blast2GO is based on the utilization of common controlled vocabulary schemas, the gene ontology (GO). It takes in consideration similarity, the extension of the homology, the database of choice, the GO hierarchy, and the quality of the original annotations. This tool is suitable for plant genomics research. It generates functional annotation and assesses the functional meaning of their experimental results.</p>

<p><a href="https://www.blast2go.com/" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://toolshed.g2.bx.psu.edu/repository?repository_id=e4de88c47079d971" target="_blank"><strong>Galaxy</strong></a></p>

<p><a href="https://www.blast2go.com/images/b2g_pdfs/b2g_user_manual.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Conesa et al., 2005) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16081474" target="_blank">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</a>.<br /></li>
<li>(Conesa and Götz, 2008) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18483572" target="_blank">Blast2GO: A comprehensive suite for functional analysis in plant genomics.</a>  Int J Plant Genomics.</li>
<li>(Götz et al., 2008) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18445632" target="_blank">High-throughput functional annotation and data mining with the Blast2GO suite.</a>  Nucleic Acids Res.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Bioinformatics Department, Centro de Investigación Príncipe Felipe, Valencia, Spain</p>

<hr />

<h3 id="g-profiler">g:Profiler</h3>

<p><p align="justify">Provides tool to perform functional enrichment analysis and mine additional information. g:Profiler is a web server that allows to characterize and manipulate gene lists of high-throughput genomics. This tool analyses flat or ranked gene lists for enriched features, converts gene identifiers of different classes, maps genes to orthologous genes in related species, finds similarly expressed genes from public microarray and maps human single nucleotide polymorphisms (SNP) to gene names, chromosomal locations and variant consequence terms from Sequence Ontology (SO).</p>

<p><a href="https://biit.cs.ut.ee/gprofiler/" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://cran.r-project.org/web/packages/gProfileR/gProfileR.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Reimand et al., 2011) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21646343" target="_blank">g:Profiler&ndash;a web server for functional interpretation of gene lists (2011 update)</a>.  Nucleic Acids Res.</li>
<li>(Reimand et al., 2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27098042" target="_blank">g:Profiler-a web server for functional interpretation of gene lists (2016 update)</a>.  Nucleic Acids Res.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Ontario Institute for Cancer Research, Toronto, ON, Canada; Department of Medical Biophysics, University of Toronto, Toronto, ON, Canada; Institute of Computer Science, University of Tartu, Tartu, Estonia</p>

<hr />

<h3 id="stem-short-time-series-expression-miner">STEM | Short Time-series Expression Miner</h3>

<p><p align="justify">A software program specifically designed for the analysis of short time series microarray gene expression data. STEM implements unique methods to cluster, compare, and visualize such data. STEM also supports efficient and statistically rigorous biological interpretations of short time series data through its integration with the Gene Ontology.</p>

<p><a href="http://www.cs.cmu.edu/~jernst/stem/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Ernst and Bar-Joseph, 2006) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16597342" target="_blank">STEM: a tool for the analysis of short time series gene expression data</a>.  BMC Bioinformatics.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Center for Automated and Learning and Discovery, School of Computer Science, Carnegie Mellon University, Pittsburgh, PA, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-gene-set-enrichment-analysis-font"><font color=#CD5555 face="黑体">Gene set enrichment analysis</font></h2>

<h3 id="gsea-gene-set-enrichment-analysis">GSEA | Gene Set Enrichment Analysis</h3>

<p><p align="justify">Evaluates microarray data at the level of gene sets. GSEA aims to determine whether members of a gene set S tend to occur toward the top (or bottom) of the list L, in which case the gene set is correlated with the phenotypic class distinction. This method eases the interpretation of a largescale experiment by identifying pathways and processes, and can boost the signal-to-noise ratio when the members of a gene set exhibit strong cross-correlation, allowing to detect modest changes in individual genes.</p>

<p><a href="http://software.broadinstitute.org/gsea/index.jsp" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://groups.google.com/forum/#!forum/gsea-help" target="_blank"><strong>Forum</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Subramanian et al., 2005) <a href="https://www.ncbi.nlm.nih.gov/pubmed/16199517" target="_blank">Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles</a>.  Proc Natl Acad Sci U S A.</li>
<li>(Mootha et al., 2003) <a href="https://www.ncbi.nlm.nih.gov/pubmed/12808457" target="_blank">PGC-1alpha-responsive genes involved in oxidative phosphorylation are coordinately downregulated in human diabetes</a>.  Nat Genet.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Broad Institute of Massachusetts Institute of Technology and Harvard, Cambridge, MA, USA; Department of Systems Biology, Harvard Medical School, Boston, MA, USA</p>

<hr />

<h3 id="deseq-differential-expression-hts-analysis">DESeq | Differential expression : HTS analysis</h3>

<p><p align="justify">Performs differential gene expression analysis. DEseq is a method that integrates methodological advances with features to facilitate quantitative analysis of comparative RNA-seq data using shrinkage estimators for dispersion and fold change. The software is suitable for small studies with few replicates as well as for large observational studies. Its heuristics for outlier detection assist in recognizing genes for which the modeling assumptions are unsuitable and so avoids type-I errors caused by these.</p>

<p><a href="http://bioconductor.org/packages/release/bioc/html/DESeq2.html" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="http://bioconductor.org/packages/release/bioc/manuals/DESeq2/man/DESeq2.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Love et al., 2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25516281" target="_blank">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</a>.  Genome Biol.</li>
<li>(Anders and Huber, 2010) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20979621" target="_blank">Differential expression analysis for sequence count data</a>.  Genome Biol.<br /></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Biostatistics and Computational Biology, Dana Farber Cancer Institute, Boston, MA, USA; Department of Biostatistics, Harvard School of Public Health, Boston, MA, USA</p>

<hr />

<h3 id="edger-empirical-analysis-of-dge-in-r">edgeR | empirical analysis of DGE in R</h3>

<p><p align="justify">Allows differential expression analysis of digital gene expression data. edgeR implements a range of statistical methodology based on the negative binomial distributions, including empirical Bayes estimation, exact tests, generalized linear models and quasi likelihood tests. The package and methods are general, and can work on other sources of count data, such as barcoding experiments and peptide counts.</p>

<p><a href="http://www.bioconductor.org/packages/release/bioc/html/edgeR.html" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://toolshed.g2.bx.psu.edu/repository?repository_id=0c447003da85ae19" target="_blank"><strong>Galaxy</strong></a></p>

<p><a href="http://www.bioconductor.org/packages/release/bioc/vignettes/edgeR/inst/doc/edgeRUsersGuide.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Robinson et al., 2010) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19910308" target="_blank">edgeR: a Bioconductor package for differential expression analysis of digital gene expression data</a>. Bioinformatics.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Cancer Program, Garvan Institute of Medical Research, Darlinghurst, NSW, Australia; Bioinformatics Division, The Walter and Eliza Hall Institute of Medical Research, Parkville, VIC, Australia</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-topology-enrichment-analysis-font"><font color=#CD5555 face="黑体">Topology enrichment analysis</font></h2>

<h3 id="david-database-for-annotation-visualization-and-integrated-discovery">DAVID | Database for Annotation, Visualization and Integrated Discovery</h3>

<p><p align="justify">Allows users to obtain biological features/meaning associated with large gene or protein lists. DAVID can determine gene-gene similarity, based on the assumption that genes sharing global functional annotation profiles are functionally related to each other. It groups related genes or terms into functional groups employing the similarity distances measure. This tool takes into account the redundant and network nature of biological annotation contents.</p>

<p><a href="https://david.ncifcrf.gov/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Jiao et al., 2012) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22543366" target="_blank">DAVID-WS: a stateful web service to facilitate gene/protein list analysis</a>.  Bioinformatics.</li>
<li>(Huang da et al., 2009) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19131956" target="_blank">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources</a>.  Nat Protoc. * (Huang da et al., 2009) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19033363" target="_blank">Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists</a>. Nucleic Acids Res.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Laboratory of Immunopathogenesis and Bioinformatics, Frederick, MD, USA; Advanced Biomedical Computing Center, Frederick, MD, USA</p>

<hr />

<h3 id="homer-hypergeometric-optimization-of-motif-enrichment">HOMER | Hypergeometric Optimization of Motif EnRichment</h3>

<p><p align="justify">Performs peak finding and downstream data analysis for next-generation sequencing analysis. HOMER affords several tools and methods to make use of ChIP-Seq, GRO-Seq, RNA-Seq, DNase-Seq, Hi-C and other types of functional genomics sequencing data sets. This software offers support to UCSC visualization, peaks annotation, quantification of transcripts and repeats or differential features, enrichment and expression.</p>

<p><a href="http://homer.ucsd.edu/homer/ngs/index.html" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://toolshed.g2.bx.psu.edu/repository?repository_id=aaa8f7315ce6dac2" target="_blank"><strong>Galaxy</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Heinz et al., 2010) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20513432" target="_blank">Simple combinations of lineage-determining transcription factors prime cis-regulatory elements required for macrophage and B cell identities</a>. Mol Cell.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Cellular and Molecular Medicine, University of California, San Diego, La Jolla, CA, USA; Department of Medicine, University of California, San Diego, La Jolla, CA, USA</p>

<hr />

<h3 id="genecodis-topology-enrichment-analysis">GeneCodis | Topology enrichment analysis</h3>

<p><p align="justify">A web-based tool for the ontological analysis of large lists of genes. It can be used to determine biological annotations or combinations of annotations that are significantly associated to a list of genes under study with respect to a reference list. As well as single annotations, this tool allows users to simultaneously evaluate annotations from different sources, for example Biological Process and Cellular Component categories of Gene Ontology.</p>

<p><a href="http://genecodis.cnb.csic.es/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Tabas-Madrid et al., 2012) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22573175" target="_blank">GeneCodis3: a non-redundant and modular enrichment analysis tool for functional genomics</a>.  Nucleic Acids Res.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Functional Bioinformatics Group, National Center for Biotechnology (CNB-CSIC), Madrid, Spain</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="bisulfite-sequencing-data-analysis-bs-seq-analysis">Bisulfite Sequencing Data Analysis (BS-seq analysis)</h1>

<p><p align="justify"><strong>DNA methylation</strong> contributes to the epigenetic regulation of many key developmental processes including genomic imprinting, X-inactivation, genome stability and gene regulation. Bisulfite conversion of genomic DNA combined with next-generation sequencing (BS-seq) is widely used to measure the methylation state of a whole genome, the methylome, at single-base resolution (Lister et al., 2009; Bock et al., 2010; Harris et al., 2010).</p>

<h2 id="font-color-cd5555-face-黑体-differential-methylation-region-detection-font-1"><font color=#CD5555 face="黑体">Differential Methylation region Detection</font></h2>

<h3 id="methylkit-methylation-annotation-bs-seq-analysis">methylKit | Methylation annotation : BS-seq analysis</h3>

<p><p align="justify">An R package for DNA methylation analysis and annotation from high-throughput bisulfite sequencing. methylKit is designed to deal with sequencing data from RRBS and its variants, but also target-capture methods such as Agilent SureSelect methyl-seq. In addition, methylKit can deal with base-pair resolution data for 5hmC obtained from Tab-seq or oxBS-seq. It can also handle whole-genome bisulfite sequencing data if proper input format is provided.</p>

<p><a href="https://github.com/al2na/methylKit" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Akalin et al., 2012) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23034086" target="_blank">methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles</a>.  Genome Biol.<br /></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Physiology and Biophysics, Weill Cornell Medical College, New York, NY, USA</p>

<hr />

<h3 id="smart-specific-methylation-analysis-and-report-tool">SMART | Specific Methylation Analysis and Report Tool</h3>

<p><p align="justify">Detects the cell type-specific methylation marks by integrating multiple methylomes from human cell lines and tissues. SMART is an entropy-based framework focused on integrating of a large number of DNA methylomes for the de novo identification of cell type-specific MethyMarks. To facilitate the specific methylation analysis, this method dynamically integrates multiple methylomes and identifies the cell type-specific methylation marks.</p>

<p><a href="http://fame.edbc.org/smart/" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://pypi.org/project/SMART-BS-Seq/" target="_blank"><strong>Issue</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Liu et al., 2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/26635396" target="_blank">Systematic identification and annotation of human methylation marks based on bisulfite sequencing methylomes reveals distinct roles of cell type-specific hypomethylation in the regulation of cell identity genes</a>. Nucleic Acids Res.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>College of Bioinformatics Science and Technology, Harbin Medical University, Harbin, China; Department of Rehabilitation, the First Affiliated Hospital of Harbin Medical University, Harbin, China</p>

<hr />

<h3 id="dss-single-differentially-methylated-region-detection-bs-seq-analysis">DSS-single | Differentially methylated region detection : BS-seq analysis</h3>

<p><p align="justify">A package based on a statistical method for detecting DMRs from WGBS (Whole Genome Bisulfite Sequencing) data without replicates. A key feature of DSS-single is to estimate biological variation when replicated data are not available. The method takes advantage of the spatial correlation of methylation levels: since the methylation levels from nearby CpG sites are similar, we can use nearby CpG sites as ‘pseudo-replicates’ to estimate dispersion. Simulations demonstrate that DSS-single has greater sensitivity and accuracy than existing methods, and an analysis of H1 versus IMR90 cell lines suggests that it also yields the most biologically meaningful results.</p>

<p><a href="http://www.bioconductor.org/packages/release/bioc/html/DSS.html" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="http://www.bioconductor.org/packages/release/bioc/manuals/DSS/man/DSS.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Feng et al., 2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24561809" target="_blank">A Bayesian hierarchical model to detect differentially methylated loci from single nucleotide resolution sequencing data</a>.  Nucleic Acids Res.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Biostatistics and Bioinformatics, Rollins School of Public Health, Emory University, Atlanta, GA, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-methylation-annotation-font"><font color=#CD5555 face="黑体">Methylation Annotation</font></h2>

<h3 id="gbsa-genome-bisulfite-sequencing-analyser">GBSA | Genome Bisulfite Sequencing Analyser</h3>

<p><p align="justify">An open-source software tool capable of analysing whole-genome bisulfite sequencing data with either a gene-centric or gene-independent focus. GBSA’s output can be easily integrated with other high-throughput sequencing data, such as RNA-Seq or ChIP-seq, to elucidate the role of methylated intergenic regions in gene regulation. In essence, GBSA allows an investigator to explore not only known loci but also all the genomic regions, for which methylation studies could lead to the discovery of new regulatory mechanisms.</p>

<p><a href="http://ctrad-csi.nus.edu.sg/gbsa/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Benoukraf et al., 2013) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23268441" target="_blank">GBSA: a comprehensive software for analysing whole genome bisulfite sequencing data</a>.  Nucleic Acids Res.<br /></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Cancer Science Institute of Singapore, National University of Singapore, Singapore, Singapore; Department of Pathology, National University of Singapore, Singapore, Singapore</p>

<hr />

<h3 id="methylkit-methylation-annotation-bs-seq-analysis-1">methylKit | Methylation annotation : BS-seq analysis</h3>

<p><p align="justify">An R package for DNA methylation analysis and annotation from high-throughput bisulfite sequencing. methylKit is designed to deal with sequencing data from RRBS and its variants, but also target-capture methods such as Agilent SureSelect methyl-seq. In addition, methylKit can deal with base-pair resolution data for 5hmC obtained from Tab-seq or oxBS-seq. It can also handle whole-genome bisulfite sequencing data if proper input format is provided.</p>

<p><a href="https://github.com/al2na/methylKit" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Akalin et al., 2012) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23034086" target="_blank">methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles</a>.  Genome Biol.<br /></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Physiology and Biophysics, Weill Cornell Medical College, New York, NY, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-ewas-font"><font color=#CD5555 face="黑体">EWAS</font></h2>

<h3 id="refactor-reference-free-adjustment-for-cell-type-composition-1">ReFACTor | Reference-Free Adjustment for Cell-Type composition</h3>

<p><p align="justify">A method based on principal component analysis (PCA) and designed for the correction of cell type heterogeneity in epigenome-wide association studies (EWAS). ReFACTor tool is based on a variant of PCA and can be applied to any tissue. It selects the sites that can be reconstructed with low error using a low-rank approximation of the original methylation matrix. Moreover, ReFACTor does not use the phenotype in the selection process, making ReFACTor useful as part of a quality control step in EWAS.</p>

<p><a href="http://www.cs.tau.ac.il/~heran/cozygene/software/refactor.html" target="_blank"><strong>Official Website</strong></a></p>

<p><a href="https://github.com/cozygene/refactor" target="_blank"><strong>Github</strong></a></p>

<p><a href="http://www.cs.tau.ac.il/~heran/cozygene/software/refactor_assets/README.pdf" target="_blank"><strong>Documentation</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Rahmani et al., 2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27018579" target="_blank">Sparse PCA corrects for cell type heterogeneity in epigenome-wide association studies</a>.  Nat Methods.
PMID: 27018579</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Blavatnik School of Computer Science, Tel-Aviv University, Tel Aviv, Israel; Department of Medicine, University of California, San Francisco, CA, USA</p>

<hr />

<h3 id="fast-lmm-ewasher-1">FaST-LMM-EWASher</h3>

<p><p align="justify">An R version of FaST-LMM-EWASher, which performs epigenome-wide association analysis in the presence of confounders such as cell-type heterogeneity. A python version of this software is also available as part of Fast-LMM-Py.</p>

<p><a href="https://www.microsoft.com/en-us/download/details.aspx?id=52501" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Zou et al., 2014) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24464286" target="_blank">Epigenome-wide association studies without the need for cell-type composition</a>.  Nat Methods.</li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>eScience Research Group, Microsoft Research, Los Angeles, CA, USA; The Broad Institute of MIT and Harvard, Cambridge, MA, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<h1 id="find-more-tools-omictools-https-omictools-com-epigenomics-category">Find more tools: <a href="https://omictools.com/epigenomics-category" target="_blank"><strong>OMICTOOLS</strong></a></h1>

<p><a href="https://www.the-scientist.com/?articles.view/articleNo/50203/title/The-Role-of-DNA-Base-Modifications/" target="_blank"><em>Image Citation</em></a></p>

<hr />

    </div>

    


<div class="article-tags">
  
  <a class="btn btn-primary btn-outline" href="/tags/epigenomics" target="_blank">epigenomics</a>
  
</div>




    
    
    <div class="article-widget">
      <div class="hr-light"></div>
      <h3>Related</h3>
      <ul>
        
        <li><a href="/too2/new_epi/position/">Nucleosome Positioning</a></li>
        
        <li><a href="/too2/new_epi/dna_pro/">DNA-Protein Interactions</a></li>
        
      </ul>
    </div>
    

    

    


  </div>
</article>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2018 &middot; 

      Copyright© 2018 Lin QI of Dr.Xin Wang's Lab. zzx. All rights reserved. 

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    <script async defer src="//maps.googleapis.com/maps/api/js?key=AIzaSyD3AGpKDAfo_LjBRHd-YkYmMjBDlIHfdlo"></script>
    

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gmaps.js/0.4.25/gmaps.min.js" integrity="sha256-7vjlAeb8OaTrCXZkCNun9djzuB2owUsaO72kXaFDBJs=" crossorigin="anonymous"></script>
    
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
      

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

