<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.38.2" />
  <meta name="author" content="Computational Biology Lab">

  
  
  
  
    
      
    
  
  <meta name="description" content="Table of Contents  Phylogenetic inference  MEGA ADAPTML  Phylogenetic Network Construction  CLC bio SPRDist  Discrete Character Evolution  CAFE DupliPHY  Tumor Progression  CT-CBN TRONCO  Tree Visualization  Dendroscope TreeKO  Find more tools: OMIVTOOLS   
In biology, phylogenetics is the study of the evolutionary history and relationships among individuals or groups of organisms (e.g. species, or populations). These relationships are discovered through phylogenetic inference methods that evaluate observed heritable traits, such as DNA sequences or morphology under a model of evolution of these traits.">

  
  <link rel="alternate" hreflang="en-us" href="/too2/new_geno/phylogenetic/">

  


  

  
  
  <meta name="theme-color" content="#3f51b5">
  
  
  
  
    
  
  
    
    
      
        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
      
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">
  
  
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7cMerriweather%7cRoboto&#43;Mono">
  
  <link rel="stylesheet" href="/styles.css">
  

  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/too2/new_geno/phylogenetic/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="og:site_name" content="CityU-Bioinformatics">
  <meta property="og:url" content="/too2/new_geno/phylogenetic/">
  <meta property="og:title" content="Phylogenetic Data Analysis | CityU-Bioinformatics">
  <meta property="og:description" content="Table of Contents  Phylogenetic inference  MEGA ADAPTML  Phylogenetic Network Construction  CLC bio SPRDist  Discrete Character Evolution  CAFE DupliPHY  Tumor Progression  CT-CBN TRONCO  Tree Visualization  Dendroscope TreeKO  Find more tools: OMIVTOOLS   
In biology, phylogenetics is the study of the evolutionary history and relationships among individuals or groups of organisms (e.g. species, or populations). These relationships are discovered through phylogenetic inference methods that evaluate observed heritable traits, such as DNA sequences or morphology under a model of evolution of these traits.">
  <meta property="og:locale" content="en-us">
  
  <meta property="article:published_time" content="2018-04-11T00:00:00&#43;00:00">
  
  <meta property="article:modified_time" content="2018-04-11T00:00:00&#43;00:00">
  

  

  <title>Phylogenetic Data Analysis | CityU-Bioinformatics</title>

</head>
<body id="top" data-spy="scroll" data-target="#toc" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/">CityU-Bioinformatics</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#bioinformatics">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/program/r">
            
            <span>R Programming</span>
            
          </a>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Course Tutorials</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/tutorial/bms4001/">
                
                <span>BMS4001 tutorials</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/bms8110/">
                
                <span>BMS8110 tutorials</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/tutorial/resources/">
            
            <span>Online Resources</span>
            
          </a>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Database</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/base2/dna">
                
                <span>DNA Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/express">
                
                <span>Genome Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/rna">
                
                <span>RNA Databses</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/protein">
                
                <span>Protein Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pheno">
                
                <span>Phenotype Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pathway">
                
                <span>Pathway Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/special">
                
                <span>Other Databases</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Tools</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/too2/genomics">
                
                <span>Genomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/epigenomics">
                
                <span>Epigenomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/transcriptomics">
                
                <span>Transcriptomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/proteomics">
                
                <span>Proteomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/metabolomics">
                
                <span>Metabolomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/phenomics">
                
                <span>Phenomics</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Journals</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/journal/comprehensive">
                
                <span>Multidisciplinary</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/specific">
                
                <span>Bioinformatics Specific</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/conferences">
                
                <span>Conferences</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/server/">
            
            <span>Servers</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>


<article class="article" itemscope itemtype="http://schema.org/Article">

  


  <div class="article-container">
    <h1 itemprop="name">Phylogenetic Data Analysis</h1>

    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-04-11 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Apr 11, 2018
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Computational Biology Lab">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    8 min read
  </span>
  

  
  

  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=Phylogenetic%20Data%20Analysis&amp;url=%2ftoo2%2fnew_geno%2fphylogenetic%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=%2ftoo2%2fnew_geno%2fphylogenetic%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-facebook"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=%2ftoo2%2fnew_geno%2fphylogenetic%2f&amp;title=Phylogenetic%20Data%20Analysis"
         target="_blank" rel="noopener">
        <i class="fa fa-linkedin"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=%2ftoo2%2fnew_geno%2fphylogenetic%2f&amp;title=Phylogenetic%20Data%20Analysis"
         target="_blank" rel="noopener">
        <i class="fa fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=Phylogenetic%20Data%20Analysis&amp;body=%2ftoo2%2fnew_geno%2fphylogenetic%2f">
        <i class="fa fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>


    <div class="article-style" itemprop="articleBody">
      

<hr />

<p><img src="/img/tools/geno/phylogenies.jpg" width="500", height="300" alt="dnastr" align="center"></p>

<h2>Table of Contents</h2>
<nav id="TableOfContents">
<ul>
<li><a href="#phylogenetic-inference">Phylogenetic inference</a>
<ul>
<li><a href="#mega">MEGA</a></li>
<li><a href="#adaptml">ADAPTML</a></li>
</ul></li>
<li><a href="#phylogenetic-network-construction">Phylogenetic Network Construction</a>
<ul>
<li><a href="#clc-bio">CLC bio</a></li>
<li><a href="#sprdist">SPRDist</a></li>
</ul></li>
<li><a href="#discrete-character-evolution">Discrete Character Evolution</a>
<ul>
<li><a href="#cafe">CAFE</a></li>
<li><a href="#dupliphy">DupliPHY</a></li>
</ul></li>
<li><a href="#tumor-progression">Tumor Progression</a>
<ul>
<li><a href="#ct-cbn">CT-CBN</a></li>
<li><a href="#tronco">TRONCO</a></li>
</ul></li>
<li><a href="#tree-visualization">Tree Visualization</a>
<ul>
<li><a href="#dendroscope">Dendroscope</a></li>
<li><a href="#treeko">TreeKO</a></li>
</ul></li>
<li><a href="#find-more-tools-omivtools-https-omictools-com-phylogenetics-and-phylogenomics-category">Find more tools: <a href="https://omictools.com/phylogenetics-and-phylogenomics-category" target="_blank"><strong>OMIVTOOLS</strong></a></a></li>
</ul>
</nav>


<p><span id="top"></span></p>

<p><p align="justify">In biology, phylogenetics is the study of the evolutionary history and relationships among individuals or groups of organisms (e.g. species, or populations). These relationships are discovered through phylogenetic inference methods that evaluate observed heritable traits, such as DNA sequences or morphology under a model of evolution of these traits. The result of these analyses is a phylogeny (also known as a phylogenetic tree) – a diagrammatic hypothesis about the history of the evolutionary relationships of a group of organisms.</p>

<p><p align="justify">Phylogenies are important for addressing various biological questions such as relationships among species or genes, the origin and spread of viral infection and the demographic changes and migration patterns of species. The advancement of sequencing technologies has taken phylogenetic analysis to a new height. Phylogenies have permeated nearly every branch of biology, and the plethora of phylogenetic methods and software packages that are now available may seem daunting to an experimental biologist.</p>

<p><a href="https://en.wikipedia.org/wiki/Phylogenetics" target="_blank"><strong>Wikipedia</strong></a></p>

<hr />

<h1 id="phylogenetic-inference">Phylogenetic inference</h1>

<p><p align="justify">The task of resolving the tree of life of extant species remains one of the grand challenges in evolutionary biology. As the number of trees grows superexponentially with the number of species for which an evolutionary tree is reconstructed, tree inference is considered a hard problem in computer science. The plethora of algorithmic challenges associated with phylogenetic trees and their efficient computation gave rise to the discipline of “phyloinformatics.”</p>

<hr />

<h2 id="mega">MEGA</h2>

<p><p align="justify">An integrated tool for conducting sequence alignment, inferring phylogenetic trees, estimating divergence times, mining online databases, estimating rates of molecular evolution, inferring ancestral sequences, and testing evolutionary hypotheses.</p>

<p><a href="https://www.megasoftware.net/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong></p>

<ul>
<li>(Kumar et al., 1994) <a href="https://www.ncbi.nlm.nih.gov/pubmed/8019868" target="_blank">MEGA: Molecular Evolutionary Genetics Analysis software for microcomputers</a>. Comput Appl Biosci.</li>
<li>(Kumar et al., 2001)<a href="https://www.ncbi.nlm.nih.gov/pubmed/11751241" target="_blank">MEGA2: molecular evolutionary genetics analysis software</a>.  Bioinformatics.</li>
<li>(Tamura et al., 2011) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21546353" target="_blank">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</a>. Mol Biol Evol.</li>
<li>(Kumar et al., 2016) <a href="https://www.ncbi.nlm.nih.gov/pubmed/27004904" target="_blank">MEGA7: Molecular Evolutionary Genetics Analysis Version 7.0 for Bigger Datasets</a>.  Mol Biol Evol.</li>
</ul>

<p><strong>Institution(s)</strong></p>

<p>Institute for Genomics and Evolutionary Medicine, Temple University Department of Biology, Temple University Center for Excellence in Genome Medicine and Research, King Abdulaziz University, Jeddah, Saudi Arabia.</p>

<hr />

<h2 id="adaptml">ADAPTML</h2>

<p><p align="justify">Allows identification of populations as groups of related strains sharing a common projected habitat, which reflects their relative abundance in the measured environmental categories. ADAPTML maps changes in environmental preference onto the tree by predicting projected habitats for each extant and ancestral strain in the phylogeny. It builds a hidden Markov model for the evolution of habitat associations.</p>

<p><a href="http://almlab.mit.edu/adaptml.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong></p>

<ul>
<li>(2008 Science) <a href="https://www.ncbi.nlm.nih.gov/pubmed/18497299" target="_blank">esource partitioning and sympatric differentiation among closely related bacterioplankton.</a></li>
</ul>

<p><strong>Institution(s)</strong></p>

<p>Department of Civil and Environmental Engineering, MA USA; Institute of Technology (MIT), Cambridge, MA, USA; Computational and Systems Biology Initiative, MIT, Cambridge, MA, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="phylogenetic-network-construction">Phylogenetic Network Construction</h1>

<p><p align="justify">The evolutionary history of a set of species is usually described by a rooted phylogenetic tree. Although it is generally undisputed that bifurcating speciation events and descent with modifications are major forces of evolution, there is a growing belief that reticulate events also have a role to play. Phylogenetic networks provide an alternative to phylogenetic trees and may be more suitable for data sets where evolution involves significant amounts of reticulate events, such as hybridization, horizontal gene transfer, or recombination.</p>

<hr />

<h2 id="clc-bio">CLC bio</h2>

<p><p align="justify">Allows to analyze, compare, and visualize next generation sequencing (NGS) data. CLC Genomics Workbench offers a complete and customizable solution for genomics, transcriptomics, epigenomics, and metagenomics. The software enables to generate custom workflows, which can combine quality control steps, adapter trimming, read mapping, variant detection, and multiple filtering and annotation steps into a pipeline.</p>

<p><a href="https://www.qiagenbioinformatics.com/products/clc-genomics-workbench/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Citation Publications</strong></p>

<ul>
<li>(2018 PMCID: 5959919) <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5959919" target="_blank">Phenotypic diversification by enhanced genome restructuring after induction of multiple DNA double-strand breaks</a></li>
</ul>

<p><strong>Documentation</strong></p>

<p><a href="http://resources.qiagenbioinformatics.com/manuals/clcgenomicsworkbench/current/User_Manual.pdf" target="_blank">Documentation</a></p>

<hr />

<h2 id="sprdist">SPRDist</h2>

<p><p align="justify">Solves the rooted Subtree Prune and Regraft (rSPR) distance problem for many datasets. SPRDist is a method that uses integer programming to resolve the problem of computing rSPR distances between two rooted binary trees. This application takes two binary phylogenetic trees and computes the smallest number of (SPR) operations that transform one tree to the other. It only uses a publicly available integer linear programming (ILP) solver.</p>

<p><a href="http://www.engr.uconn.edu/~ywu/SPRDist.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong></p>

<ul>
<li>(2009 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19019848" target="_blank">
A practical method for exact computation of subtree prune and regraft distance.</a></li>
</ul>

<p><strong>Documentation</strong></p>

<p><a href="http://www.engr.uconn.edu/~ywu/SPRDist/README.txt" target="_blank">Documentation</a></p>

<p><strong>Institution(s)</strong></p>

<p>Department of Computer Science and Engineering, University of Connecticut, Storrs, CT, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="discrete-character-evolution">Discrete Character Evolution</h1>

<p><p align="justify">The study of discrete characters is crucial for the understanding of evolutionary processes. Even though great advances have been made in the analysis of nucleotide sequences, computer programs for non-DNA discrete characters are often dedicated to specific analyses and lack flexibility. Discrete characters often have different transition rate matrices, variable rates among sites and sometimes contain unobservable states.</p>

<hr />

<h2 id="cafe">CAFE</h2>

<p><p align="justify">Enables the accurate estimation of rates of gene family evolution when there are errors in the observed gene family sizes. By allowing users to marginalize over the uncertainty in the observed gene family sizes, CAFE 3 provides a platform for expanding comparative genomic analyses into clades consisting solely of draft genome sequences.</p>

<p><a href="https://sourceforge.net/projects/cafehahnlab/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong></p>

<ul>
<li>(2013 Mol Biol Evol) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23709260" target="_blank">Estimating gene gain and loss rates in the presence of error in genome assembly and annotation using CAFE 3.</a></li>
</ul>

<p><strong>Documentation</strong></p>

<p><a href="https://sourceforge.net/p/cafehahnlab/wiki/Home/" target="_blank">Documentation</a></p>

<p><strong>Institution(s)</strong></p>

<p>K.U.Leuven, OKP Research GroupTiensestraat, Leuven, Belgium; Department of Statistics, U.C. Davis Kerr Hall, Davis, CA, USA</p>

<hr />

<h2 id="dupliphy">DupliPHY</h2>

<p><p align="justify">A software tool to determine the evolutionary histories of gene families over a phylogenetic tree. Given a set of gene family sizes and a phylogenetic tree DupliPHY will calculate the ancestral family sizes at each internal node within the tree. DupliPHY returns a list of ancestral family sizes and a phylogenetic tree for each family with the ancestral family sizes listed as internal node labels.</p>

<p><a href="http://www.bioinf.manchester.ac.uk/dupliphy/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong></p>

<ul>
<li>(2015 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25294920" target="_blank">DupliPHY-Web: a web server for DupliPHY and DupliPHY-ML</a></li>
</ul>

<p><strong>Institution(s)</strong></p>

<p>Faculty of Life Sciences, University of Manchester, Oxford Road, Manchester, UK</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="tumor-progression">Tumor Progression</h1>

<p>Cancer can be a result of accumulation of different types of genetic mutations such as copy number aberrations. The data from tumors are cross-sectional and do not contain the temporal order of the genetic events. Finding the order in which the genetic events have occurred and progression pathways are of vital importance in understanding the disease.</p>

<hr />

<h2 id="ct-cbn">CT-CBN</h2>

<p><p align="justify">A specific probabilistic graphical model for the accumulation of mutations and their interdependencies. The Bayesian network models cancer progression by an explicit unobservable accumulation process in time that is separated from the observable but error-prone detection of mutations. Model parameters are estimated by an expectation-maximization algorithm and the underlying interaction graph is obtained by a simulated annealing procedure.</p>

<p><a href="https://www.bsse.ethz.ch/cbg/software/ct-cbn.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong></p>

<ul>
<li><p>(2011 PLoS One) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22069497" target="_blank">The temporal order of genetic and pathway alterations in tumorigenesis.</a></p></li>

<li><p>(2009 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19692554" target="_blank">Quantifying cancer progression with conjunctive Bayesian networks</a></p></li>
</ul>

<p><strong>Institution(s)</strong></p>

<p>Department of Biosystems Science and Engineering, ETH Zurich, Basel, Switzerland; Department of Mathematics, North Carolina State University, Raleigh, NC, USA</p>

<hr />

<h2 id="tronco">TRONCO</h2>

<p><p align="justify">Infers cancer progression models from heterogeneous genomic data. TRONCO is an R package built to extract population-level models describing the trends of accumulation of alterations in a cohort of cross-sectional samples and individual-level models that reveal the clonal evolutionary history in single cancer patients, when multiple samples are available. It also implements an oncoprint system to visualize the processed data.</p>

<p><a href="https://sites.google.com/site/troncopackage/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong></p>

<ul>
<li>(2017 BioRxiv) <a href="https://doi.org/10.1101/132183" target="_blank">Learning mutational graphs of individual tumor evolution from multi-sample sequencing data.</a></li>
</ul>

<p><strong>Institution(s)</strong></p>

<p>Department of Pathology, Stanford University, CA, USA; Department of Informatics, Systems and Communication, University of Milan-Bicocca, Milan, Italy</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="tree-visualization">Tree Visualization</h1>

<p><p align="justify">Phylogenetic trees are pervasively used to depict evolutionary relationships. Increasingly, researchers need to visualize large trees and compare multiple large trees inferred for the same set of taxa (reflecting uncertainty in the tree inference or genuine discordance among the loci analysed). Existing tree visualisation tools are however not well suited to these tasks. In particular, side-by-side comparison of trees can prove challenging beyond a few dozen taxa</p>

<hr />

<h2 id="dendroscope">Dendroscope</h2>

<p><p align="justify">Allows users to draw and compare rooted phylogenetic networks. Dendroscope is a program that includes characteristics allowing researchers to treat realistic phylogenetic trees. It also provides an interactive platform for researchers to explore the application of rooted phylogenetic networks to their phylogenetic trees and data. Moreover, this tool is available with a graphical user interface (GUI) or a command line interface (CLI).</p>

<p><a href="http://dendroscope.org/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong></p>

<ul>
<li>(2012 Syst Biol) <a href="https://www.ncbi.nlm.nih.gov/pubmed/22780991" target="_blank">Dendroscope 3: an interactive tool for rooted phylogenetic trees and networks.</a></li>
</ul>

<p><strong>Documentation</strong></p>

<p><a href="http://ab.inf.uni-tuebingen.de/data/software/dendroscope3/download/manual.pdf" target="_blank">Documentation</a></p>

<p><strong>Institution(s)</strong></p>

<p>Center for Bioinformatics (ZBIT), University of Tubingen, Germany; Department of Computer Science at the University of Auckland, New Zealand</p>

<hr />

<h2 id="treeko">TreeKO</h2>

<p><p align="justify">Enables the comparison of tree topologies, even in the presence of duplication and loss events. To do so treeKO recursively splits gene trees into pruned trees containing only orthologs to subsequently compute a distance based on the combined analyses of all pruned tree comparisons. In addition, treeKO implements the possibility of computing phylome support values, and reconciliation-based measures such as the number of inferred duplication and loss events.</p>

<p><a href="http://treeko.cgenomics.org/treeko" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong></p>

<ul>
<li>(2011 Nucleic Acids Res) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21335609" target="_blank">TreeKO: a duplication-aware algorithm for the comparison of phylogenetic trees</a></li>
</ul>

<p><strong>Institution(s)</strong></p>

<p>Bioinformatics and Genomics Programme, Centre for Genomic Regulation (CRG), UPF, Barcelona, Spain</p>

<hr />

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<h1 id="find-more-tools-omivtools-https-omictools-com-phylogenetics-and-phylogenomics-category">Find more tools: <a href="https://omictools.com/phylogenetics-and-phylogenomics-category" target="_blank"><strong>OMIVTOOLS</strong></a></h1>

<p><a href="https://arthropoda.wordpress.com/2010/01/19/introduction-to-phylogenetics/" target="_blank"><em>Imaghe Citation</em></a></p>

<hr />

    </div>

    


<div class="article-tags">
  
  <a class="btn btn-primary btn-outline" href="/tags/genomics" target="_blank">genomics</a>
  
</div>




    
    
    <div class="article-widget">
      <div class="hr-light"></div>
      <h3>Related</h3>
      <ul>
        
        <li><a href="/too2/new_geno/dnastr/">DNA Structure</a></li>
        
        <li><a href="/too2/new_geno/anno/">Genome Annotation</a></li>
        
        <li><a href="/too2/new_geno/edit/">Genome Editing</a></li>
        
        <li><a href="/too2/new_geno/dna_pro/">DNA-Protein Interactions</a></li>
        
        <li><a href="/too2/new_geno/population/">Population Genomics</a></li>
        
      </ul>
    </div>
    

    

    


  </div>
</article>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2018 &middot; 

      Copyright© 2018 Lin QI of Dr.Xin Wang's Lab. zzx. All rights reserved. 

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    <script async defer src="//maps.googleapis.com/maps/api/js?key=AIzaSyD3AGpKDAfo_LjBRHd-YkYmMjBDlIHfdlo"></script>
    

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gmaps.js/0.4.25/gmaps.min.js" integrity="sha256-7vjlAeb8OaTrCXZkCNun9djzuB2owUsaO72kXaFDBJs=" crossorigin="anonymous"></script>
    
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
      

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

