<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.38.2" />
  <meta name="author" content="Computational Biology Lab">

  
  
  
  
    
      
    
  
  <meta name="description" content="Table of Contents  Connectivity Analysis  Functional Brain Connectivity  BCT | Brain Connectivity Toolbox WholeBrain  Structural Brain Connectivity  BNRR WholeBrain   Functional Magnetic Resonance Imaging (fMRI)  3D Image Surfacing  ANTs | Advanced Normalization Tools Surfing | 3D image surfacing   Diffusion Magnetic Resonance Imaging Analysis  Tractography  SlicerDMRI Dipy | Diffusion Imaging in Python   Neuron Tracing  NeurphologyJ | Neuron tracing : Bioimaging analysis Vaa3D | 3D Visualization-Assisted Analysis  Find more tools: OMICTOOLS   Connectivity Analysis The functional organization of the brain is characterized by segregation and integration of information being processed.">

  
  <link rel="alternate" hreflang="en-us" href="/too2/new_pheno/brain/">

  


  

  
  
  <meta name="theme-color" content="#3f51b5">
  
  
  
  
    
  
  
    
    
      
        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
      
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">
  
  
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7cMerriweather%7cRoboto&#43;Mono">
  
  <link rel="stylesheet" href="/styles.css">
  

  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/too2/new_pheno/brain/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="og:site_name" content="CityU-Bioinformatics">
  <meta property="og:url" content="/too2/new_pheno/brain/">
  <meta property="og:title" content="Brain Connectomics | CityU-Bioinformatics">
  <meta property="og:description" content="Table of Contents  Connectivity Analysis  Functional Brain Connectivity  BCT | Brain Connectivity Toolbox WholeBrain  Structural Brain Connectivity  BNRR WholeBrain   Functional Magnetic Resonance Imaging (fMRI)  3D Image Surfacing  ANTs | Advanced Normalization Tools Surfing | 3D image surfacing   Diffusion Magnetic Resonance Imaging Analysis  Tractography  SlicerDMRI Dipy | Diffusion Imaging in Python   Neuron Tracing  NeurphologyJ | Neuron tracing : Bioimaging analysis Vaa3D | 3D Visualization-Assisted Analysis  Find more tools: OMICTOOLS   Connectivity Analysis The functional organization of the brain is characterized by segregation and integration of information being processed.">
  <meta property="og:locale" content="en-us">
  
  <meta property="article:published_time" content="2018-04-11T00:00:00&#43;00:00">
  
  <meta property="article:modified_time" content="2018-04-11T00:00:00&#43;00:00">
  

  

  <title>Brain Connectomics | CityU-Bioinformatics</title>

</head>
<body id="top" data-spy="scroll" data-target="#toc" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/">CityU-Bioinformatics</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#bioinformatics">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Programming</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/program/r">
                
                <span>R Language</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Tutorials</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/tutorial/bms4001/">
                
                <span>BMS4001 tutorials</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/bms8110/">
                
                <span>BMS8110 tutorials</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/resources/">
                
                <span>Other Resources</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Database</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/base2/dna">
                
                <span>DNA Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/express">
                
                <span>Genome Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/rna">
                
                <span>RNA Databses</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/protein">
                
                <span>Protein Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pheno">
                
                <span>Phenotype Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pathway">
                
                <span>Pathway Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/special">
                
                <span>Other Databases</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Tools</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/too2/genomics">
                
                <span>Genomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/epigenomics">
                
                <span>Epigenomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/transcriptomics">
                
                <span>Transcriptomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/proteomics">
                
                <span>Proteomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/metabolomics">
                
                <span>Metabolomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/phenomics">
                
                <span>Phenomics</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Journals</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/journal/comprehensive">
                
                <span>Multidisciplinary</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/specific">
                
                <span>Bioinformatics Specific</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/conferences">
                
                <span>Conferences</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/server/">
            
            <span>Servers</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>


<article class="article" itemscope itemtype="http://schema.org/Article">

  


  <div class="article-container">
    <h1 itemprop="name">Brain Connectomics</h1>

    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-04-11 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Apr 11, 2018
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Computational Biology Lab">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    8 min read
  </span>
  

  
  

  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=Brain%20Connectomics&amp;url=%2ftoo2%2fnew_pheno%2fbrain%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=%2ftoo2%2fnew_pheno%2fbrain%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-facebook"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=%2ftoo2%2fnew_pheno%2fbrain%2f&amp;title=Brain%20Connectomics"
         target="_blank" rel="noopener">
        <i class="fa fa-linkedin"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=%2ftoo2%2fnew_pheno%2fbrain%2f&amp;title=Brain%20Connectomics"
         target="_blank" rel="noopener">
        <i class="fa fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=Brain%20Connectomics&amp;body=%2ftoo2%2fnew_pheno%2fbrain%2f">
        <i class="fa fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>


    <div class="article-style" itemprop="articleBody">
      

<hr />

<p><img src="/img/tools/pheno/brain.jpg"  width="500" height="400" alt="anno" align="center"></p>

<p><span id="top"></span></p>

<h2>Table of Contents</h2>
<nav id="TableOfContents">
<ul>
<li><a href="#connectivity-analysis">Connectivity Analysis</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-functional-brain-connectivity-font"><font color=#CD5555 face="黑体">Functional Brain Connectivity</font></a>
<ul>
<li><a href="#bct-brain-connectivity-toolbox">BCT | Brain Connectivity Toolbox</a></li>
<li><a href="#wholebrain">WholeBrain</a></li>
</ul></li>
<li><a href="#font-color-cd5555-face-黑体-structural-brain-connectivity-font"><font color=#CD5555 face="黑体">Structural Brain Connectivity</font></a>
<ul>
<li><a href="#bnrr">BNRR</a></li>
<li><a href="#wholebrain-1">WholeBrain</a></li>
</ul></li>
</ul></li>
<li><a href="#functional-magnetic-resonance-imaging-fmri">Functional Magnetic Resonance Imaging (fMRI)</a>
<ul>
<li><a href="#3d-image-surfacing">3D Image Surfacing</a>
<ul>
<li><a href="#ants-advanced-normalization-tools">ANTs | Advanced Normalization Tools</a></li>
<li><a href="#surfing-3d-image-surfacing">Surfing | 3D image surfacing</a></li>
</ul></li>
</ul></li>
<li><a href="#diffusion-magnetic-resonance-imaging-analysis">Diffusion Magnetic Resonance Imaging Analysis</a>
<ul>
<li><a href="#font-color-cd5555-face-黑体-tractography-font"><font color=#CD5555 face="黑体">Tractography</font></a>
<ul>
<li><a href="#slicerdmri">SlicerDMRI</a></li>
<li><a href="#dipy-diffusion-imaging-in-python">Dipy | Diffusion Imaging in Python</a></li>
</ul></li>
</ul></li>
<li><a href="#neuron-tracing">Neuron Tracing</a>
<ul>
<li><a href="#neurphologyj-neuron-tracing-bioimaging-analysis">NeurphologyJ | Neuron tracing : Bioimaging analysis</a></li>
<li><a href="#vaa3d-3d-visualization-assisted-analysis">Vaa3D | 3D Visualization-Assisted Analysis</a></li>
</ul></li>
<li><a href="#find-more-tools-omictools-https-omictools-com-phenomics-category">Find more tools: <a href="https://omictools.com/phenomics-category" target="_blank"><strong>OMICTOOLS</strong></a></a></li>
</ul>
</nav>


<h1 id="connectivity-analysis">Connectivity Analysis</h1>

<p><p align="justify">The functional organization of the <strong>brain</strong> is characterized by segregation and integration of information being processed. A central paradigm in modern neuroscience is that anatomical and functional connections between brain regions are organized in a way such that information processing is near optimal. Functional interactions seem to be provided by synchronized activity, both locally and between distant brain regions. Brain networks thus consist of spatially distributed but functionally connected regions that process information. <a href="https://www.hindawi.com/journals/cin/2012/412512/" target="_blank">Source</a></p>

<h2 id="font-color-cd5555-face-黑体-functional-brain-connectivity-font"><font color=#CD5555 face="黑体">Functional Brain Connectivity</font></h2>

<p><p align="justify">Functional connectivity (FC) which is defined as the temporal dependency of neuronal activation patterns of anatomically separated brain regions. It reflects statistical dependencies between distinct and distant regions of information processing neuronal populations. Hence, it is basically a statistical concept which relies on such statistical measures as correlation, covariance, spectral coherence, or phase locking. Statistical dependencies are highly time dependent and fluctuate on multiple time scales ranging form milliseconds to seconds.</p>

<hr />

<h3 id="bct-brain-connectivity-toolbox">BCT | Brain Connectivity Toolbox</h3>

<p><p align="justify">A Matlab toolbox for analysis of complex brain networks. The toolbox contains network analysis functions and complex brain network datasets. These measures are increasingly used to characterize structural and functional brain connectivity datasets.</p>

<p><a href="https://sites.google.com/site/bctnet/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2010 Neuroimage) <a href="https://www.ncbi.nlm.nih.gov/pubmed/19819337" target="_blank">Complex network measures of brain connectivity: uses and interpretations.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Black Dog Institute and School of Psychiatry, University of New South Wales, Sydney, Australia;</p>

<p>Mental Health Research Division, Queensland Institute of Medical Research, Brisbane, Australia</p>

<hr />

<h3 id="wholebrain">WholeBrain</h3>

<p><p align="justify">Allows users to analyze, visualize, annotate and share whole-brain data at cellular resolution. WholeBrain supplies a method to generate brain maps containing data from neuron function, neuron identity, and connectivity. It quantifies and spatially maps multidimensional data from whole-brain experiments. It also compares results across experiments in a single standardized anatomical reference atlas.</p>

<p><a href="https://github.com/tractatus/wholebrain" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Nat Neurosci) <a href="https://doi.org/10.1038/s41593-017-0027-7" target="_blank">An interactive framework for whole-brain maps at cellular resolution.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Neuroscience, Karolinska Institutet, Stockholm, Sweden;</p>

<p>Department of Neuroscience, Scripps Research Institute, Jupiter, FL, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h2 id="font-color-cd5555-face-黑体-structural-brain-connectivity-font"><font color=#CD5555 face="黑体">Structural Brain Connectivity</font></h2>

<p><p align="justify">Anatomical connectivity (AC), also called structural connectivity, which forms the connectome through synaptic contacts between neighboring neurons or fiber tracks connecting neuron pools in spatially distant brain regions. The whole set of such fiber tracks in the brain is called white matter. On short time scales (sec, min), anatomical connections are quite persistent and stable, while for longer time spans substantial plasticity may be observed.</p>

<hr />

<h3 id="bnrr">BNRR</h3>

<p><p align="justify">Infers flexibly changes in a network-valued random variable with a continuous trait. BNRR is a network–response regression model, which considers the brain network as an object-type response variable having conditional expectation changing flexibly. This method can be easily adapted to incorporate directed networks via two subsets of latent coordinates—for each brain region—modeling outgoing and incoming edges, respectively.</p>

<p><a href="https://github.com/wangronglu/BNRR" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Bioinformatics) <a href="https://academic.oup.com/bioinformatics/article/33/12/1859/2971437" target="_blank">Bayesian network–response regression.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Statistical Science, Duke University, Durham, NC, USA;</p>

<p>Department of Statistical Sciences, University of Padova, Padova, Italy</p>

<hr />

<h3 id="wholebrain-1">WholeBrain</h3>

<p><p align="justify">Allows users to analyze, visualize, annotate and share whole-brain data at cellular resolution. WholeBrain supplies a method to generate brain maps containing data from neuron function, neuron identity, and connectivity. It quantifies and spatially maps multidimensional data from whole-brain experiments. It also compares results across experiments in a single standardized anatomical reference atlas.</p>

<p><a href="https://github.com/tractatus/wholebrain" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 Nat Neurosci) <a href="https://doi.org/10.1038/s41593-017-0027-7" target="_blank">An interactive framework for whole-brain maps at cellular resolution.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Neuroscience, Karolinska Institutet, Stockholm, Sweden;</p>

<p>Department of Neuroscience, Scripps Research Institute, Jupiter, FL, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="functional-magnetic-resonance-imaging-fmri">Functional Magnetic Resonance Imaging (fMRI)</h1>

<p><p align="justify">The relationship between <strong>brain structure and function</strong> has been one of the centers of research in neuroimaging for decades. In recent years, diffusion tensor imaging (DTI) and functional magnetic resonance imaging (fMRI) techniques have been widely available and popular in cognitive and clinical neurosciences for examining the brain&rsquo;s white matter (WM) micro-structures and gray matter (GM) functions, respectively. <a href="https://www.ncbi.nlm.nih.gov/pubmed/24103849" target="_blank">Source</a></p>

<p>Image preprocessing, processing, 3D Image Analysis could be found in <a href="/too2/new_pheno/bioimaging/" target="_blank"><strong>Bioimaging:</strong></a></p>

<h2 id="3d-image-surfacing">3D Image Surfacing</h2>

<p><p align="justify">The recent advent of increasingly affordable and powerful 3D scanning devices capable of capturing high resolution range data about real-world objects and environments has fueled research into effective 3D surface reconstruction techniques for rendering the raw point cloud data produced by many of these devices into a form that would make it usable in a variety of application domains.</p>

<hr />

<h3 id="ants-advanced-normalization-tools">ANTs | Advanced Normalization Tools</h3>

<p><p align="justify">Computes high-dimensional mappings to capture the statistics of brain structure and function. ANTs allows users to organize, visualize and statistically explore large biomedical image sets. It integrates imaging modalities and related information in space and time, and works across species or organ systems with minimal customization. ANTs depends on the Insight ToolKit (ITK), a widely used medical image processing library to which ANTs developers contribute. ANTs can be used paired with ANTsR, an emerging tool supporting standardized multimodality image analysis. ANTs is popularly considered a state-of-the-art medical image registration and segmentation toolkit.</p>

<p><a href="http://stnava.github.io/ANTs/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2011 Neuroimage) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20851191" target="_blank">A reproducible evaluation of ANTs similarity metric performance in brain image registration.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Penn Image Computing and Science Laboratory, University of Pennsylvania, Philadelphia, PA, USA</p>

<hr />

<h3 id="surfing-3d-image-surfacing">Surfing | 3D image surfacing</h3>

<p><p align="justify">Guides voxel selection for information mapping by using a cortical surface reconstruction. Surfing uses only voxels that are classified as grey matter based on an anatomical scan. It uses a surface-based geodesic distance metric to define neighbourhoods of voxels, and does not select voxels across a sulcus. The software employs a distance metric that takes into account the folded nature of the cortical sheet.</p>

<p><a href="https://github.com/nno/surfing" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2011 Neuroimage) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20621701" target="_blank">A comparison of volume-based and surface-based multi-voxel pattern analysis.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>School of Psychology, Bangor University, UK; Institute of Cognitive Neuroscience, University College London, UK</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="diffusion-magnetic-resonance-imaging-analysis">Diffusion Magnetic Resonance Imaging Analysis</h1>

<p><p align="justify">The relationship between brain structure and function has been one of the centers of research in neuroimaging for decades. In recent years, diffusion tensor imaging (DTI) and functional magnetic resonance imaging (fMRI) techniques have been widely available and popular in cognitive and clinical neurosciences for examining the brain&rsquo;s white matter (WM) micro-structures and gray matter (GM) functions, respectively.</p>

<hr />

<h2 id="font-color-cd5555-face-黑体-tractography-font"><font color=#CD5555 face="黑体">Tractography</font></h2>

<p><p align="justify">Tractography (or fiber tracking) consists of three-dimensional modeling of the preferential movement of water molecules in the form of fiber tracks from the tensor field information. This technique allows a new approach for the microarchitectural analysis of anisotropic structures such as nerves, white matter, and muscles. Many disorders have been studied including cervical myelopathy, carpal tunnel syndrome, nerve root compression, and nerve tumors.</p>

<hr />

<h3 id="slicerdmri">SlicerDMRI</h3>

<p><p align="justify">Allows diffusion of imaging research in a clinical oncology setting and provides tools for end-to-end diffusion image analysis as well as interoperation with clinical imaging systems. SlicerDMRI is a suite of open-source software tools for diffusion magnetic resonance imaging (dMRI) research. The software is built upon and integrated with 3D Slicer, an NIH-supported open-source platform for medical image computing. SlicerDMRI is used for both neuroscience research and cancer imaging research.</p>

<p><a href="http://dmri.slicer.org/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(Cancer Res) <a href="http://cancerres.aacrjournals.org/content/77/21/e101" target="_blank">SlicerDMRI: Open Source Diffusion MRI Software for Brain Cancer Research.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Brigham &amp; Women&rsquo;s Hospital and Harvard Medical School, Boston, MA, USA;</p>

<p>Isomics, Inc., Cambridge, MA, USA</p>

<hr />

<h3 id="dipy-diffusion-imaging-in-python">Dipy | Diffusion Imaging in Python</h3>

<p><p align="justify">Allows to study diffusion Magnetic Resonance Imaging (MRI) data. Dipy is a program allowing users to share their code and experiments. One of its objectives is to provide transparent implementations for all the different steps of the dMRI analysis with a uniform programming interface. It implements two interfaces for probabilistic Markov fiber tracking: (1) it allows the user to provide the distribution evaluated on a discrete set of possible tracking directions, and (2) it accommodates tracking methods where the fiber orientation distribution function (fODF) cannot be easily computed.</p>

<p><a href="http://nipy.org/dipy/cite.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2014 Front Neuroinform) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24600385" target="_blank">Dipy, a library for the analysis of diffusion MRI data.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Wolfson College, University of Cambridge, Cambridge, UK;</p>

<p>Medical Research Council Cognition and Brain Sciences Unit, Cambridge, UK</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="neuron-tracing">Neuron Tracing</h1>

<p><p align="justify">The reconstruction of neuron morphology allows to investigate how the brain works, which is one of the foremost challenges in neuroscience. This process aims at extracting the neuronal structures from microscopic imaging data. The great advances in microscopic technologies have made a huge amount of data available at the micro-, or even lower, resolution where manual inspection is time consuming, prone to error and utterly impractical. This has motivated the development of methods to automatically trace the neuronal structures, a task also known as neuron tracing.</p>

<hr />

<h2 id="neurphologyj-neuron-tracing-bioimaging-analysis">NeurphologyJ | Neuron tracing : Bioimaging analysis</h2>

<p><p align="justify">A quantification method capable of automatically quantifying neuronal morphologies such as soma number and size, neurite length, and neurite branching complexity (which is highly related to the numbers of attachment points and ending points). NeurphologyJ is implemented as a plugin to ImageJ, an open-source Java-based image processing and analysis platform.</p>

<p><a href="http://life.nctu.edu.tw/~ehwang/neurphologyJ.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2011 BMC Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/21651810" target="_blank">NeurphologyJ: an automatic neuronal morphology quantification method and its application in pharmacological discovery.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Institute of Bioinformatics and Systems Biology, National Chiao Tung University, Hsinchu, Taiwan</p>

<hr />

<h2 id="vaa3d-3d-visualization-assisted-analysis">Vaa3D | 3D Visualization-Assisted Analysis</h2>

<p><p align="justify">Allows users to visualize 3D/4D/5D image and to analyze system for bioimages and surface objects. Vaa3D consists of a container of modules for 3D image analysis (cell segmentation, neuron tracing, brain registration, annotation, quantitative measurement and statistics) and data management. Moreover, this platform can be used for developing 3D image analysis algorithms for high-throughput processing.</p>

<p><a href="http://home.penglab.com/proj/vaa3d/home/index.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li><p>(2018 BioRxiv) <a href="https://doi.org/10.1101/254318" target="_blank">DeepNeuron: An Open Deep Learning Toolbox for Neuron Tracing.</a></p></li>

<li><p>(2018 BioRxiv) <a href="https://www.biorxiv.org/content/early/2018/01/26/254425" target="_blank">Exploring morphological motifs for a single neuron based on multiple 3D reconstructions.</a></p></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Allen Institute for Brain Science, Seattle, WA, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="find-more-tools-omictools-https-omictools-com-phenomics-category">Find more tools: <a href="https://omictools.com/phenomics-category" target="_blank"><strong>OMICTOOLS</strong></a></h1>

<p><a href="http://blog.myesr.org/mri-reveals-the-human-connectome/" target="_blank"><em>Image Citation1</em></a><a href="https://www.moshimoshi-nippon.jp/zh-hant/80554" target="_blank"><em>2</em></a></p>

<hr />

    </div>

    


<div class="article-tags">
  
  <a class="btn btn-primary btn-outline" href="/tags/phenomics" target="_blank">phenomics</a>
  
</div>




    
    
    <div class="article-widget">
      <div class="hr-light"></div>
      <h3>Related</h3>
      <ul>
        
        <li><a href="/too2/new_pheno/neuroinformatics/">Computational Neuroinformatics</a></li>
        
        <li><a href="/too2/new_pheno/bioimaging/">Bioimaging</a></li>
        
        <li><a href="/too2/new_pheno/neurophysiology/">Neorophysiology</a></li>
        
        <li><a href="/too2/new_pheno/medicalimage/">Medical Imaging</a></li>
        
      </ul>
    </div>
    

    

    


  </div>
</article>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2018 &middot; 

      Copyright© 2018 Lin QI of Dr.Xin Wang's Lab. zzx. All rights reserved. 

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    <script async defer src="//maps.googleapis.com/maps/api/js?key=AIzaSyD3AGpKDAfo_LjBRHd-YkYmMjBDlIHfdlo"></script>
    

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gmaps.js/0.4.25/gmaps.min.js" integrity="sha256-7vjlAeb8OaTrCXZkCNun9djzuB2owUsaO72kXaFDBJs=" crossorigin="anonymous"></script>
    
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
      

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

