<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="theme" content="hugo-academic">
  <meta name="generator" content="Hugo 0.38.2" />
  <meta name="author" content="Computational Biology Lab">

  
  
  
  
    
      
    
  
  <meta name="description" content="Table of Contents  Lipidomics Databases  LIPID MAPS | LIPID Metabolites And Pathways Strategy LipidBlast | Lipidomic spectral libraries : Lipidomics databases  MS-based untargeted lipidomics  LipidMatch | Lipid identification : MS-based untargeted lipidomics Quantinetix | Image signal quantification : Mass spectrometry  MS-based targeted lipidomics  MRMPROBS | Multiple Reaction Monitoring-based PROBabilistic System MRM-DIFF | Multiple Reaction Monitoring-based DIFFerential  Find more tools: OMICTOOLS   Lipidomics or profiling specific classes of lipids biomarkers also provides a unique opportunity to identify diseases other than cancer.">

  
  <link rel="alternate" hreflang="en-us" href="/too2/new_meta/lipidomics/">

  


  

  
  
  <meta name="theme-color" content="#3f51b5">
  
  
  
  
    
  
  
    
    
      
        <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">
      
    
  
  
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha512-6MXa8B6uaO18Hid6blRMetEIoPqHf7Ux1tnyIQdpt9qI5OACx7C+O3IVTr98vwGnlcg0LOLa02i9Y1HpVhlfiw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.1/css/academicons.min.css" integrity="sha512-NThgw3XKQ1absAahW6to7Ey42uycrVvfNfyjqcFNgCmOCQ5AR4AO0SiXrN+8ZtYeappp56lk1WtvjVmEa+VR6A==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw==" crossorigin="anonymous">
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">
  
  
  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Lato:400,700%7cMerriweather%7cRoboto&#43;Mono">
  
  <link rel="stylesheet" href="/styles.css">
  

  

  
  <link rel="alternate" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  <link rel="feed" href="/index.xml" type="application/rss+xml" title="CityU-Bioinformatics">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="/too2/new_meta/lipidomics/">

  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="og:site_name" content="CityU-Bioinformatics">
  <meta property="og:url" content="/too2/new_meta/lipidomics/">
  <meta property="og:title" content="Lipidomics | CityU-Bioinformatics">
  <meta property="og:description" content="Table of Contents  Lipidomics Databases  LIPID MAPS | LIPID Metabolites And Pathways Strategy LipidBlast | Lipidomic spectral libraries : Lipidomics databases  MS-based untargeted lipidomics  LipidMatch | Lipid identification : MS-based untargeted lipidomics Quantinetix | Image signal quantification : Mass spectrometry  MS-based targeted lipidomics  MRMPROBS | Multiple Reaction Monitoring-based PROBabilistic System MRM-DIFF | Multiple Reaction Monitoring-based DIFFerential  Find more tools: OMICTOOLS   Lipidomics or profiling specific classes of lipids biomarkers also provides a unique opportunity to identify diseases other than cancer.">
  <meta property="og:locale" content="en-us">
  
  <meta property="article:published_time" content="2018-04-11T00:00:00&#43;00:00">
  
  <meta property="article:modified_time" content="2018-04-11T00:00:00&#43;00:00">
  

  

  <title>Lipidomics | CityU-Bioinformatics</title>

</head>
<body id="top" data-spy="scroll" data-target="#toc" data-offset="71" >

<nav class="navbar navbar-default navbar-fixed-top" id="navbar-main">
  <div class="container">

    
    <div class="navbar-header">
      
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse"
              data-target=".navbar-collapse" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
      <a class="navbar-brand" href="/">CityU-Bioinformatics</a>
    </div>

    
    <div class="collapse navbar-collapse">

      
      
      <ul class="nav navbar-nav navbar-right">
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#bioinformatics">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/program/r">
            
            <span>R Programming</span>
            
          </a>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Course Tutorials</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/tutorial/bms4001/">
                
                <span>BMS4001 tutorials</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/tutorial/bms8110/">
                
                <span>BMS8110 tutorials</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/tutorial/resources/">
            
            <span>Online Resources</span>
            
          </a>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Database</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/base2/dna">
                
                <span>DNA Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/express">
                
                <span>Genome Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/rna">
                
                <span>RNA Databses</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/protein">
                
                <span>Protein Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pheno">
                
                <span>Phenotype Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/pathway">
                
                <span>Pathway Databases</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/base2/special">
                
                <span>Other Databases</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Tools</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/too2/genomics">
                
                <span>Genomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/epigenomics">
                
                <span>Epigenomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/transcriptomics">
                
                <span>Transcriptomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/proteomics">
                
                <span>Proteomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/metabolomics">
                
                <span>Metabolomics</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/too2/phenomics">
                
                <span>Phenomics</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        <li class="dropdown">
          <a href="#" class="dropdown-toggle" data-toggle="dropdown" aria-haspopup="true">
            
            <span>Journals</span>
            
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            
            <li class="nav-item">
              <a href="/journal/comprehensive">
                
                <span>Multidisciplinary</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/specific">
                
                <span>Bioinformatics Specific</span>
                
              </a>
            </li>
            
            <li class="nav-item">
              <a href="/journal/conferences">
                
                <span>Conferences</span>
                
              </a>
            </li>
            
          </ul>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/server/">
            
            <span>Servers</span>
            
          </a>
        </li>

        
        

        
        
        
        
        
          
        

        <li class="nav-item">
          <a href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        
      

      
      </ul>

    </div>
  </div>
</nav>


<article class="article" itemscope itemtype="http://schema.org/Article">

  


  <div class="article-container">
    <h1 itemprop="name">Lipidomics</h1>

    

<div class="article-metadata">

  <span class="article-date">
    
    <time datetime="2018-04-11 00:00:00 &#43;0000 UTC" itemprop="datePublished dateModified">
      Apr 11, 2018
    </time>
  </span>
  <span itemscope itemprop="author publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Computational Biology Lab">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    5 min read
  </span>
  

  
  

  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=Lipidomics&amp;url=%2ftoo2%2fnew_meta%2flipidomics%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=%2ftoo2%2fnew_meta%2flipidomics%2f"
         target="_blank" rel="noopener">
        <i class="fa fa-facebook"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=%2ftoo2%2fnew_meta%2flipidomics%2f&amp;title=Lipidomics"
         target="_blank" rel="noopener">
        <i class="fa fa-linkedin"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=%2ftoo2%2fnew_meta%2flipidomics%2f&amp;title=Lipidomics"
         target="_blank" rel="noopener">
        <i class="fa fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=Lipidomics&amp;body=%2ftoo2%2fnew_meta%2flipidomics%2f">
        <i class="fa fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>


    <div class="article-style" itemprop="articleBody">
      

<hr />

<p><img src="/img/tools/meta/lipidomics.jpg"  width="500" height="400" alt="anno" align="center"></p>

<p><span id="top"></span></p>

<h2>Table of Contents</h2>
<nav id="TableOfContents">
<ul>
<li><a href="#lipidomics-databases">Lipidomics Databases</a>
<ul>
<li><a href="#lipid-maps-lipid-metabolites-and-pathways-strategy">LIPID MAPS | LIPID Metabolites And Pathways Strategy</a></li>
<li><a href="#lipidblast-lipidomic-spectral-libraries-lipidomics-databases">LipidBlast | Lipidomic spectral libraries : Lipidomics databases</a></li>
</ul></li>
<li><a href="#ms-based-untargeted-lipidomics">MS-based untargeted lipidomics</a>
<ul>
<li><a href="#lipidmatch-lipid-identification-ms-based-untargeted-lipidomics">LipidMatch | Lipid identification : MS-based untargeted lipidomics</a></li>
<li><a href="#quantinetix-image-signal-quantification-mass-spectrometry">Quantinetix | Image signal quantification : Mass spectrometry</a></li>
</ul></li>
<li><a href="#ms-based-targeted-lipidomics">MS-based targeted lipidomics</a>
<ul>
<li><a href="#mrmprobs-multiple-reaction-monitoring-based-probabilistic-system">MRMPROBS | Multiple Reaction Monitoring-based PROBabilistic System</a></li>
<li><a href="#mrm-diff-multiple-reaction-monitoring-based-differential">MRM-DIFF | Multiple Reaction Monitoring-based DIFFerential</a></li>
</ul></li>
<li><a href="#find-more-tools-omictools-https-omictools-com-metabolomics-category">Find more tools: <a href="https://omictools.com/metabolomics-category" target="_blank"><strong>OMICTOOLS</strong></a></a></li>
</ul>
</nav>


<p><p align="justify">Lipidomics or profiling specific classes of lipids biomarkers also provides a unique opportunity to identify diseases other than cancer. Biolipomics describes profiling classes of lipidomic analytes that are bioactive and may be a more direct analysis of functional lipomics.</p>

<hr />

<h1 id="lipidomics-databases">Lipidomics Databases</h1>

<p><p align="justify">Development of databases and the related bioinformatics tools has become an essential part of the omics studies. Over the recent years, empowered by high-throughput technologies for omics fields, the creation of databases devoted to lipids has been undertaken. Consequently, lipid-centered databases have been developed that enable researchers to comfortably analyze lipid species.</p>

<hr />

<h2 id="lipid-maps-lipid-metabolites-and-pathways-strategy">LIPID MAPS | LIPID Metabolites And Pathways Strategy</h2>

<p><p align="justify">Organizes lipids into eight well-defined categories that cover eukaryotic and prokaryotic sources. LIPID MAPS filters data in functional hierarchies involving lipids, reactions, and pathways. It is a multi-institutional effort created in 2003 and uses a systems biology approach and sophisticated mass spectrometers (MS), all of the major — and many minor — lipid species in mammalian cells. The database quantitates the changes in species in response to perturbation.</p>

<p><a href="http://www.lipidmaps.org/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2012 J Chem Educ) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24764601" target="_blank">LIPID MAPS-Nature Lipidomics Gateway: An Online Resource for Students and Educators Interested in Lipids.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>University of California, San Diego, La Jolla, CA, USA</p>

<hr />

<h2 id="lipidblast-lipidomic-spectral-libraries-lipidomics-databases">LipidBlast | Lipidomic spectral libraries : Lipidomics databases</h2>

<p><p align="justify">Current tandem mass spectral libraries for lipid annotations in metabolomics are limited in size and diversity. LipidBlast is a freely available computer-generated tandem mass spectral library of 212,516 spectra covering 119,200 compounds from 26 lipid compound classes, including phospholipids, glycerolipids, bacterial lipoglycans and plant glycolipids.</p>

<p><a href="http://fiehnlab.ucdavis.edu/projects/LipidBlast" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2013 Nat Methods) <a href="https://www.ncbi.nlm.nih.gov/pubmed/23817071" target="_blank">LipidBlast in silico tandem mass spectrometry database for lipid identification.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Metabolics Core, UC Davis Genome Center, University of California, Davis, Davis, CA, USA</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="ms-based-untargeted-lipidomics">MS-based untargeted lipidomics</h1>

<p><p align="justify">Due to its sensitivity and selectivity <strong>mass spectrometry</strong> (MS) is the method of choice for qualitative and quantitative lipidomic analysis. Although it is not yet possible to detect and quantify all individual lipids in a given cellular system, the aim of lipidomic analysis is to determine as many individual lipids as possible.</p>

<hr />

<h2 id="lipidmatch-lipid-identification-ms-based-untargeted-lipidomics">LipidMatch | Lipid identification : MS-based untargeted lipidomics</h2>

<p><p align="justify">Allows annotation of lipids across a wide range of high resolution tandem mass spectrometry (MS/MS) studies. LipidMatch is based on an accurate assumption of multiple co-eluting lipids sharing m/z values. It was evaluated using data acquired from a Q-Exactive orbitrap mass spectrometer and an Agilent 6540 Q-TOF. The tool contains a large diversity in lipid types of any current open-source software platform and a rule-based strategy for identification and summed fragment intensity based strategy for ranking top hits.</p>

<p><a href="http://secim.ufl.edu/secim-tools/lipidmatch/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2017 BMC Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/28693421" target="_blank">LipidMatch: an automated workflow for rule-based lipid identification using untargeted high-resolution tandem mass spectrometry data.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Department of Chemistry, University of Florida, Gainesville, FL, USA;</p>

<p>College of Engineering, University of Florida, Gainesville, FL, USA</p>

<hr />

<h2 id="quantinetix-image-signal-quantification-mass-spectrometry">Quantinetix | Image signal quantification : Mass spectrometry</h2>

<p><p align="justify">Provides quantitation of target compounds (taking into account biological matrix effect) following Mass spectrometry imaging experiments. Quantinetix is a Quantitative Imaging Mass Spectrometry Software which offers normalization to get “real images” and provides concentration of target compounds. It is specially designed to ADMET study, PK/PD study, Toxicity study, In support of Whole Body Autoradiography and Proteomics and Lipidomics studies.</p>

<p><a href="https://www.imabiotech.com/quantinetix/" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Video</strong>:</p>

<p><a href="https://www.youtube.com/watch?v=jEWtjR7KUUs" target="_blank">Video</a></p>

<ul>
<li>(2010 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/20139469" target="_blank">Metscape: a Cytoscape plug-in for visualizing and interpreting metabolomic data in the context of human metabolic networks.</a></li>
</ul>

<p><strong>MAINTAINER</strong>:</p>

<p>ImaBiotech <a href="mailto:fontaine.edma@imabiotech.com" target="_blank">fontaine.edma@imabiotech.com</a></p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="ms-based-targeted-lipidomics">MS-based targeted lipidomics</h1>

<p>Taken together, LC-ESI-MS is currently the gold-standard technique for targeted lipidomics of second messengers when in ‘discovery’ mode with the research goal of identifying all possible species in a given tissue/ sample, whereas shotgun methodologies enable highthroughput identification and quantification of changes in known profiles.</p>

<hr />

<h2 id="mrmprobs-multiple-reaction-monitoring-based-probabilistic-system">MRMPROBS | Multiple Reaction Monitoring-based PROBabilistic System</h2>

<p><p align="justify">Allows metabolome analysis of large-scale multiple reaction monitoring (MRM) assays. MRMPROBS is a universal program for targeted metabolomics using multiple reaction monitoring (MRM)- or selected reaction monitoring (SRM), as well as SCAN and data independent tandem mass spectrometry (MS/MS) acquisition (DIA) data. The software was developed for metabolomics but can also be applied to lipidomics and proteomics studies.</p>

<p><a href="http://prime.psc.riken.jp/Metabolomics_Software/MRMPROBS/index.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2014 Bioinformatics) <a href="https://www.ncbi.nlm.nih.gov/pubmed/24753485" target="_blank">MRMPROBS suite for metabolomics using large-scale MRM assays.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>RIKEN Center for Sustainable Resource Science, Tsurumi-ku, Yokohama, Kanagawa, Japan;</p>

<p>Department of Biotechnology, Graduate School of Engineering, Osaka University, Suita, Osaka, Japan</p>

<hr />

<h2 id="mrm-diff-multiple-reaction-monitoring-based-differential">MRM-DIFF | Multiple Reaction Monitoring-based DIFFerential</h2>

<p><p align="justify">Allows multiple reaction monitoring (MRM)-based differential analysis. MRM-DIFF supports chromatographic alignment and compound identification with estimation of isotopic peaks. The software semi-automatically performs lipid identification and quantification of large scale MRM datasets. It includes three features: (1) use of every peaks detected by each transition, (2) manual curation of the identification and quantification results through the GUI interface, and (3) support of all data processing steps and pooled quality control (QC) data sets.</p>

<p><a href="http://prime.psc.riken.jp/Metabolomics_Software/MRMPROBS/index.html" target="_blank"><strong>Official Website</strong></a></p>

<p><strong>Publications</strong>:</p>

<ul>
<li>(2014 Front Genet) <a href="https://www.ncbi.nlm.nih.gov/pubmed/25688256" target="_blank">MRM-DIFF: data processing strategy for differential analysis in large scale MRM-based lipidomics studies.</a></li>
</ul>

<p><strong>Institutions(s)</strong>:</p>

<p>Metabolome Informatics Research Team, Metabolomics Research Group, RIKEN Center for Sustainable Resource Science, Yokohama, Japan</p>

<p><a href="#top"><i class="fa fa-hand-o-up fa-1x "></i>Top</a></p>

<hr />

<h1 id="find-more-tools-omictools-https-omictools-com-metabolomics-category">Find more tools: <a href="https://omictools.com/metabolomics-category" target="_blank"><strong>OMICTOOLS</strong></a></h1>

<p><a href="https://www.thermofisher.com/hk/en/home/industrial/mass-spectrometry/mass-spectrometry-learning-center/mass-spectrometry-applications-area/metabolomics-mass-spectrometry/lipidomics-workflows.html" target="_blank"><em>Image Citation</em></a></p>

<hr />

    </div>

    


<div class="article-tags">
  
  <a class="btn btn-primary btn-outline" href="/tags/metabolomics" target="_blank">metabolomics</a>
  
</div>




    
    
    <div class="article-widget">
      <div class="hr-light"></div>
      <h3>Related</h3>
      <ul>
        
        <li><a href="/too2/new_meta/profiles/">Metabolic Profiles</a></li>
        
        <li><a href="/too2/new_meta/engineer/">Metabolic Engineering</a></li>
        
        <li><a href="/too2/new_meta/drug/">Drug Discovery</a></li>
        
      </ul>
    </div>
    

    

    


  </div>
</article>

<footer class="site-footer">
  <div class="container">
    <p class="powered-by">

      &copy; 2018 &middot; 

      Copyright© 2018 Lin QI of Dr.Xin Wang's Lab. zzx. All rights reserved. 

      <span class="pull-right" aria-hidden="true">
        <a href="#" id="back_to_top">
          <span class="button_icon">
            <i class="fa fa-chevron-up fa-2x"></i>
          </span>
        </a>
      </span>

    </p>
  </div>
</footer>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <button type="button" class="close btn-large" data-dismiss="modal">&times;</button>
        <h4 class="modal-title">Cite</h4>
      </div>
      <div>
        <pre><code class="modal-body tex"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-primary btn-outline js-copy-cite" href="#" target="_blank">
          <i class="fa fa-copy"></i> Copy
        </a>
        <a class="btn btn-primary btn-outline js-download-cite" href="#" target="_blank">
          <i class="fa fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    <script async defer src="//maps.googleapis.com/maps/api/js?key=AIzaSyD3AGpKDAfo_LjBRHd-YkYmMjBDlIHfdlo"></script>
    

    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js" integrity="sha512-3P8rXCuGJdNZOnUx/03c1jOTnMn3rP63nBip5gOP2qmUh5YAdVAvFZ1E+QLZZbC1rtMrQb+mah3AfYW11RUrWA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/js/bootstrap.min.js" integrity="sha512-iztkobsvnjKfAtTNdHkGVjAYTrrtlC7mGp/54c40wowO7LhURYl3gVzzcEqGl/qKXQltJ2HwMrdLcNUdo+N/RQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/gmaps.js/0.4.25/gmaps.min.js" integrity="sha256-7vjlAeb8OaTrCXZkCNun9djzuB2owUsaO72kXaFDBJs=" crossorigin="anonymous"></script>
    
    
    <script src="/js/hugo-academic.js"></script>
    

    
    
      
      
      <script src="//cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
      

      

      

      <script>hljs.initHighlightingOnLoad();</script>
    

    
    

  </body>
</html>

