+++
# Date this page was created.
date = "2018-04-11"

# Project title.
 title = "Nucleosome Positioning"

weight = 20
# Project summary to display on homepage.
summary = "Including: Nucleosome positioning, MNase-seq analysis, ATAC-seq analysis."

# Optional image to display on homepage (relative to `static/img/` folder).
image_preview = "tools/geno/position.jpg"

# Tags: can be used for filtering projects.
# Example: `tags = ["machine-learning", "deep-learning"]`
tags = ["epigenomics"]

# Optional external URL for project (replaces project detail page).
#external_link = "https://cityu-bioinformatics.netlify.com/tools/seq_new/sequence alignment/"


# Does the project detail page use math formatting?
math = false

# Optional featured image (relative to `static/img/` folder).
#[header]
#image = "headers/bubbles-wide.jpg"
#caption = "My caption :smile:"


+++
---

<img src="/img/tools/geno/position.jpg" width="500" height="400" alt="position" align="center">

<span id="top"></span>

{{% toc %}}

# Nucleosome Positioning


# MNase-seq analysis

<p align="justify">The technology of micrococcal nuclease (MNase) digestion combined with high-throughput sequencing (MNase-seq) is a powerful method to map the genome-wide distribution of nucleosome occupancy (Mavrich et al., 2008; Schones et al., 2008; Jiang and Pugh, 2009). Source text: Chen et al., 2014.

# ATAC-seq analysis

<p align="justify">ATAC-seq allows simultaneous interrogation of factor occupancy, nucleosome positions in regulatory sites, and chromatin accessibility genome wide by considering the position of insertion and the distribution of insert lengths captured during the transposition reaction. 

[<i class="fa fa-hand-o-up fa-1x "></i>Top](#top)

## Find more tools: [**OMIVTOOLS**](https://omictools.com/epigenomics-category)

---